<!DOCTYPE html><html> <head><meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta name="viewport" content="width=device-width, initial-scale=1.0" /> <!--metatextblock--><title>Научные проекты</title><meta name="description" content="Институт биоинформатики" /><meta property="og:url" content="https://bioinf.me/education/projects" /><meta property="og:title" content="Научные проекты" /><meta property="og:description" content="Институт биоинформатики" /><meta property="og:type" content="website" /><meta property="og:image" content="https://static.tildacdn.com/tild3338-3131-4434-a536-336433386231/share-pic-18.png" /> <link rel="canonical" href="https://bioinf.me/education/projects"><!--/metatextblock--><meta property="fb:app_id" content="257953674358265" /><meta name="format-detection" content="telephone=no" /><meta http-equiv="x-dns-prefetch-control" content="on"><link rel="dns-prefetch" href="https://tilda.ws"><link rel="dns-prefetch" href="https://static.tildacdn.com"><link rel="shortcut icon" href="https://static.tildacdn.com/tild6265-3030-4832-b937-623232356261/favicon.ico" type="image/x-icon" /><!-- Assets --><link rel="stylesheet" href="https://static.tildacdn.com/css/tilda-grid-3.0.min.css" type="text/css" media="all" /><link rel="stylesheet" href="https://tilda.ws/project29135/tilda-blocks-2.12.css?t=1569356471" type="text/css" media="all" /><link rel="stylesheet" href="https://static.tildacdn.com/css/tilda-animation-1.0.min.css" type="text/css" media="all" /><link rel="stylesheet" href="https://static.tildacdn.com/css/tilda-menusub-1.0.min.css" type="text/css" media="all" /><link rel="stylesheet" href="https://static.tildacdn.com/css/tooltipster.min.css" type="text/css" media="all" /><script src="https://static.tildacdn.com/js/jquery-1.10.2.min.js"></script><script src="https://static.tildacdn.com/js/tilda-scripts-2.8.min.js"></script><script src="https://tilda.ws/project29135/tilda-blocks-2.7.js?t=1569356471"></script><script src="https://static.tildacdn.com/js/lazyload-1.3.min.js" charset="utf-8"></script><script src="https://static.tildacdn.com/js/tilda-animation-1.0.min.js" charset="utf-8"></script><script src="https://static.tildacdn.com/js/tilda-menusub-1.0.min.js" charset="utf-8"></script><script src="https://static.tildacdn.com/js/jquery.tooltipster.min.js" charset="utf-8"></script><script type="text/javascript">window.dataLayer = window.dataLayer || [];</script></head><body class="t-body" style="margin:0;"><!--allrecords--><div id="allrecords" class="t-records" data-hook="blocks-collection-content-node" data-tilda-project-id="29135" data-tilda-page-id="2093640" data-tilda-page-alias="education/projects" data-tilda-formskey="f0696df0d2ceecbcfdc18140c73b1757" ><div id="rec41128846" class="r t-rec" style="background-color:#ffffff; " data-animationappear="off" data-record-type="230" data-bg-color="#ffffff"><div id="nav41128846" class="t199__header t199__js__header " style="" data-menu="yes"> <div class="t199__holder"> <a class="t199__logo" href="/" style=""> <img style="max-width: 60px; width: 100%; height: auto;" src="https://static.tildacdn.com/tild6464-3064-4237-a433-383539613333/bi_logo.png" class="t199__logo-image-one" imgfield="img"><img src="https://static.tildacdn.com/tild3261-3534-4537-a238-653361653739/bi_logo_white.png" class="t199__logo-image-two" style="max-width: 60px; width: 100%; height: auto;" imgfield="img2"> <div class="t199__logo-text-mobile t-title" field="title" style=""></div> </a> <a class="t199__mmenu-toggler t199__js__menu-toggler" href="#"> <span class="t199__mmenu-toggler-in"></span> </a> <div class="t199__mmenu t199__js__menu"> <nav class="t199__menu"> <div class="t199__menu-item-wrap"> <a class="t199__menu-item t-title t-menu__link-item" href="#submenu:education" >ОБРАЗОВАНИЕ</a> </div> <div class="t199__menu-item-wrap"> <a class="t199__menu-item t-title t-menu__link-item" href="#submenu:science" >НАУКА</a> </div> <div class="t199__menu-item-wrap"> <a class="t199__menu-item t-title t-menu__link-item" href="#submenu:community" >СООБЩЕСТВО</a> </div> <div class="t199__menu-item-wrap"> <a class="t199__menu-item t-title t-menu__link-item" href="#submenu:popularization" >ПОПУЛЯРИЗАЦИЯ</a> </div> </nav> <div class="t199__lang"> <a class="t199__lang-item" style="" href="http://bioinf.me/en">En</a> </div> </div> </div></div><script type="text/javascript">$(document).ready(function() { t199_showMenu('41128846'); t199_positionHeader('41128846');
});</script><script type="text/javascript">$(document).ready(function() { t199_highlight('41128846'); t199_checkAnchorLinks('41128846');
});</script><style>#rec41128846 .t-menu__link-item{ -webkit-transition: color 0.3s ease-in-out, opacity 0.3s ease-in-out; transition: color 0.3s ease-in-out, opacity 0.3s ease-in-out;	}
#rec41128846 .t-menu__link-item.t-active{ color:#3a81e8 !important; }
#rec41128846 .t-menu__link-item:not(.t-active):not(.tooltipstered):hover{ opacity: 0.9 !important;}
@supports (overflow:-webkit-marquee) and (justify-content:inherit)
{ #rec41128846 .t-menu__link-item, #rec41128846 .t-menu__link-item.t-active { opacity: 1 !important; }
}</style><script type="text/javascript"> $(document).ready(function() { setTimeout(function(){ t_menusub_init('41128846'); }, 500); });</script><style>@media screen and (max-width: 980px) { #rec41128846 .t-menusub__menu .t-menusub__link-item { color:#000000 !important; }
}</style></div><div id="rec101626036" class="r t-rec" style=" " data-animationappear="off" data-record-type="398" ><div class="t398" data-tooltip-hook="#submenu:education" data-tooltip-position="bottom" data-add-arrow="on" data-tooltip-margin="15px"> <div class="t398__content"> <ul class="t398__list"> <li class="t398__list_item t-name t-name_xs"><a class="t398__typo t-name t-name_xs" style="font-weight:400;letter-spacing:0.5px;font-family:&apos;Open Sans&apos;;" href="/education" data-menu-item-number="1">Образование</a></li> <li class="t398__list_item t-name t-name_xs"><a class="t398__typo t-name t-name_xs" style="font-weight:400;letter-spacing:0.5px;font-family:&apos;Open Sans&apos;;" href="/education/program" data-menu-item-number="2">Программы в Санкт-Петербурге</a></li> <li class="t398__list_item t-name t-name_xs"><a class="t398__typo t-name t-name_xs" style="font-weight:400;letter-spacing:0.5px;font-family:&apos;Open Sans&apos;;" href="/education/program/moscow" data-menu-item-number="3">Программы в Москве</a></li> <li class="t398__list_item t-name t-name_xs"><a class="t398__typo t-name t-name_xs" style="font-weight:400;letter-spacing:0.5px;font-family:&apos;Open Sans&apos;;" href="/education/projects" data-menu-item-number="4">Научные проекты</a></li> <li class="t398__list_item t-name t-name_xs"><a class="t398__typo t-name t-name_xs" style="font-weight:400;letter-spacing:0.5px;font-family:&apos;Open Sans&apos;;" href="/education/summer" data-menu-item-number="5">Летняя школа</a></li> <li class="t398__list_item t-name t-name_xs"><a class="t398__typo t-name t-name_xs" style="font-weight:400;letter-spacing:0.5px;font-family:&apos;Open Sans&apos;;" href="/education/ngs" data-menu-item-number="6">Интенсив</a></li> <li class="t398__list_item t-name t-name_xs"><a class="t398__typo t-name t-name_xs" style="font-weight:400;letter-spacing:0.5px;font-family:&apos;Open Sans&apos;;" href="/education/sbw" data-menu-item-number="7">Выездной семинар</a></li> <li class="t398__list_item t-name t-name_xs"><a class="t398__typo t-name t-name_xs" style="font-weight:400;letter-spacing:0.5px;font-family:&apos;Open Sans&apos;;" href="/education/online" data-menu-item-number="8">Онлайн-образование</a></li> <li class="t398__list_item t-name t-name_xs"><a class="t398__typo t-name t-name_xs" style="font-weight:400;letter-spacing:0.5px;font-family:&apos;Open Sans&apos;;" href="/education/masters/itmo" data-menu-item-number="9">Магистратура «Биоинформатика и системная биология»</a></li> <li class="t398__list_item t-name t-name_xs"><a class="t398__typo t-name t-name_xs" style="font-weight:400;letter-spacing:0.5px;font-family:&apos;Open Sans&apos;;" href="/education/faq" data-menu-item-number="10">FAQ</a></li> </ul> </div></div><style>.t398__tooltipster-noir_101626036 .t398__list_item .t398__typo.t-active{ font-weight:700 !important; opacity:0.40 !important;
}</style><style type="text/css"> .t398__tooltipster-noir_101626036 { background-color: #fff !important; color: #222 !important; text-align: left !important; max-width: 200px !important; border-radius: 3px !important; ; box-shadow: 0px 0px 7px 0px rgba(0, 0, 0, 0.20); }</style><script type="text/javascript"> $(document).ready(function() { t398_init('101626036'); });</script> </div><div id="rec41128848" class="r t-rec" style=" " data-animationappear="off" data-record-type="398" ><div class="t398" data-tooltip-hook="#submenu:science" data-tooltip-position="bottom" data-add-arrow="on" data-tooltip-margin="15px"> <div class="t398__content"> <ul class="t398__list"> <li class="t398__list_item t-name t-name_xs"><a class="t398__typo t-name t-name_xs" style="font-weight:400;letter-spacing:0.5px;font-family:&apos;Open Sans&apos;;" href="/research" data-menu-item-number="1">Научные исследования</a></li> </ul> </div></div><style>.t398__tooltipster-noir_41128848 .t398__list_item .t398__typo.t-active{ font-weight:700 !important; opacity:0.40 !important;
}</style><style type="text/css"> .t398__tooltipster-noir_41128848 { background-color: #fff !important; color: #222 !important; text-align: left !important; max-width: 200px !important; border-radius: 3px !important; ; box-shadow: 0px 0px 7px 0px rgba(0, 0, 0, 0.20); }</style><script type="text/javascript"> $(document).ready(function() { t398_init('41128848'); });</script> </div><div id="rec41128849" class="r t-rec" style=" " data-animationappear="off" data-record-type="398" ><div class="t398" data-tooltip-hook="#submenu:community" data-tooltip-position="bottom" data-add-arrow="on" data-tooltip-margin="15px"> <div class="t398__content"> <ul class="t398__list"> <li class="t398__list_item t-name t-name_xs"><a class="t398__typo t-name t-name_xs" style="font-weight:400;letter-spacing:0.5px;font-family:&apos;Open Sans&apos;;" href="/community" data-menu-item-number="1">Сообщество</a></li> <li class="t398__list_item t-name t-name_xs"><a class="t398__typo t-name t-name_xs" style="font-weight:400;letter-spacing:0.5px;font-family:&apos;Open Sans&apos;;" href="/community/team" data-menu-item-number="2">Команда</a></li> <li class="t398__list_item t-name t-name_xs"><a class="t398__typo t-name t-name_xs" style="font-weight:400;letter-spacing:0.5px;font-family:&apos;Open Sans&apos;;" href="/community/lectors" data-menu-item-number="3">Преподаватели и кураторы</a></li> <li class="t398__list_item t-name t-name_xs"><a class="t398__typo t-name t-name_xs" style="font-weight:400;letter-spacing:0.5px;font-family:&apos;Open Sans&apos;;" href="/community/students" data-menu-item-number="4">Студенты</a></li> <li class="t398__list_item t-name t-name_xs"><a class="t398__typo t-name t-name_xs" style="font-weight:400;letter-spacing:0.5px;font-family:&apos;Open Sans&apos;;" href="/community/graduates" data-menu-item-number="5">Выпускники</a></li> </ul> </div></div><style>.t398__tooltipster-noir_41128849 .t398__list_item .t398__typo.t-active{ font-weight:700 !important; opacity:0.40 !important;
}</style><style type="text/css"> .t398__tooltipster-noir_41128849 { background-color: #fff !important; color: #222 !important; text-align: left !important; max-width: 200px !important; border-radius: 3px !important; ; box-shadow: 0px 0px 7px 0px rgba(0, 0, 0, 0.20); }</style><script type="text/javascript"> $(document).ready(function() { t398_init('41128849'); });</script> </div><div id="rec41128850" class="r t-rec" style=" " data-animationappear="off" data-record-type="398" ><div class="t398" data-tooltip-hook="#submenu:popularization" data-tooltip-position="bottom" data-add-arrow="on" data-tooltip-margin="15px"> <div class="t398__content"> <ul class="t398__list"> <li class="t398__list_item t-name t-name_xs"><a class="t398__typo t-name t-name_xs" style="font-weight:400;letter-spacing:0.5px;font-family:&apos;Open Sans&apos;;" href="/popularization" data-menu-item-number="1">Популяризация</a></li> <li class="t398__list_item t-name t-name_xs"><a class="t398__typo t-name t-name_xs" style="font-weight:400;letter-spacing:0.5px;font-family:&apos;Open Sans&apos;;" href="http://biohack.ru" target="_blank" data-menu-item-number="2">Хакатон BioHack</a></li> <li class="t398__list_item t-name t-name_xs"><a class="t398__typo t-name t-name_xs" style="font-weight:400;letter-spacing:0.5px;font-family:&apos;Open Sans&apos;;" href="/contest" target="_blank" data-menu-item-number="3">Онлайн-олимпиада Bioinformatics Contest</a></li> <li class="t398__list_item t-name t-name_xs"><a class="t398__typo t-name t-name_xs" style="font-weight:400;letter-spacing:0.5px;font-family:&apos;Open Sans&apos;;" href="http://slack.bioinf.me/" target="_blank" data-menu-item-number="4">Чат BioHub</a></li> </ul> </div></div><style>.t398__tooltipster-noir_41128850 .t398__list_item .t398__typo.t-active{ font-weight:700 !important; opacity:0.40 !important;
}</style><style type="text/css"> .t398__tooltipster-noir_41128850 { background-color: #fff !important; color: #222 !important; text-align: left !important; max-width: 200px !important; border-radius: 3px !important; ; box-shadow: 0px 0px 7px 0px rgba(0, 0, 0, 0.20); }</style><script type="text/javascript"> $(document).ready(function() { t398_init('41128850'); });</script> </div><div id="rec41128851" class="r t-rec" style=" " data-record-type="231" > <script type="text/javascript"> $('.t199__header').toggleClass('t199__dark'); </script> </div><div id="rec41129283" class="r t-rec t-rec_pb_0" style="padding-bottom:0px;background-color:#3c3c50; " data-animationappear="off" data-record-type="167" data-bg-color="#3c3c50"><!-- cover --><div class="t-cover" id="recorddiv41129283" bgimgfield="img" style="height:50vh; background-image:url('https://static.tildacdn.com/tild6265-3831-4430-a637-663833393364/-/resize/20x/3af8e78675d7443e83bb.jpg');" ><div class="t-cover__carrier" id="coverCarry41129283" data-content-cover-id="41129283" data-content-cover-bg="https://static.tildacdn.com/tild6265-3831-4430-a637-663833393364/3af8e78675d7443e83bb.jpg" data-content-cover-height="50vh" data-content-cover-parallax="fixed" style="height:50vh; "></div> <div class="t-cover__filter" style="height:50vh;background-image: -moz-linear-gradient(top, rgba(60,60,80,0.90), rgba(60,60,80,0.50));background-image: -webkit-linear-gradient(top, rgba(60,60,80,0.90), rgba(60,60,80,0.50));background-image: -o-linear-gradient(top, rgba(60,60,80,0.90), rgba(60,60,80,0.50));background-image: -ms-linear-gradient(top, rgba(60,60,80,0.90), rgba(60,60,80,0.50));background-image: linear-gradient(top, rgba(60,60,80,0.90), rgba(60,60,80,0.50));filter: progid:DXImageTransform.Microsoft.gradient(startColorStr='#193c3c50', endColorstr='#7f3c3c50');"></div><div class="t-container"> <div class="t-col t-col_12"><div class="t-cover__wrapper t-valign_middle" style="height:50vh;"> <div class="t153 t-align_center" data-hook-content="covercontent"> <div class="t153__uptitle t-uptitle t-uptitle_sm" style="" field="subtitle"><div style="color:rgba(255,255,255,0.77);" data-customstyle="yes"></div></div> <h1 class="t153__title t-title t-title_xxs" style="" field="title"><div style="font-size:52px;line-height:66px;text-align:center;" data-customstyle="yes"><span style="font-weight: 300;">Научные проекты</span></div></h1> <div class="t153__descr t153__descr_center t-descr t-descr_xs" style="" field="descr"><div style="text-align:center;" data-customstyle="yes"></div></div> <span class="space"></span> </div> </div> </div> </div> </div> </div><div id="rec130268376" class="r t-rec t-rec_pt_60 t-rec_pb_90" style="padding-top:60px;padding-bottom:90px;background-color:#f5f5f5; " data-animationappear="off" data-record-type="61" data-bg-color="#f5f5f5"><!-- T051 --><div class="t051"><div class="t-container"><div class="t-col t-col_12 "><div class="t051__text t-text t-text_md" style="" field="text"><div style="font-size:28px;line-height:44px;text-align:center;color:#3c3c50;" data-customstyle="yes"><br />За время обучения в Институте биоинформатики каждый студент занимается несколькими <strong>научно-исследовательскими проектами.</strong> Руководят проектами ведущие специалисты из российских и зарубежных научных лабораторий и компаний, работающих в области биоинформатики и биотехнологий.<br /><br />Если у вас есть интересные задачи по биоинформатике для студентов пишите нам на <span style="font-size: 24px; text-align: start;"><a href="mailto:info@bioinf.me." style=""><span style="font-size: 28px;">info@bioinf.me.</span></a></span><br /></div></div></div></div></div></div><div id="rec101625275" class="r t-rec t-rec_pt_60 t-rec_pb_15" style="padding-top:60px;padding-bottom:15px; " data-record-type="106" ><!-- T004 --><div class="t004"><div class="t-container "> <div class="t-col t-col_12 "><div field="text" class="t-text t-text_md " style=""><div style="font-size:24px;" data-customstyle="yes">Проект подразумевает еженедельные встречи с руководителем очно или по скайпу, а также 5-10 часов в неделю самостоятельной работы. Многие проекты выливаются в дальнейшее сотрудничество со студентами и потом выпускниками. Подробнее о формате проектов можно прочитать <a href="https://docs.google.com/document/d/e/2PACX-1vTmGv76TwH7TmS9GHSX473iVcxo-Ip9i_tjudAMZFxNCmh0E9RtY3mcfeXq-_Qoxup5b18jgAU5ysQM/pub" style="font-size: 24px;" target="_blank">в этом гиде</a>.<br /><br />Институт биоинформатики публикует на сайте итоги и презентации всех студенческих проектов. С 2017 года результаты семестровых проектов переводятся на английский язык<br /><br />Научные проекты разных лет:<br /><br /> <ul> <li> <a href="https://tilda.cc/page/2018" style="color:#1155cc !important;">2018/19</a></li> <li> <a href="https://tilda.cc/page/?pageid=2093640#2017">2017/18</a></li> <li> <a href="https://tilda.cc/page/?pageid=2093640#2018">2014 — 2016</a></li></ul></div></div></div></div></div></div><div id="rec69613513" class="r t-rec" style=" " data-record-type="215" ><a name="2018" style="font-size:0;"></a></div><div id="rec69561779" class="r t-rec t-rec_pt_60 t-rec_pb_60" style="padding-top:60px;padding-bottom:60px; " data-record-type="30" ><!-- T015 --><div class="t015"> <div class="t-container t-align_center"> <div class="t-col t-col_10 t-prefix_1"> <div class="t015__title t-title t-title_lg" field="title" style="font-size:40px;">Научные проекты 2018/2019</div> </div> </div></div></div><div id="rec51524595" class="r t-rec t-rec_pt_30 t-rec_pb_30" style="padding-top:30px;padding-bottom:30px; " data-record-type="397" ><!-- t397 --><div class="t397"> <div class="t-container"> <div class="t397__col t-width t-width_6"> <div class="t397__wrapper t-align_center"> <div class="t397__tab t397__tab_active t397__width_50" data-tab-rec-ids="41447892,69562533,69562653,69562610,69562784,69562915,69563064,69563304,69563440,69563678,69563799,69563952,69564224,69564327,69564439,69564564,69564649,69564775,69566202,69564885,69565047,69565153,69564065,72914430" style="border-bottom: 1px solid #eee;;"> <div class="t397__title t-name t-name_xs" style="font-weight:400;" field="title">Осень 2018</div> </div> <div class="t397__tab t397__width_50" data-tab-rec-ids="97588121,98523045,97588079,97588007,97587865,97587629,97587484,97587312,97587235,97587129,97586915,97586713,97586567,98521917,97586480,97586332,97586098,97585986,97585832,97585615,97585399,97584482,97586190" style="border-bottom: 1px solid #eee;;"> <div class="t397__title t-name t-name_xs" style="font-weight:400;" field="title2">Весна 2019</div> </div> </div> <div class="t397__wrapper_mobile"> <div class="t397__firefoxfix"></div> <select class="t397__select t-name"> <option value="41447892,69562533,69562653,69562610,69562784,69562915,69563064,69563304,69563440,69563678,69563799,69563952,69564224,69564327,69564439,69564564,69564649,69564775,69566202,69564885,69565047,69565153,69564065,72914430">Осень 2018</option> <option value="97588121,98523045,97588079,97588007,97587865,97587629,97587484,97587312,97587235,97587129,97586915,97586713,97586567,98521917,97586480,97586332,97586098,97585986,97585832,97585615,97585399,97584482,97586190">Весна 2019</option> </select> </div> </div> </div></div><style type="text/css"> #rec51524595 .t397__tab_active { border-bottom-color: #222 !important; } #rec51524595 .t397__tab_active .t397__title { color: #222 !important; font-weight: 600 !important; } #rec51524595 .t397__select { border: 1px solid #222; color: #222; } #rec51524595 .t397__wrapper_mobile:after { border-color: #222 transparent transparent transparent; } #rec51524595 .t397__firefoxfix { top: 1px; bottom: 1px; right: 1px; } </style><script type="text/javascript"> $(document).ready(function() { t397_init('51524595'); }); </script></div><div id="rec41447892" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Study spectrum of genetic variants in TTN gene</strong> | Federal Almazov Medical Research Centre<br /><br /><span style="font-size: 20px;"><strong>students:</strong> Olga Lebedenko<br /><strong>scientific adviser:</strong> Artem Kiselev</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">The TTN gene with 363 coding exons encodes titin, a giant muscle protein spanning from the Z-disk to the M-band within the sarcomere. Titin has roles in assembling and maintaining sarcomere structure, flexibility, stability, stretch and force transmission. Mutations in the TTN gene have been associated with various cardiomyopathies. <br /><br /> The main aim of this study was to investigate spectrum of genetic variants in TTN gene within group of patients with cardiomyopathy. 151 different type of cardiomyopathy samples, sequenced with Haloplex custom targeted capture, were processed with SNP Calling pipeline implemented on Snakemake and annotated by snpEff. Among 418 discovered SNP 64.44% variants were missens and 35.56% variants were silence. The PCA analysis showed absence of clustering SNP by type of cardiomyopathy. Fisher's exact tests with Bonferroni correction were used to compare allele frequencies of observed variants against all gnomAD population. Pathogenicity of 12 discovered statistically meaningful missense variants was predicted by algorithms SIFT, PolyPhen-2, Mutation Assessor, Provean and I-Mutant 3.0. <br /><br /> Almost all variants showed neutral effect on protein structure and stability. The most interesting SNP was the mutation rs9808377 I62T, presumably affecting on stability of the subunit Fn3-102 titin by I-Mutant 3.0. Presumably, this result may be explained by difficulties in multiple comparison connected with a high rate of spontaneous mutation owing to enormous size of TTN gene. Another reasons in analyzing accompanying TTN variants in cardiomyopathy group with confirmed well-knowing causative mutations. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/lebedenko_15122018.pdf#overlay-context=node/1526" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec69562533" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>A library of functions for express analysis of FASTA/FASTQ files</strong> | Bioinformatics Institute<br /><span style="font-size: 20px;"><strong><br />students:</strong> Alena Kizenko, Alisa Morshneva, Polina Pavlova<br /><strong><strong data-redactor-tag="strong">scientific adviser:</strong></strong> Eugene Bakin</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Frequently, when carrying out bioinformatics projects including FASTA/FASTQ files processing, one has to solve routine tasks, e.g. deduplication of sequences. A common approach for this is writing little scripts in Python/bash or dealing with existing programs, which may be complicated for usage. Therefore, we decided to create a flexible tool containing functions for processing files with sequencing data. We created a program called BreakFAST which is based on Python 3 and the following libraries: Biopython, argparse, pandas, numpy, matplotlib and re. The tool consists of three modules. Basic statistics module can be used for counting: <br /><br /> ● minimum, maximum, mean, total length of reads; <br /> ● GC-content; <br /> ● quality scores; <br /> ● N bases. <br /><br /> Filtering module can be used for deleting: <br /> ● reads shorter than X; <br /> ● reads containing Ns; <br /> ● poor quality reads; <br /> ● duplicates; <br /> ● reads with a particular motif. <br /><br /> Matching module can be used for: <br /> ● joining reads from files; <br /> ● finding overlapping between files; <br /> ● subtracting sets of reads from files. <br /><br /> While applying commonly suggested Biopython functions we've faced performance problems while parsing a large volume of data. For mitigation of this effect while iterating over FASTA/FASTQ files, we compared SeqIO.parser and Iterator from Biopython. We have found that usage of Iterator in Filtering and Matching modules was optimal for iteration (10 times speed gain). Notably, we compared function "delete reads shorter than X" with the same Trimmomatic's function and found that BreakFAST occupies up to 7 times less RAM, which may be useful when a computer`s capacity is limited. As a result, BreakFAST is a simple and customizable tool, which can be potentially updated with new modules and functions. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/morshneva_pavlova_kizenko_15122018.pdf#overlay-context=node/1526" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec69562784" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Visualization of signaling pathways basing on genes differential expression profile</strong> | First Pavlov State University of St. Petersburg<br /><br /><span style="font-size: 20px;"><strong>students:</strong> Stanislav Legkovoy, Olga Romanova<br /><strong><strong data-redactor-tag="strong"><strong>scientific advisers</strong>:</strong></strong> Eugene Bakin, Oksana Stanevich</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:20px;" data-customstyle="yes">In recent years, Gene Expression Omnibus (GEO) NCBI database has accumulated a significant amount of data obtained via mRNA microarrays, which are widely used for an analysis of differential expression profile. During a research of genes expressions, a proper visualization of results is an important task. One of the best ways is to use R language and related packages for statistical analysis, preprocessing and visualization of expression data through interaction with KEGG database. <br /><br /> An aim of this study was to visualize signaling pathways according to the gene expression data obtained from GEO NCBI database. To achieve our goal, we implemented an easy-to-use script based on pathview, gage and GEOquery packages, which allowed us to obtain gene expression data directly from the GEO NCBI database and to find the most significant signaling pathways from KEGG PATHWAY. Using the developed script, we analyzed the Affymetrix microarray data, identified and visualized the most significant signaling pathways involved in reprogramming of lymphatic endothelial cells infected by human Kaposi's sarcoma-associated herpesvirus (KSHV). <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/legkovoy_romanova_15122018.pdf#overlay-context=node/1526" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec69562915" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Comparative analysis of NUMT in underground and terrestrial rodents</strong> | Zoological Institute RAS<br /><br /><span style="font-size: 20px;"><strong>students:</strong> Ekaterina Sytnik<br /><strong>scientific adviser:</strong> Olga Bondareva</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">NUMT (nuclear mitochondrial DNA segment) is a transposition of mitochondrial DNA into nuclear genome. They are found in all eukaryotes but significantly differ in length and number among different species. Particular factors that can be associated with NUMTs are still not determined. Due to the specificity of mitochondrial genes, habitat conditions may be one of the factors. <br /><br /> They aim of this study was to estimate the number of NUMT for underground and terrestrial rodents. For this work we only analyzed long (&gt;300 n.p.) NUMTs of protein- coding regions. We used Genbank database for mitochondrial and nuclear genomes (4 species for each group) and BLAST for NUMT searching. It was found that some genes like ND4L and ATP8 are not likely to be included in NUMTs, which may be caused by the small size of the genes. Some underground species are shown to have a larger amount of long NUMTs but it is yet unclear if the same is true for whole group. <br /><br /> Further study should include larger amount of species and dN/dS analysis for each gene to determine whether some of the NUMTs may have a functional role. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/sytnik_15122018.pdf#overlay-context=node/1526" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec69563064" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>The comparative analysis of MDR <em data-redactor-tag="em">Klebsiella pneumoniae</em> genome</strong> | Children's Scientific and Clinical Center for Infectious Diseases<br /><br /><span style="font-size: 20px;"><strong>students:</strong> Anastasia Kapanina, Nina Lukashina, Daria Likholetova<br /><strong>scientific advisers: </strong>Eugene Bakin<span class="redactor-invisible-space">, Oksana Stanevich<span class="redactor-invisible-space" data-redactor-class="redactor-invisible-space">, Sergey S. Sidorenko </span></span></span></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"><em>Klebsiella pneumoniae</em> is a gram-negative bacteria that is known as opportunistic, hypervirulent, and multidrug resistant hospital pathogen. The problem of resistance to carbenemase group of antibiotics makes it one of the main threats during hospitalisation. The diversity of <em>K. pneumoniae</em> is studied by whole-genome sequencing (WGS) and multiple typing methods including multi-locus sequence typing (MLST), that separate strains into different lineages. In our study we assembled and analysed genomes of 22 isolates of different years and sources from the Saint-Petersburg hospitals to identify their origin and describe their pangenome. <br /><br /> With use of Kleborate tool, we found that our strains belong to common european and asian MLST types (ST147, ST11, ST340 and ST395). All of them carry NDM-1 and ParC resistance genes, and only one - OXA-48. According to the genes discovered in strains, we listed inefficient antibiotics for their treatment. Via PlasmidFinder we detected a presence of plasmid R27 of <em>Salmonella typhi</em>, that can be explained by contamination of samples or by horizontal transfer between <em>K.pneumoniae</em> and <em>S. typhi</em>. <br /><br /> According to an existing literature, the obtained MLST types are spread in Europe and Asia. However, for obtaining a more detailed result about an origin of the strains, a genome structure analysis is needed. In conclusion, we can say that within the period from 2012 to 2016 there were no invasions of new sequence types on a territory of mentioned hospitals. The obtained results of pangenome analysis can be used in treatment prescription. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/lukashina_licholetova_15122018.pdf#overlay-context=node/1526" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec72914430" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Comparative analysis of the human pathogens genomes <em data-redactor-tag="em">Neisseria meningitidis</em></strong> | Children`s Scientific and Clinical Center for Infectious Diseases<br /><br /><span style="font-size: 20px;"><strong>students:</strong> Anton Matiiv, Ilia Sheshukov<br /><strong>scientific advisers:</strong> Eugene Bakin, Oksana Stanevich, Sergey S. Sidorenko</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Neisseria meningitidis or meningococcus often colonizes the mucous membrane of the oropharynx, causing no visible symptoms, but is also the main cause of bacterial meningitis and sepsis throughout the world. The epidemiological profile of N. meningitidis varies in different populations, and over time, the virulence of meningococcus is based on the plastic genome and the expression of certain capsular polysaccharides and non-capsular antigens. Twelve different serogroups based on the polysaccharide capsule have been identified, but only six of them (A, B, C, W, X and Y) account for 90% of the invasive meningococcal disease worldwide. Seven housekeeping genes for meningococcal strains are used for MLST (multilocus sequence typing) to determine their sequence types (ST). <br /><br />The aim of our work was to compare whole genome sequencing data of 20 Neisseria meningitis samples isolated from carriers and sick people, to use phylogenetic analysis and to find a connection with antibiotic resistance, virulence and carriage. <br /> <br />Before the analysis of sequences, we have written a computer script for interfacing and downloading reference genomes from NCBI. We analyzed the antigen-encoding, virulence and carriage associated and antibiotic resistant gene profiles. We also searched for amino acid changes leading to penicillin resistance. To estimate the relationship between samples, phylogenetic trees were constructed on the basis of isolates assemblies by using CSI Phylogeny and REALPHY. We constructed phylogenetic trees for carriage associated genes to figure out if the samples would cluster according to their origin of isolation. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/matiiv_sheshykov_15122018.pdf#overlay-context=node/1526" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec69563304" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Finding of cis-regulatory elements in promoters</strong> | University of La Verne<br /><br /> <span style="font-size: 20px;"><strong>students:</strong><strong></strong> Daria Balashova, Elena Polyakova<br /> <strong>scientific adviser:</strong><strong></strong> Tatiana Tatarinova</span></span><br /></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">We consider a genome-wide statistical approach for the detection of specific DNA sequence motifs based on similarities between the promoters of similarly expressed genes. A comprehensive landscaping of major regulatory motifs can contribute to understanding molecular mechanisms of many complex diseases. <br /><br /> Assuming position-specificity of the function of promoter motifs, providing gene expression data of reasonable measurements of the number of transcripts and reflecting of the activity of the promoter, we develop cisExpress software that includes the algorithm for finding statistically significant associations between words of defined length with respect to the transcription start site in the expression dataset. Subsequent optimization includes combining motifs that have small differences and clustering basic words of fixed size into larger composite motifs. The analysis of time series, conducted on the basis of Hidden Markov Models, allows us to observe the significance of the found motifs over time. The tool is complemented by interactive graphical representations. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/polyakova_balashova_15122018.pdf" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec69563440" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Analysis of the Drosophila melanogaster full genome sequences</strong> | EPAM Systems, Lifesciencs department<br /><br /><span style="font-size: 20px;"><strong></strong><strong>students: </strong><strong></strong>Anna Namyatova<br /><strong>scientific adviser:</strong> Gennady Zakharov</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Drosophila melanogaster is a model object for studying insect genomes. The results can be used to make prediction on the human diseases. We had the Illumina full genome sequences for two wild type lines and two mutant lines (ts3 and X1). In the ts3 line, the defects were artificially induced with the behaviour being restored to normal after the thermal shock. The defects in the line X1 were spontaneous and permanent. The aim was to compare the mutant line genomes with each other and with those of the wild type lines, and find the genes responsible for the abnormalities in the nervous system structure and function. We used the following tools for our analysis: <br /><br /> 1. FastQC. Sequences quality check. <br /> 2. Trimmomatic Trimming the bad quality nucleotides. <br /> 3. Bwa. Genome assembly. <br /> 4. Samtools. Creating, sorting and indexing the .bam file. <br /> 5. Picard. Adding the Readgroups into .bam files. <br /> 6. Gatk. Variation calling. <br /> 7. Vcf-merge. Merging the wild type lines mutations. <br /> 8. Rtg vcfeval. Comparing the each mutant line mutations with those in the wild type <br /> lines. <br /><br /> The genomes were mapped against the reference genome Drosophila_melanogaster.BDGP6.dna_sm.toplevel.fa. FastQS showed that there were around 30 million reads in each genome, the length of reads ranges between 35 and 76 bp in the raw sequences. The total number of mutations in the wild type line was 1209056. Each mutant line had around 800000 mutations. There were 195743 unique mutations in the ts3 line, and 174653 unique mutations in the X1 line. In the future we are going to perform snpeff and snpsift tools to annotate the mutations, to assign the biological meaning to them and to exclude the nonsense mutations. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/namyatova_15122018.pdf#overlay-context=node/1526" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec69563678" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Transcriptional response of pea roots to symbiosis markers</strong> | All-Russia Research Institute for Agricultural Microbiology<br /><br /><span style="font-size: 20px;"><strong>students:</strong> Varvara Tvorogova<br /><strong>scientific advisers: </strong><strong></strong>Polina Kozyulina, Elena Dolgikh</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Root nodules in legumes are symbiotic organs hosting nitrogen-fixing bacteria. At the beginning of the formation of nodule, bacteria enter the intercellular space of the root; therefore, the host plant needs an accurate recognition system that allows it to let symbiotic bacteria pass inside its tissues and block parasitic organisms from doing the same. The main external signals that provide such recognition are chitooligosaccharides of different lengths. Thus, chitooligosaccharides consisting of five monomers (co5) are markers of symbiotic bacteria, while chitooligosaccharides consisting of eight monomers (co8) are markers of parasitic organisms (insects and fungi). <br /><br /> The purpose of this study was to analyze the data of MACE-sequencing of pea (Pisum sativum) RNA from roots pretreated with co5 or co8 chitoligosaccharides. Using the pea nodule transcriptome obtained previously (Zhukov et al., 2015) and the Dedupe software from BBTools package, we removed ambiguous transcripts and got the optimal reference transcriptome for our data. Then, using the DESeq2 and GSEABase packages, we analyzed differential gene expression in our samples and performed gene enrichment analysis. According to the results obtained, co5 treatment shows more prominent differential gene expression compared to co8 probably due to incomplete reference transcriptome. However, both co5 and co8 chitoligosaccharide treatments activate gene sets that are responsible for parasite-host interaction, chitin binding and cleavage, as well as numerous signaling pathways which include different phytohormones, receptor kinases and transcription factors. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/tvorogova_15122018.pdf#overlay-context=node/1526" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec69563799" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Modeling of mouse chromosome banding pattern</strong> | Bioinformatics institute<br /><br /><span style="font-size: 20px;"><strong>students:</strong> Yury Lebeda<br /><strong></strong><strong>scientific adviser:</strong><strong></strong> Yury Barbitoff</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Differential chromosome staining is a method of chromosome staining with special dyes to detect certain discs or regions of the chromosome (also called chromosome bands). The resulting banding pattern is an important marker of genome architecture; however, no specific molecular determinants of it are known to date. Previously, our group discovered the relationship between the pattern of differential staining of chromosomes and several genomic features (ChIP-Seq tracks of Smc1a/Smc3, CTCF, polyA and polyT repeats). However, when validation of this relationship using the genome of Mus musculus was attempted, it was found that the distribution of genomic elements within the M. musculus bands differs from that observed in humans. In this project, we took an effort to develop a model that would be able to predict the border regions between bands, on the basis of the human genome data, and apply this model to predict the banding of M. musculus chromosomes. <br /> <br />We built a random forest model to predict the borders of the bands based on the number of genomic elements (i.e., ChIP-Seq peaks or k-mers) lying in the intervals of a given width inside and outside of the band borders. For prediction, the M. musculus genome was cut into intervals of the same width by a sliding window; and the resulting intervals were annotated with the same features that were used to train the model. Unfortunately, all the models constructed (despite high cross-validation AUC scores) failed to provide reasonable predictions – both for the mouse and human genomes, the results of the prediction of band boundaries differed from the already existing markup. The results can be explained by a large number of false-positive results, which becomes significant even with a small false-positive rate at large numbers of trials. Hence, a new model has to be sought for to explain the nature of chromosome bands. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/lebeda_15122018_0.pdf" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec69563952" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>De novo assembly and analysis of <em data-redactor-tag="em">Platynereis dumiliii</em> (Nereididae, Annelida) transcriptome <br /> at different stages of regeneration</strong> | Saint-Petersburg State University<br /><br /><span style="font-size: 20px;"><strong></strong><strong>students:</strong><strong></strong> Natalia Zenkova, Ruslan Abasov<br /><strong>scientific adviser:</strong> Maxim Nesterenko</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Regeneration – the regrowth or repair of cells, tissues and organs – is widely but non- uniformly represented among all animal phyla. However, the potency of its highly variable even within a single group. The object of this study is the polychaeta Platynereis dumerilii (Nereididae, Annelida), capable to recover only tail. RNA-seq data of different time points after amputation (0, 4, 12, 24 hours, 2 and 4 days) from "head" and "tail" sites of regeneration were analyzed. Libraries of corrected read pairs (Karect, Trimmomatic, BBtools) were used to the de novo assembly of reference transcriptome (Trinity). <br /><br />The resulting assembly was characterized by high quality (TransRate-score = 0.2441) and completeness (BUSCO vs Metazoa-odb9 = 99.5%). The amino acid sequences predicted by TransDecoder (N = 160381) were compared to the Swiss-Prot database using the Diamond (e-value = 1e-10). More than 61% of the sequences were successfully annotated, but among the sequences without hits we assume the presence of species-specific proteins. Based on the normalized expression levels analysis results (Salmon), sets of "associated" sequences were highlighted for each of the samples. We suggest that incomplete overlap between "associated" sets both between time points and between sites indicate complex dynamics of gene activity during postamputation events. However, expression patterns of regeneration conservative genes (for instance: Piwi-, Vasa-, Wnt- and Notch-like) varies slightly between "head" and "tail" sites. Based on the results obtained, it can be assumed that cell proliferation is not over on 4 days after amputation and damaged structure recovery will be observed at later stages of generation. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/abasov_zenkova_15122018_0.pdf#overlay-context=node/1526" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec69564065" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Analysis of nonsense alleles of Caenorhabditis elegans genes</strong> | Lomonosov Moscow State University<br /><br /><span style="font-size: 20px;"><strong>students:</strong> Daria Chaplygina<br /><strong>scientific adviser:</strong> Nadezda Potapova</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Nonsense mutation in gene is a mutation that results in a premature stop codon. Most of genes with nonsense alleles translates into a nonfunctional proteins, which makes such genes to be a pseudogenes. The purpose of this study was to analyze the distribution of nonsense mutations in Caenorhabditis elegans genes and to perform a direct measurement of the strength of negative selection acting on nonsense alleles. For the measurement we counted the average ratio of the number of nonsynonymous mutation to the number of synonymous mutations for each gene (pN/pS ratio). The obtained pN/pS ratio then was compared to the pN/pS ratio in genes without nonsense mutations. Genome sequences processing was performed by SAMtools, VarScan and SnpEff. <br /><br /> According to the obtained results, the most of synonymous mutations are located at the 3′-end of gene, where they are less harmful. Also it was shown that nonsense alleles, common for many species in population, are rare, which must be due to negative selection against them. The average pN/pS ratio appears to be about 1 for genes without nonsense alleles and slightly more than 1 (1.2) for genes with nonsense alleles. Such results means that negative selection does not act on any gene, which can not be true. The mistake could be explained by possibly wrong variant annotation. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/chaplygina_15122018.pdf#overlay-context=node/1526" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec69564224" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Prioritization of genetic variants</strong> | Bioinformatics Institute<br /><br /><span style="font-size: 20px;"><strong>students:</strong> Vasiliy Isaev, Liubov Lonishin <br /><strong>scientific adviser:</strong> Yury Barbitoff</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">The identification of deleterious mutations within candidate genes is a crucial step in the elucidation of the genetic bases of human disease, consequently, there is a need to aim attention at classifying appropriate mutations. The goal of our programme, which is called MutationsPriorityPredictionTool (MPPT) is to find out these genetic variants from thousands of others in order to help clinicians and geneticists. <br /> To calculate the coefficient we have developed an <a href="https://github.com/Vasiliy566/Mutations-Priority-Prediction-ToolVCFparser/" target="_blank" style="">tool</a>. Tool get on input vcf-file and set of simple configurations that contain rules on how to calculate mutation priority score depends from parameters given in the file. After calculating tool will print top of mutation by parameter specified by user. It can be top 10% of mutations or 100 mutations ot other option. We have tested our programme on whole exome sequencing data, obtained from the resource centre. First of all, the selection of the test sample was made in accordance with the ClinVar database and was compared with results of <a href="https://franklin.genoox.com/" target="_blank" style="">Franklin</a>, which is based on ACMG recommendations (The American College of Medical Genetics and Genomics). The percentage number of correct calls by MPPT was calculated, and the sensitivity and specificity of the method was determined. Accuracy of our programme is 67,5%, sensitivity is about 100% (95% CI = 79.4% to 100.00) and specificity is 60,6% (95% CI =53.9% to 67.3%). Testing on the whole data, we obtained 114 mutations above the threshold from more than 22 thousand at all. <br /><br /> MPPT focuses on pathogenic variants without losing them, but also keeps some benign variants which should be manually checked by a specialist after running. In the future, we will add this functionality to NGB (New Genome Browser). <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/lonishin_isaev_15122018.pdf#overlay-context=node/1526" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec69564327" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Assembly of yeast genome with Oxford Nanopore data</strong> | Bioinformatics Institute<br /><br /><span style="font-size: 20px;"><strong>students:</strong> Andrew Matveenko<br /><strong>scientific advisers:</strong> Yury Barbitoff, Alexander Predeus</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Baker's yeast Saccharomyces cerevisiae is a widely used model organism. The Peterhof genetic collection (PGC) is a large laboratory stock unrelated to the yeast reference strain. Previously, several PGC strains were sequenced using Ion Torrent technology. However, the resulting assemblies were incomplete and required substantial improvement. We attempted to obtain a reference quality assembly of one PGC strain, 1A-D1628, using Oxford Nanopore Technology (ONT) sequencing. <br /><br /> Raw data was obtained from one ONT MinIon flowcell which generated 10.15 Gbp total sequence length (836x coverage). To create draft genome assembly we used three long-read assemblers: Canu, Flye and wtdbg2. Canu produced the best results, with 17 large (&gt; 50 kbp) contigs that correspond to 16 yeast chromosomes and mitochondrial DNA. Flye was slightly worse with 18 large contigs as it failed to assemble chromosome III as a single molecule. Wtdbg2 failed to produce any sensible sequence. Comparison of the Canu assembly with the reference showed that it contained 105 misassemblies, and large amount of mismatches and short indels. We also analyzed structural variations in the strain using NGMLR-Sniffles pipeline. The results of analysis were concordant with variations described previously in 1A- D1628. <br /><br /> In conclusion, the data obtained from the Oxford Nanopore sequencing can be used to analyse structural variations in the 1A-D1628 strain. However, the de novo assembly requires additional correction and polishing to reach the reference quality. Several strategies can be used to achive this goal. First, an alternative basecaller can be used to improve the quality of reads. Second, exclusively for ONT, the assembly can be improved by polishing with the MinIon raw signal using Nanopolish tool. And finally, polishing the assembly with the obtained Illumina reads should improve the accuracy of the sequence producing high quality reference, which can be used for comparative genomic studies. <br /> The work is supported the RSF grant 18-14-00050. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/matveenko_15122018_edit.pdf" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec69564439" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>The study of processes of gene gain and loss within Lactobacillus species</strong> | Zoological Institute RAS<br /><br /><span style="font-size: 20px;"><strong>students:</strong> Anastasia Kosolapova<br /><strong>scientific adviser:</strong> Olga Bondareva</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Lactobacillus genus includes Gram-positive non-sporulating bacteria known for their ability to produce lactic acid as a result of carbohydrate fermentation. To date more than 180 species refer to Lactobacillus genus. A hallmark of that genus is a high level of intra-group diversity. Firstly, the diversity exhibits in ecology of the group as lots of Lactobacillus species are associated with cavities of human and animals, for example gastrointestinal tract and urogenital tract, while others can be found on plants, in dairy and fermented products. Secondly, the genome size of Lactobacillus bacteria can vary between 1.2 Mb and 5 Mb. <br /><br />The aim of this work was to study connection between ecological specificity and genome organization within various strains of Lactobacillus and analyze influence of ecological specificity on processes of gain and loss of genes. As a data for analysis we used protein and CDS sequences for 185 Lactobacillus species (1708 strains) from RefSeq database. Lactococcus lactis subsp. lactis Il1403 protein and CDS sequences were used as an outgroup. We classified species into 7 groups based on ecological niche. We revealed orthologous proteins within strains using Proteinortho5/POFF software. Further research should involve a phylogenetic tree reconstruction based on full orthologous genes groups followed by gain-loss analysis performed with GLOOME software. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/kosolapova_15122018.pdf#overlay-context=node/1526" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec69564564" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Detection of interchromosomal rearrangements from Hi-C data</strong> | ITMO University<br /><br /><span style="font-size: 20px;"><strong>students:</strong> Elena Kartysheva, Dmitriy Orekhov<br /><strong><strong data-redactor-tag="strong">scientific adviser</strong>:</strong> Nikita Alexeev</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"> Chromosomal rearrangements disturb complex 3D structure of eukaryotic genome and may lead to various disease among which is cancer, detecting them may be useful in early diagnostics. Hi-C is a relatively recent sequencing method that estimates 3D proximity between different regions of a sequenced genome, this type of data allows for detection of different chromosomal abnormalities. We have developed an algorithm that scans through Hi-C map and reports the presence of interchromosomal rearrangements with the coordinates of their breakpoints. The algorithm relies on 2D convolution and GMM for filtering out the data and detection of interchromosomal interactions, then a sliding-window approach is used for breakpoint localization. The method is tested on Hi-C maps obtained from glioblastoma cells of <em>H.Sapiens</em>, showing both high precision and high recall.<br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/kartysheva_orechov_15122018.pdf" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec69564649" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Plasmid host range prediction based on CRISPR arrays. Plasmids CRISPR Cas systems search.</strong> | CAB SPbU<br /><br /><span style="font-size: 20px;"><strong>students:</strong> Mikhail Kongoev, Iana Fedorova<br /><strong>scientific adviser:</strong> Mikhail Rayko</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Horizontal gene transfer plays a highly important role in evolution of bacteria. Presumably, gene exchange between bacteria occurs by genetic mobile elements such as plasmids and bacteriophages. But for nowadays there is no reliable way to check if the certain plasmid can "travel" between bacteria of different origin, and how wide the plasmid host range could be. Also it is important to be able to predict plasmid host in case of metagenomic data, where we usually have dozens of novel plasmids without any information of host species. <br /><br /> To answer this question, we analyzed CRISPR cassettes in bacterial genomes – repetitive sequences in bacterial DNA, interspaced with unique "spacer" sequences, which were extracted from genetic mobile elements infected the bacteria or its ancestors. Spacers in CRISPR cassette can be considered as a link between the plasmid and its host. <br /><br /> We used CRISPR Finder spacers database and the RefSeq database of all plasmids known to date (November 2018). Blasting spacers over plasmids sequences allowed us to determine plasmid host ranges: variety of bacterial organisms where the plasmid can exist. By taxonomy analysis we found some plasmids which can live in different families of organisms, they can be useful in genetic engineering as a natural shuttle vectors. Taxonomy analysis showed that a bunch of plasmids have additional hosts except of host they were related to according to RefSeq database: 543 blast hits – additional hosts of different genus, 29 blast hits - different family, 19 - different order, 12 - different class and even 2 blast hits – additional hosts of different phylum! Thus, plasmids are actually "travelling" between bacteria species and can be important players in process of evolution. <br /><br /> We also found, that a lot of plasmids carry their own defense CRISPR systems (10% of RefSeq plasmids). Part of these systems (10%) seems to be active – there are Cas1 genes near CRISPR cassettes. Role of these systems in plasmid propagation, host fitness and evolutionary relationship with the known chromosomal CRISPR-Cas systems is the subjects of future research. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/fedorova_kongoev_15122018_0.pdf" target="_blank" style="">Slides</a><br /></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec69564775" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Automated pathway annotation for single-cell RNA-seq</strong> | Washington University in Saint Louis / ITMO University<br /><br /><span style="font-size: 20px;"><strong>students:</strong> Maria Firuleva<br /><strong>scientific adviser:</strong> Konstantin Zaitsev</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Method of single-cell RNA-seq expands the opportunities to research a biological difference between cells of interest by the individual transcriptome analysis of each cell simultaneously and to study the cell's processes more deeply. Increasing pace of RNA-seq methods expects automated approaches to process a huge amount of that data. Different cell processes are mediated by a different set of genes (signal pathways), and expression of appropriate genes changes due to activation or deactivation of appropriate signal pathways. The main target of this project is to develop a method for automated annotation of pathways which are significantly upregulated in the single-cell dataset. <br /><br />We developed a three-step approach to identify differentially expressed pathways, which is applied after performing the usual single-cell rna-seq pipeline using Seurat package. First, we calculate how each pathway is expressed in every cell. Second, randomly sampling gene sets we identify candidate cells that in which pathways are upregulated more than at random. Third, we identify clusters in which there are more candidate cells than at random, using hypergeometric distribution. As a result, our program returns a matrix with cell clusters as columns and pathways as rows which values are adjusted p-values. Developed approach combined with cumulative statistic approaches allows to quickly find significantly upregulated pathways in a single-cell dataset for all clusters and large gene set databases. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/firuleva_15122018.pdf#overlay-context=node/1526" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec69566202" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Automated marker descriptor for single-cell RNA-seq</strong> | Washington University in Saint Louis / ITMO University<br /><br /><span style="font-size: 20px;"><strong>students:</strong> Daria Gorbach<br /><strong>scientific adviser:</strong> Konstantin Zaitsev</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Method of single-cell RNA-seq provides an opportunity to detect gene expression and specific cellular processes from lots of cells simultaneously, and each cell type has its own combination of expressed markers, which helps to discriminate one cell population from another. Increasing rate of RNA-seq technologies demands automated methods to obtain increasing amount of that data. Popular approach of cell types identification is based on "one versus all" method, which compares gene expression profiles for one cellular cluster with all the rest. It is a way to find statistically significant markers for each cluster, however, this method fails to identify "unique" cluster markers and quite often reports markers that are not unique for a certain cluster. Cell surface markers are of particular interest for that kind of research, as they are most frequently serve as markers of specific cell types. <br /><br /> Our approach was to make an automated descriptor, using pair-wise comparison of expressed markers. We used MGI database for mice cell surface proteins and "Seurat" package – R toolkit for single-cell data analysis. Thus, we compared expression levels of each cell surface marker between different cellular subtypes (T h -lymphocytes, macrophages, etc.) and obtained one or several unique markers expressed uniquely in each subtype. This method allows us to describe each cluster with a set of unique surface marker genes identifying any of that type. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/gorbach_15122018.pdf#overlay-context=node/1526" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec69564885" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>The effect of X chromosome inactivation on the expression of autosomal genes</strong> | Bioinformatic institute<br /><br /><span style="font-size: 20px;"><strong>students:</strong> Daria Kilina<br /><strong>scientific advisers:</strong> Yury Barbitov, Rostoslav Skitchenko</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">X chromosome inactivation (XCI) silences the transcription of genes located on one of the X chromosomes to balance expression dosage between XX females and XY males. According to a recent work by Tukiainen et al., there is no total X chromosome inactivation in humans as up to one-third of X-chromosomal genes are expressed from both the active and inactive X chromosomes in female cells. Howeer, the effects of XCI on the expression profile of autosomal genes have not yet been assessed. In this study we compared the expression of autosomal genes in cells with different active X chromosome copies. <br /><br /> To answer this qestion we analyzed public experimental data of single-cell RNA-sequencing of pancreatic islets from one female individual. We aligned the reads coming from each cell to a reference genome assembly using bowtie2. We then performed variant calling with samtools/bcftools in order to group the cells by active X chromosome by visual inspection of the alignments and SNP calls in IGV. We grouped the cells by alleles at variant sites in the XIST gene that is totally expressed from only one X chromosome. We then quantified gene expression levels with RSEM and used LIMMA plugin in the Phantasus browser to compare gene expression in the two groups of cells defined above. <br /><br /> We identified some candidate genes, expression of which depends on the active X chromosome copy; however, the difference in the expression levels of these genes between groups was not significant (P-value &lt; 0.05, adjusted P-value &gt; 0.05). Furthermore, we observed lack of clear separation between the two groups of cells based on principal component analysis (PCA), which may indicate confounding effect of cell types or other factors. Hence, the effect of differential genes expression from the X chromosomes on the expression profile of autosomal genes needs further investigation. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/kilina_15122018.pdf" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec69565047" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Increasing the length of introns due to transposable elements</strong> | Vavilov Institute of General Genetics Russian RAS<br /><br /><span style="font-size: 20px;"><strong>students:</strong> Anastasiia Murzina<br /><strong>scientific adviser:</strong> Irina Poverennaya</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Unlike exons - coding regions of a gene, intron sequences are known for a high degree of mutagenesis and, in accordance with this, great variability, so that even the length of an intron can differ greatly in related organisms. A significant increase in the length of introns may be due to the active accumulation in introns of a large number of transposable elements (TE) and repeats. In this project, our goal was to get depence between intron length and count of transposable elements. <br /><br /> The Dfam database is a collection of Repetitive DNA element sequence alignments. This database was used with RepeatMasker programm, which based on usage Hidden Markov Models, to search TE of human genome. TE and repeats take up 43 percent on the average of the length of all introns. There is a correlation between the intron length and the TE length for introns with a length of more than 300 nucleotides, however, very long introns (&gt; 12000) correlate with the TE length better than the average length introns. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/mursina_a_271018.pdf#overlay-context=node/1525" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec69565153" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Аnalysis of factors affecting the course of chronic myeloid leukemia</strong> | Pavlov First Saint Petersburg State Medical University<br /><br /><span style="font-size: 20px;"><strong>students:</strong> Nadezda Pogodina, Irina Babkina<br /><strong><strong data-redactor-tag="strong">scientific advisers</strong>:</strong> Eugene Bakin, Oksana Stanevich</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Chronic myeloid leukemia (CML) is a myeloproliferative disorder characterized by unregulated granulocytic proliferation. A standard treatment includes tyrosine kinase inhibitor (TKI) and hematopoietic stem cell transplantation (HSCT). In this project, our goal was to identify the factors that have an impact on survival after HSCT. The most common method of survival analysis is a Kaplan-Meier approach. These method works on censored data, when observation ended up before event of interest occurred. We plotted overall and event-free survival curves. Cumulative survival probability was 37.9%. Then, we performed analysis of single variables (conditioning regimens, phase of CML et al.) and compared survival curves in log-rank test. We identified one statistically significant factor: cyclophosphamide therapy after HSCT (p-value = 0.03). CML therapy has been improved, so we used multivariate analysis to assess the influence of new treatment methods on the survival using correlation test. Correlation matrix showed weak association, that's why we selected the following factors with the specialist`s help: conditioning regimens, phase of CML, graft compatibility, TKI therapy, cyclophosphamide therapy after HSCT. Ordination methods (PCA and MDS) showed 3 clusters of therapy factors, distributed between 3 era: 1995-2006, 2007-2012, 2013-2018. The eras' survival curves were also statistically different (p-value = 0.009). <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/pogodina_babkina_15122018.pdf#overlay-context=node/1526" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec97584482" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Поиск ключевых генов, инициирующих смену профиля экспрессии в клетке</strong> | ПСПбГМУ им. академ. И.П. Павлова<br /><br /><span style="font-size: 20px;"><strong>студенты:</strong> Горбач Дарья<br /><strong><strong data-redactor-tag="strong">руководители</strong>:</strong> Бакин Евгений, Станевич Оксана</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Важным направлением в биоинформатике является анализ смены профиля экспрессии генов в клетке под воздействием внешних факторов (например, при заражении вирусом). При этом может быть затронуто значительное количество генов, однако, как правило, этот масштабный процесс инициируется единицами ключевых генов, таргетных для вируса. Привлечение дополнительной информации о взаимосвязи генов, хранящейся в таких базах как KEGG, позволяет выявить это множество. Целью проекта является создание алгоритма, позволяющего по данным дифференциальной экспрессии восстановить причинно-следственных связи для процессов, проистекающих в клетке при воздействии вируса.<br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/gorbach_25052019.pdf" target="_blank" style="">Слайды</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec97585399" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Анализ геномов дрожжей из Петергофской генетической коллекции</strong> | Институт биоинформатики<br /><br /><span style="font-size: 20px;"><strong>студенты:</strong> Матиив Антон<br /><strong><strong data-redactor-tag="strong">руководители</strong>:</strong> Барбитов Юрий, Предеус Александр</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Данный проект является продолжением проекта по сборке генома дрожжей из данных Oxford Nanopore. В ходе данного проекта студенту будет предложено закончить сборку референсного генома штамма 1А-Д1628, а также сравнить геномы других штаммов (74-Д694 и др.) с полученной референсной сборкой. Также в рамках выполнения проетка студенту будет предложено найти генетические варианты, ассоциированные с определенными чертами фенотипа мутантных производных 1А-Д1628 (на основании данных, полученных с использованием технологии Illumina).<br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/matiiv_25052019.pdf" target="_blank" style="">Слайды</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec97585615" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Построение среды для обработки данных экзомного секвенирования</strong> | Институт биоинформатики<br /><br /><span style="font-size: 20px;"><strong>студенты:</strong> Творогова Варвара, Абасов Руслан<br /><strong><strong data-redactor-tag="strong">руководители</strong>:</strong> Барбитов Юрий, Шиков Антон</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Данный проект посвящен анализу данных полноэкзомного секвенирования для клинических целей. В данном проекте студенту(-ам) будет предложено разобраться в самых современных технологиях поиска вариантов в геноме человека (GATK4, WDL/Cromwell, Hail). Конечной целью проекта является построение пайплайна анализа данных на основе GATK4, валидации его с использованием стандартных бенчмарк-датасетов (Genome In A Bottle), а также написание серии утилит для интеграции анализа с веб-приложениями и удобного хранения всех метаданных в виде базы данных (SQL).<br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/abasov_tvorogova_25052019.pdf" target="_blank" style="">Слайды</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec97585832" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Implied weighting as a measure of clade support: automation of the task and comparative assessment of results </strong>| Санкт-Петербургский государственный университет<br /><br /><span style="font-size: 20px;"><strong>студенты:</strong> Сытник Екатерина<br /><strong><strong data-redactor-tag="strong">руководители</strong>:</strong> Данилов Лаврентий Глебович, Константинов Федор Владимирович</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">IW (implied weighting) - это метод дифференциального взвешивания признаков в парсимониальном филогенетическом анализе. Он используется как для молекулярных, так и для морфологических данных, однако по-настоящему широкое распространение получил именно в работах морфологов. Хорошо известно, что признаки (группы данных) неравноценны в плане адекватного отражения родственных связей отдельных таксонов и могут нести как сильный, так и слабый филогенетический сигнал или даже не иметь его вовсе. Основная идея любого типа взвешивания состоит в придании относительно большего веса хорошо укладывающимся на филогению признакам и меньшего – гомоплазиям. Анализ опубликованных за последние два десятилетия статей с реконструкцией филогении на основе морфологических данных свидетельствует о частом использовании апостериорного взвешивания. При этом исследователи крайне редко сравнивают результаты SAW и IW, используя что-то одно. Выбор определяется в первую очередь тем, какая именно программа была использована для парсимониального анализа. При этом популярность платного и многие годы не развивавшегося пакета PAUP за последние десятилетия катастрофически снизилась, а программы TNT, напротив, значительно возросла. Это и определило практически безальтернативное использование implied weighting в опубликованных в последнее время реконструкциях филогении на основе морфологических данных. Оправданность применения апостериорного взвешивания данных в парсимониальном анализе вызывает активные споры с момента появления соответствующих методов (e.g., Kluge 1997, 2005, Turner &amp; Zandee 1995, Goloboff 1995, Goloboff et al. 2008) с теоретической и философской точек зрения. Давняя дискуссия о биологическом смысле IW в целом и разных значений К в частности по-прежнему актуальна и значительно оживилась в последние три года (e.g., Tschopp et al., 2015; O'Reilly et al., 2016; Congreve &amp; Lamsdell, 2016; Goloboff, Torres &amp; Arias, 2017, etc.). При этом как сторонники, так и противники концепции апостериорного взвешивания единодушны во мнении о том, что она остро нуждается в эмпирическом материале. Казалось бы, к настоящему моменту мы уже располагаем результатами множества исследований, сделанных с использованием IW. Основная практическая проблема, однако, состоит в заложенной в IW возможности в широчайших пределах менять константу К и, соответственно, определяющую вес признаков функцию. В подавляющем большинстве случаев, исследователи выбирают два-четыре значения К, сравнивают результаты и выбирают одно из деревьев за основу для дальнейшей работы, иногда никак не поясняя причины выбора. Нередко используют и единственное значение K=3, заданное в программе TNT по умолчанию, хотя нежелательность использования низких, в диапазоне 1-4, значений К признавал и сам создатель IW и TNT (Goloboff, 2008). Апостериорное взвешивание во многом превратилось из метода оценки и выявления филогенетического сигнала в данных, в инструмент для получения более разрешенных и, по тем или иным причинам более привлекательных для исследователя топологий. Следует однако оговориться, что «размах изменчивости» топологий при разных К определяется данными и в случаях с сильным филогенетическим сигналом оказывается незначительным. Возможным вариантом является использование IW в качестве стресс-теста (Legg et al., 2013; Garwood &amp; Dunlop, 2014; Smith &amp; Ortega-Hernandez, 2014). Иными словами, разброс значений К, при котором сохраняется та или иная клада, может служить в качестве количественной оценки степени ее поддержки данными. С практической точки зрения, сторонники идеи использования IW в качестве меры поддержки ветвей проводят анализ с разными значениями К и затем вручную согласовывают полученные деревья правилом большинства (Majority-rule consensus tree). Чаще всего при этом подряд используют целые значения К в определенном диапазоне, например от 3 до 15.<br />Более технологичным решением стало бы создание TNT-скрипта, позволяющего автоматизировать этот процесс, параллельно сравнить полученные результаты (в табличном и majority-rule tree виде) с независимо посчитанными в каждом случае традиционными мерами поддержки (Bootstrap, GC-supports). Немаловажно также сохранить полученные результаты в читаемом другими программами виде, что также может быть решено с помощью скриптов TNT. Сами значения К, очевидно, должны быть стандартизированы, например через равномерные значения F.<br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/sytnik_25052019.pdf" target="_blank" style="">Слайды</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec97585986" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Предсказание антимикробных пептидов с помощью данных транскриптома немодельного вида </strong>| НИИ биологии ИГУ<br /><br /><span style="font-size: 20px;"><strong>студенты:</strong> Бабкина Ирина<br /><strong><strong data-redactor-tag="strong">руководители</strong>:</strong> Дроздова Полина</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Идея: озеро Байкал обладает уникальной глубоководной пресноводной фауной. Глубоководные рачки питаются трупами других обитателей озера, заражённые разнообразными бактериями. Логично предположить, что у них есть способы с этим бороться. Предлагаем посмотреть, есть ли у них известные или новые антимикробные пептиды: взять опубликованные данные секвенирования РНК для глубоководных (у которых мы ожидаем большое разнообразие) и литоральных видов (у которых мы ожидаем меньшее разнообразие) и посмотреть, что можно по ним предсказать.<br /><br />Цель: предсказание и характеристика спектра антимикробных пептидов (АМП) у глубоководных амфипод в сравнении с литоральными.<br /><br />С технической точки зрения здесь можно будет поработать с данными секвенирования РНК, собрать транскриптом и поиграть с предсказанием свойств пептидов.<br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/babkina_25052019.pdf" target="_blank" style="">Слайды</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec97586098" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Разнообразие опсинов в транскриптомах байкальских эндемичных амфипод </strong>| НИИ биологии ИГУ<br /><br /><span style="font-size: 20px;"><strong>студенты:</strong> Федорова Яна, Кизенко Алена<br /><strong><strong data-redactor-tag="strong">руководители</strong>:</strong> Дроздова Полина</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Зрение -- это важно, в том числе в воде. В озере Байкал обитает 350 видов амфипод, они очень ярко окрашены (в том числе есть внутривидовое разнообразие) и обитают на всех глубинах. Тут надо отметить, что свет с разной длиной волны с разной эффективностью проникает на разные глубины, и для байкальских коттоидных рыб (кстати, хищников, которые питаются в том числе амфиподами) известен сдвиг максимальной чувствительности опсинов в синюю область с увеличением глубины обитания.<br /><br />Цель: изучить разнообразие генов, кодирующих белки зрительной системы у байкальских амфипод, и поискать связь с их филогенией, глубиной обитания, яркостью окраски или другими особенностями вида.<br /><br />С технической точки зрения здесь можно будет поработать с собранными транскриптомами, предсказанием генов, разными методами выравнивания, построением филогенетических деревьев. И, конечно, поиграть в игру "запусти чужой код".<br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/kizenko_fedorova.pdf" target="_blank" style="">Слайды</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec97586190" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Whole-genome Drosophila sequence analysis </strong>| EPAM Systems, Lifesciencs department<br /><br /><span style="font-size: 20px;"><strong>студенты:</strong> Лебеда Юрий, Косолапова Анастасия<br /><strong><strong data-redactor-tag="strong">руководители</strong>:</strong> Захаров Геннадий</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Основу курса составляют задачи по управлению вычислениями и анализу данных геномного секвенирования. В качестве исходных данных предлагаются полногеномные сиквенсы на платформе Illumina пяти линий дрозофилы – три линии дикого типа и два ранее не секвенированных мутанта.<br />В осеннем семестре команда построила из набора открытых утилит пайплайн («конвейер», набор совместно работающих утилит) для автоматизированного анализа данных. С его помощью была выполнена базовая задача – очистка исходных прочтений, выравнивание на референсный геном и обнаружение вариаций (значащих отличия от референсного генома) у всех 5 линий.<br />Во втором семестре команде предлагается продолжить эту работу – очистить и проаннотировать полученные вариации, сравнить линии между собой и попытаться обнаружить вариации, ответственные за формирование мутантного фенотипа.<br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/kosolapova_lebeda_25052019.pdf" target="_blank" style="">Слайды</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec97586332" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Поиск латентных вирусов в данных полногеномного секвенирования человека </strong>| Genotek<br /><br /><span style="font-size: 20px;"><strong>студенты:</strong> Погодина Надежда, Моршнева Алиса, Орлов Юрий<br /><strong><strong data-redactor-tag="strong">руководители</strong>:</strong> Ильинский Валерий</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Известно, что в организме большинства людей присутствуют вирусы, не приводящие к каким-либо острым состояниям (HPV, HHV, EVB, TTV и другие). В ходе проекта предполагается проанализировать данные полногеномного секвенирования людей с целью поиска таких вирусов. Описание разнообразия латентных вирусов может отличаться в разных регионах и у людей с разными генетическими особенностями.<br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/orlov_pogodina_morshneva_25052019.pdf" target="_blank" style="">Слайды</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec97586480" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Поиск патогенных вариантов (SNPs, indels) в экзомах пациентов с различными вариантами идеопатических кардиомиопатий. </strong>| НМИЦ Алмазова<br /><br /><span style="font-size: 20px;"><strong>студенты:</strong> Килина Дарья<br /><strong><strong data-redactor-tag="strong">руководители</strong>:</strong> Киселёв Артём</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Лаборатория Молекулярной биологии и генетики НМИЦ и В.А. Алмазова регулярно проводит экзомное секвенирование у пациентов с идеопатическими случаями каридиомиопатий. Как правило, перед экзомным секвениерованием проводят анализ наследственного характера заболевания и секвенируют таких пациентов на малых панелях с целевым обогащением генов, описанных в литературе и известных в качестве причинных в различных вариантах кариомиопатий. Если малая панель не дает результатов, то проводят WES. WES и анализ редких вариантов при помощи баз данных частот позволяет однозначно определить патогенные варианты у ~70% таких пациентов. Оставшиеся пациенты нуждаются в расширенном анализе.<br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/kilina_25052019.pdf" target="_blank" style="">Слайды</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec98521917" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Поиск CNV в экзомах у пациентов с различными вариантами идеопатических кардиомиопатий </strong>| НМИЦ Алмазова<br /><br /><span style="font-size: 20px;"><strong>студенты:</strong> Романова Ольга<br /><strong><strong data-redactor-tag="strong">руководители</strong>:</strong> Киселёв Артём</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Лаборатория Молекулярной биологии и генетики НМИЦ и В.А. Алмазова регулярно проводит экзомное секвенирование у пациентов с идеопатическими случаями каридиомиопатий. Как правило, перед экзомным секвениерованием проводят анализ наследственного характера заболевания и секвенируют таких пациентов на малых панелях с целевым обогащением генов, описанных в литературе и известных в качестве причинных в различных вариантах кариомиопатий. Если малая панель не дает результатов, то проводят WES. WES и анализ редких вариантов при помощи баз данных частот позволяет однозначно определить патогенные варианты у ~70% таких пациентов. Оставшиеся пациенты нуждаются в расширенном анализе.<br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/romanova_25092019.pdf" target="_blank" style="">Слайды</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec97586567" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Поддержка сторонних графовых форматов в SPAdes </strong>| СПбГУ, ЦАБ<br /><br /><span style="font-size: 20px;"><strong>студенты:</strong> Зенкова Наталья<br /><strong><strong data-redactor-tag="strong">руководители</strong>:</strong> Коробейников Антон, Пржибельский Андрей</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Многие современные ассемблеры (в том числе и SPAdes) предоставляют на выходе не только геномные последовательности (контиги), но граф сборки. Однако, далеко не все программы имеют возможность принимать такие графы на вход. В этом проекте предлагается реализовать поддержку различных графовых форматов (FASTG/GFA) в качестве входных данных для модуля разрешения повторов SPAdes. Реализация такой функциональности заметно расширит область использования SPAdes, в частности, позволит осуществлять скаффолдинг на основе графов, построенных другими сборщиками.<br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/zenkova_25052019.pdf" target="_blank" style="">Слайды</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec97586713" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Байесовская оптимизация для вывода демографических историй </strong>| Университет ИТМО, Исследовательская лаборатория им. П. Л. Чебышева, ПОМИ РАН<br /><br /><span style="font-size: 20px;"><strong>студенты:</strong> Шешуков Илья<br /><strong><strong data-redactor-tag="strong">руководители</strong>:</strong> Носкова Екатерина, Боровицкий Вячеслав</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Демографическая история популяций это последовательность таких событий как миграция, разделение/слияние популяций, изменение их численностей. Современные методы позволяют строить правдоподобные гипотезы о демографической истории популяций по набору геномов, взятых у их "современных" представителей. Одним из ключевых этапов автоматизированных методов вывода демографической истории является оптимизация некоторой сложно-вычислимой функции без доступа к ее градиенту.<br /><br />Недавно вышедший тул GADMA (https://github.com/ctlab/GADMA) решает эту задачу с помощью генетических алгоритмов. В проекте предлагается реализовать возможность использования методов Байесовской оптимизации для тула GADMA, сравнить два подхода.<br /><br />Отметим, что байесовская оптимизация - не просто очередной алгоритм оптимизации, который можно за пять минут изучить в википедии, это весьма обширная и современная тема с множеством неожиданных возможностей, например оптимизации функции, используя приближения разной степени точности к ней, распараллеливание оптимизации и т.д. (вместо википедии см., например, https://arxiv.org/abs/1807.02811).<br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/sheshukov_25052019.pdf" target="_blank" style="">Слайды</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec97586915" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Адаптации рыб к жизни на большой глубине </strong>| ФББ МГУ<br /><br /><span style="font-size: 20px;"><strong>студенты:</strong> Намятова Анна, Полякова Елена<br /><strong><strong data-redactor-tag="strong">руководители</strong>:</strong> Потапова Надежда</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Рыбы обитают не в особенно гостеприимной среде, порой испытывая на себе колоссальное давление. Становится интересным узнать механизмы, с помощью которых они всё же могут прекрасно существовать и на глубинах в несколько километров. Известно всего несколько генов-кандидатов, которые могут обеспечивать существование в таких условиях, но количество таких исследований, равно как и число генов, ограничено.<br />Цель проекта -- составить список имеющихся данных для глубоководных рыб и понять, какие гены вовлечены в адаптации к глубоководному образу жизни.<br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/polyakova_namyatova_-_adaptation_of_the_fish_to_depth.pdf" target="_blank" style="">Слайды</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec97587129" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Интерактивные сети фенотипов / генов / метаболических путей. </strong>| Genotek<br /><br /><span style="font-size: 20px;"><strong>студенты:</strong> Лонишин Любовь<br /><strong><strong data-redactor-tag="strong">руководители</strong>:</strong> Ракитько Александр</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Для консультаций по результатам генетического теста удобно иметь перед глазами информацию о фенотипе (например, заболевании) и его взаимосвязях, представленную в структурированном виде. В данном проекте предлагается реализовать R-Shiny приложение, которое будет отображать в виде интерактивных сетей: - взаимосвязи между фенотипами (что является фактором риска чего);<br />- генные сети (с отмеченными на них вариантами пациента);<br />- метаболические пути.<br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/lonichin_25052019.pdf" target="_blank" style="">Слайды</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec97587235" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Влияние курения на эпигеном лейкоцитов человека </strong>| НИИ ФХБ имени А.Н.Белозерского МГУ; Институт биоинженерии РАН, ФИЦ «Фундаментальные основы биотехнологии» РАН<br /><br /><span style="font-size: 20px;"><strong>студенты:</strong> Павлова Полина, Фирулёва Мария<br /><strong><strong data-redactor-tag="strong">руководители</strong>:</strong> Сергеев Олег, Медведева Юлия</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Факторы окружающей среды, включая химические вещества, способны вызывать эпигенетические изменения, которые могут быть прослежены у последующих поколений. Наиболее изученными эпигенетическими изменениями являются метилирование ДНК, малые некодирующие РНК (к которой относят транспортную РНК (tRNA), микро РНК (mi RNA) и piRNA) и модификация гистонов. Курение остается одним из наиболее неблагоприятных добровольных рисков, влияющих на здоровье.<br /><br />Объект заявляемого молекулярно-эпидемиологического исследования – данные метилирования ДНК (RRBS) и данные профиля малой РНК лейкоцитов периферической крови, полученные в ходе родительского проспективного когортного исследования «Russian Children's Health. Male Reproductive SubStudy», у молодых мужчин в возрасте 18 лет.<br /><br />Что известно из предыдущих исследований?<br />Воздействие таких стойких токсикантов, как диоксины, в течение полового развития (Pilsner et al., 2018) и курения (Корниенко с соавт., 2018, студенческий проект Института биоинформатики) влияет на метилирование ДНК сперматозоидов в возрасте 18 лет.<br /><br />Что неизвестно?<br />Как влияет курение на метилирование ДНК и изменение профиля малой и миРНК лейкоцитов периферической крови в возрасте 18 лет и насколько сопряжены изменения эпигенома сперматозоидов и лейкоцитов у тех же участников исследования.<br /><br />Цель исследования:<br />Изучить роль влияния курения на метилирование ДНК и изменение профиля малой и миРНК лейкоцитов крови молодых мужчин.<br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/pavlova_firuleva_25052019.pdf" target="_blank" style="">Слайды</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec97587312" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Определение корректных параметров МД моделирования для исследования белковой динамики (в сотрудничестве с проф. Дэвидом Кейсом, создателем силового поля Амбер) </strong>| СПбГУ / Purdue University<br /><br /><span style="font-size: 20px;"><strong>студенты:</strong> Легковой Станислав, Лебеденко Ольга<br /><strong><strong data-redactor-tag="strong">руководители</strong>:</strong> Скрынников Николай</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Определение корректных параметров МД моделирования для исследования белковой динамики (в сотрудничестве с проф. Дэвидом Кейсом, создателем силового поля Амбер)<br /><br />Моделирование биомолекул методом Молекулярной Динамики (МД) является одним из самых важных и перспективных инструментов структурной биологии. Как правило, в такого рода исследованиях используется так называемый ансамбль NPT. Однако использование NPT со стандартным набором параметров баростата/термостата ведёт к существенному замедлению динамики. Мы выявили этот эффект путём сопоставления экспериментальных данных, полученных методом спектроскопии ЯМР, с предсказаниями, полученными из траекторий МД. В частности, нам удалось показать, что стандартный метод моделирования ведёт к существенной (более 2 раз) переоценке характерного времени вращения глобулярного белка в растворе. В настоящее время мы ставим перед собой следующие цели.<br />(1) Нам предстоит выяснить насколько существенно этот эффект проявляется по отношению к движениям боковых цепей белка.<br />(2) Нам также предстоит установить в какой мере данный эффект сказывается на динамике нативно разупорядоченных белков.<br />(3) Помимо этого нас интересует его влияние на трансляционное движение белка - в особенности, в приложении к разупорядоченным белкам.<br />(4) Мы намерены предложить альтернативные методы моделирования, позволяющие с высокой точностью воспроизводить временной масштаб различных форм белковой динамики (например, применение ансамбля NPT с пониженным коэффициентом трения, применение ансамбля NVE).<br />Для записи траекторий МД наша лаборатория оснащена компьютерами на графических процессорах последнего поколения. Работа над проектом ведётся в сотрудничестве с проф. Дэвидом Кейсом (Rutgers University), создателем и ведущим разработчиком одного из двух самых совершенных силовых полей (Amber).<br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/legkovoy_lebedenko_25052019.pdf" target="_blank" style="">Слайды</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec97587484" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Определение наиболее вероятного генотипа и этнической принадлежности индивида исходя из генотипа его потомков </strong>| University of La Verne<br /><br /><span style="font-size: 20px;"><strong>студенты:</strong> Исаев Василий<br /><strong><strong data-redactor-tag="strong">руководители</strong>:</strong> Татьяна Татаринова</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">В судебно-медицинской практике есть такая задача: есть жертва изнасилования, есть ребенок родившийся в результате преступления. Преступник (отец) неизвестен. Как наиболее точно получить его описание исходя из генотипов матери и ребенка? Какие методы (полногеномное секвенирование или анализ на чипах) будут наиболее эффективными? Можно ли достоверно определить этническую принадлежность отца? На эти вопросы мы будем искать ответ в этом семестре.<br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/isaev_25052019.pdf" target="_blank" style="">Слайды</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec97587629" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Локальное выравнивание последовательностей с использованием внутрипроцессорного параллелизма </strong>| University of Warwick<br /><br /><span style="font-size: 20px;"><strong>студенты:</strong> Орехов Дмитрий<br /><strong><strong data-redactor-tag="strong">руководители</strong>:</strong> Тискин Александр</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Локальное выравнивание символьных последовательностей - одна из фундаментальных задач биоинформатики. Широко распространены как быстрые эвристические (тима BLAST), так и точные, но более трудоемкие методы локального выравнивания. Руководителем предлагаемого проекта был разработан относительно простой и эффективный метод точного локального выравнивания по принципу "скользящего окна", реализация которого позволила получить биологически значимые результаты, опубликованные в 2010-2012 гг. Эффективность реализации была достигнута, в том числе, использованием команды архитектуры Intel MMX с низкоуровневым внутрипроцессорным SIMD (single instruction, multiple data) параллелизмом. В последние годы данный тип внутрипроцессорного параллелизма интенсивно развивался. Компанией Intel были последовательно внедрены несколько новых наборов команд для параллельного манипулирования данными: SSE (несколько версий до SSE4 включительно), AVX, AVX-512. Аналогичные расширения архитектуры микропроцессоров были реализованы и другими производителями.<br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/orekhov_25052019.pdf" target="_blank" style="">Слайды</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec97587865" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Association Rule Mining on genome regions using fishbone diagrams </strong>| JetBrains Research<br /><br /><span style="font-size: 20px;"><strong>студенты:</strong> Лукашина Нина, Лихолетова Дарья<br /><strong><strong data-redactor-tag="strong">руководители</strong>:</strong> Цуринов Пётр</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">В биологии поиск ассоциаций является важной частью построения гипотез. Текущие подходы включают в себя поиск ассоциаций по набору генов (GREAT), а также по геномным позициям (ChIP-Atlas). Хотелось бы реализовать ещё один подход – построить зависимости используя Association Rule Mining, а затем визуализировать результаты с помощью диаграмм Исикавы. Проверить работу нового сервиса можно будет на известных статьях о взаимосвязи различных модификаций (изменение в гистонах, метилировании) и областях посадки транскрипционных факторов, а затем и попробовать поискать какие-то новые закономерности.<br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/lukashina_likhoketova_250519.pdf" target="_blank" style="">Слайды</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec97588007" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Noisy peak calling </strong>| JetBrains Research<br /><br /><span style="font-size: 20px;"><strong>студенты:</strong> Чаплыгина Дарья<br /><strong><strong data-redactor-tag="strong">руководители</strong>:</strong> Шпынов Олег</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Провести сравнительный анализ инструментов ChIP-Seq Peak Calling на устойчивость к соотношению сигнал-шум в экспериментальных данных.<br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/chaplygina_25052019.pdf" target="_blank" style="">Слайды</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec97588079" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Rescue failures </strong>| JetBrains Research<br /><br /><span style="font-size: 20px;"><strong>студенты:</strong> Балашова Дарья<br /><strong><strong data-redactor-tag="strong">руководители</strong>:</strong> Шпынов Олег</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Задача заключается в улучшении качества эксперимента Ultra Low Input ChIP-Seq (новый протокол, требующий 10 тысяч клеток вместо 2-5 миллионов) в случае нескольких доноров с помощью нейронной сети.<br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/balashova_25052019.pdf" target="_blank" style="">Слайды</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec98523045" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Улучшение в SPAN модели поиска пиков с помощью модели линейной регрессии </strong>| JetBrains Research<br /><br /><span style="font-size: 20px;"><strong>студенты:</strong> Картышева Елена<br /><strong><strong data-redactor-tag="strong">руководители</strong>:</strong> Алексей Диевский</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Задача проекта: улучшить SPAN, а именно добавить возможность дообучать модель, используя не только треки, но и какую-то информацию о данных, например: <br />• GC-content <br />• Mappability <br />• Local BG Estimate<br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/kartysheva_25052019.pdf" target="_blank" style="">Слайды</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec97588121" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Towards detection of differential RNA editing events in transcriptomics datasets. </strong>| St.Petersburg State University<br /><br /><span style="font-size: 20px;"><strong>студенты:</strong> Матвеенко Андрей<br /><strong><strong data-redactor-tag="strong">руководители</strong>:</strong> Anastasia Samsonova, Alexander Kanapin</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Throughout their lifetimes, RNA molecules undergo a variety of alterations that can result in beneficial or deleterious consequences for the organism. Modification of RNA nucleosides, known as RNA editing (RNAe), is a powerful instrument for the diversification of regulatory landscapes in eukaryotic genomes, promoting variability in the protein repertoire and silencing patterns, and fighting the genome instability mediated by mobile genetic elements. Abnormalities in the RNA editing process can be devastating for the whole organism, as they can provoke a wide spectrum of genetic disorders and cancers. Despite the relevance of RNA editing to human health, understanding of the process and its regulation is impeded by a lack of robust, straightforward experimental assays and computational methods for in-depth studies of the mechanism.<br /><br />Modern bioinformatics algorithms for RNAe analyses focus mostly on discovery of the editing sites from transcriptomics data. However, these tools are not designed for discovery of differential editing between various experimental conditions and tissues; i.e. for estimation of changes in editing landscape or editing efficiency either for a single nucleotides or for genes/transcripts.<br />In this project we aim to evaluate applicability of existing tools and statistical approaches for analyses of similar data modalities, such as bisulfite sequencing data, for discovery of differential RNAe events. The goal will be to propose a prototype of a statistical framework and and a blueprint of an algorithm for estimation of statistically significant changes in editing efficiency on a single nucleotides and/or gene level.<br /><br />This project is a continuation of a successful project undertaken by Irina Shchukina (Bioinformatics Inst., class of 2017) who developed a framework for discovery of RNA editing events in RNASeq data.<br /><br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/matveenko_25052019.pdf" target="_blank" style="">Слайды</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec69613579" class="r t-rec" style=" " data-record-type="215" ><a name="2017" style="font-size:0;"></a></div><div id="rec69562061" class="r t-rec t-rec_pt_60 t-rec_pb_60" style="padding-top:60px;padding-bottom:60px; " data-record-type="30" ><!-- T015 --><div class="t015"> <div class="t-container t-align_center"> <div class="t-col t-col_10 t-prefix_1"> <div class="t015__title t-title t-title_lg" field="title" style="font-size:40px;">Научные проекты 2017/2018</div> </div> </div></div></div><div id="rec69565318" class="r t-rec t-rec_pt_30 t-rec_pb_30" style="padding-top:30px;padding-bottom:30px; " data-record-type="397" ><!-- t397 --><div class="t397"> <div class="t-container"> <div class="t397__col t-width t-width_6"> <div class="t397__wrapper t-align_center"> <div class="t397__tab t397__tab_active t397__width_50" data-tab-rec-ids="41447892,41447935,41447953,41447995,41448007,41448026,41448068,41448073,41448090,41448185,41448198,41448245,41448254,41448273,41448282,41448285,41448293,41448302,41448306,41448374,41448377,41463854,69562162" style="border-bottom: 1px solid #eee;;"> <div class="t397__title t-name t-name_xs" style="font-weight:400;" field="title">Осень 2017</div> </div> <div class="t397__tab t397__width_50" data-tab-rec-ids="51524981,51526125,51526629,51526982,51539987,51527481,51547404,51538213,51538666,51539268,51796966,51797277,51798023,51799445,51798887,51529168" style="border-bottom: 1px solid #eee;;"> <div class="t397__title t-name t-name_xs" style="font-weight:400;" field="title2">Весна 2018</div> </div> </div> <div class="t397__wrapper_mobile"> <div class="t397__firefoxfix"></div> <select class="t397__select t-name"> <option value="41447892,41447935,41447953,41447995,41448007,41448026,41448068,41448073,41448090,41448185,41448198,41448245,41448254,41448273,41448282,41448285,41448293,41448302,41448306,41448374,41448377,41463854,69562162">Осень 2017</option> <option value="51524981,51526125,51526629,51526982,51539987,51527481,51547404,51538213,51538666,51539268,51796966,51797277,51798023,51799445,51798887,51529168">Весна 2018</option> </select> </div> </div> </div></div><style type="text/css"> #rec69565318 .t397__tab_active { border-bottom-color: #222 !important; } #rec69565318 .t397__tab_active .t397__title { color: #222 !important; font-weight: 600 !important; } #rec69565318 .t397__select { border: 1px solid #222; color: #222; } #rec69565318 .t397__wrapper_mobile:after { border-color: #222 transparent transparent transparent; } #rec69565318 .t397__firefoxfix { top: 1px; bottom: 1px; right: 1px; } </style><script type="text/javascript"> $(document).ready(function() { t397_init('69565318'); }); </script></div><div id="rec69562162" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Phylogenetic networks comparison</strong> | ITMO University<br /><br /><span style="font-size: 20px;"><strong>students:</strong> Anton Eliseev, Natalia Klimenko, Elena Pazhenkova<br /><strong>scientific adviser:</strong> Nikita Alekseev</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Phylogenetic networks are used to visualize evolutionary relationships that reflect any reticulations (such as hybridization). The amount of reticulation edges is a widely used criterion of networks, however, this measure is often identical in different topologies. We propose to use the number of possible convex colorings as metrics to distinguish networks with equal number of hybridizations. The number of convex colorings shows how many homoplasy-free characters are possible within this phylogeny. <br /><br /> Six species of Heliconius butterflies was chosen as model group to test our algorithm. A peculiar trait of genus Heliconius is the prevalence of interspecific hybridization, which reflects on phylogenetic networks as reticulation events. As suggested earlier, H. heurippa and H. elevatus have resulted from hybrid speciation [1, 2]. We analyze 20 nuclear genes, obtain NJ trees for each gene, compare these trees using pairwise Branch Score Distances, concatenate genes providing the most similar trees (distances up to 0.015), calculate hybridization networks and estimate numbers of convex colorings for each network. The network with the largest count of convex colorings is congruent with the hypothesis of hybrid origin of heurippa and elevatus species. <br /><br /> Another part of the study concerned the phylogeny of potatoes. 420 potato plants classified earlier as 29 species (7 cultivated and 22 wild) were analyzed by 15 plastid SSR-markers. As genomes were plastid, no hybridization was observed. We concentrated on building the most accurate phylogenetic tree for this data. Dendrograms were based on the Manhattan distance matrix. Cultivated and wild species of potato are clearly distinguishable. The idea of dividing Solanum tuberosum into Andigenum group and Chilotanum groups (according to [3]) is correct. Results of molecular analysis don't correspond to classification based on morphological features. <br /><br /> References: <br /> 1.Kronforst M.R., Papa R. The Functional Basis of Wing Patterning in Heliconius Butterflies: The Molecules Behind Mimicry. GENETICS. 2015. 200(1): 1-19 <br />2.Salazar C., Baxter S.W., Pardo-Diaz C., Wu G., Surridge A., Linares M., Bermingham E., Jiggins C.D. Genetic Evidence for Hybrid Trait Speciation in Heliconius Butterflies. PLoS Genet. 2010. 6(4): e1000930. <br /> 3.Spooner D.M., Ghislain M., Simon R., Jansky S.H., Gavrilenko T. Systematics, Diversity, Genetics, and Evolution of Wild and Cultivated Potatoes. Bot. Rev. 2014. 80: 283–383 <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/eliseev_klimenko_pazhenkova_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec41447935" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Optimization of spectral network parameters</strong> | EPAM Lifescience<br /><br /><strong></strong><span style="font-size: 20px;"><strong>students:</strong> Evgenia Fedotova , Rostislav Skitchenko , Ksenia Cherenkova <br /><strong><strong data-redactor-tag="strong">scientific adviser:</strong></strong> Gennadiy Zacharov</span></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">A pipeline for exome and target-sequencing analyses was developed. It's results could be used by physicians for diagnosis refinement. Such problems as pipeline deployment, it's utilities versions and dependencies control was solved by using Docker software. Pipeline quality control was obtained for NA12848 GIAB sample: Precision 0.95 and Sensitivity 0.78. <br /><br /> We've analyzed sequence results of cardyo-panel for families, whose members had diagnosis cardiomyopathy. Dependencies between variations and clinical diagnosis cardiomyopathy was found. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/skitchenko_cherenkova_fedotova_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec41447953" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Construction of RNA fragment database</strong> | University of North Carolina at Chapel Hill<br /><span style="font-size: 20px;"><br /><strong>student:</strong> Alexandr Ilin<br /><strong></strong><strong>scientific advisers:</strong> J. Wang, N. Dokholyan<br /></span></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">RNA plays significant role in regulation of gene expression at transcriptional and translational levels. This is achieved because of appropriate spatial structure of RNA molecule (i.e. motif), which is obtained after folding. Ability to predict 3-dimensional structure given sequence of RNA oligonucleotide is very important due to possibility to make use of this information in construction molecules with predefined structure – thus with known properties and targets to interact. Therefore, it supports design of new RNAs, which can be used as medications against wide spectrum of diseases caused by consequences of problems with gene product abundance. <br /><br /> In this work we developed RNA secondary structure decomposition algorithm to decompose an integrated RNA into many motifs. According to the RNA secondary structure decomposition algorithm, an RNA 3D motifs database was built by decomposing all the RNA 3D structures downloaded from PDB. We devised an algorithm to analyze and compare the base interactions networks between different RNA 3D motifs. We classified and clustered all the RNA 3D motifs in the database by using the network comparison algorithm. We utilized the supervised machine learning method to learn the relationship between sequences and base interactions networks of clustered RNA 3D motifs. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/ilin_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec41447995" class="r t-rec t-rec_pt_0 t-rec_pb_0" style="padding-top:0px;padding-bottom:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Assembly of mammalian genomes using GemCode data </strong>| Center for Algorithmic Biotechnology, St. Petersburg State University<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Angira Kekteeva<br /><strong></strong><strong>scientific advisers:</strong> Ivan Tolstoganov, Anton Bankevich</span></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">GemCode technology that was developed by 10X Genomics Company is actively used for assembly of mammalian genomes. CloudSPAdes is a genome assembly algorithm which was designed for metagenome assembly. However, algorithms in this tool, that were developed for resolving repetitions in the assembly graph, can be successfully used for assembly of mammalian genomes. <br /><br /> In this work we've examined exisiting metagenome assembly algorithms and analysed the disadvatages of using them for large genomes. Our analysis has shown that the average number of close edges in a human genome graph is more, than in metagenomes assembly graph, so it requires additional methods for sequencing long edges in the genomes of mammals. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/kekteeva_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec41448007" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Searching for molecular markers of chromosome bands</strong> | Bioinformatics Institute<br /><br /><span style="font-size: 20px;"><strong></strong><strong>student:</strong> Alexandra Klimina<br /><strong>scientific advisers: </strong>Yury Barbitoff</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Giemsa staining produces specific bands on metaphase chromosomes that have coloring of different intensity (G-bands). There are known correlations between the intensity of coloring and the degree of chromatin condensation, GC-content, and replication time. However, little is known about molecular markers of such banding pattern. <br /><br /> Main purpose of this project was to develop a tool for analysis of genome-wide correlation between different genomic features. We implemented the tool in Java with possibility to work on Spark-cluster for distributed computitions. <br /><br /> We chose previously described Projection test and Jaccard test to analyze the dependence between the reference (e.g., chromosome bands) and query feature of interest. We estimated the significance of correlation by sampling 1000 sets of randomly distributed intervals of the same length as the query feature followed by Kolmogorov-Smirnov normality test and one-sample t-test to obtain the p-value of association. <br /><br /> We tested our tool by analyzing correlation between chromosome banding pattern and such features as CpG-islands, microsatellite repeats and DNAse hypersensitivity regions. Expectedly, we showed that G-positive bands are positively correlated with microsattelite repeats, and negatively - with CpG-islands and open chromatin, DNAse hypersensitive regions. Thus, our tool can be used to further analyze genome-wide correlations between banding pattern and diverse molecular features. <br /><br /> <a href="http://bioinformaticsinstitute.ru/sites/default/files/klimina_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec41448026" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-size: 20px;"><strong></strong></span><span style="font-weight: 100;"><strong>Association of methylation level CpG-islands and IQ-level</strong> | University of Houston<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Daria Krytskaya<br /><strong></strong><strong>scientific adviser: </strong>Olga Naumova</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Methylation is an epigenomics modification of DNA. It change the activity of a DNA segment without changing the sequence. The most methylated region is region riched guanine and cytosine called CpG-islands. <br /><br /> In this work, we evaluate the methylation level of all known 26640 CpG-islands as average value of this region. Than we made a correlation test with using the Benjamini-Hochberg procedure for decreases the false discovery rate. <br /><br /> Analysis of biological role of this region was make with UCSC Genome Browser on Human Feb. 2009 (GRCh37/hg19) Assembly and GeneCards. <br /> Among our results are predicted transcription and translation region and known protein. For example, on 1 chromosome the most significant region is (38059428, 38063740) contained: prediction region are ENST00000373062, ENST00000463351, ENST00000488496 and gene of known proteins GLN2 – Homo sapiens guanine nucleotide binding protein-like 2 (nucleolar). For region (54951893, 54957287) of 1 chromosome are not predictions. For 2 chromosome the most significant region is (65213598, 65219212) contained gene of known proteins SLC1A4 Gene – Solute Carrier Family 1 Member 4. It is a transporter of alanine, serine, cysteine, threonine. Predicts a transport of a glucose by this protein. Disorders associated with this protein are spastic tetraplegia, thin corpus callosum, progressive microcephaly, microcephaly. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/krytskaya_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec41448068" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Genome structure of Mycobacterium tuberculosis strains in different world regions </strong>|<br />Theodosius Dobzhansky Center for Genome Bioinformatics, St. Petersburg State University<br /><br /><span style="font-size: 20px;"><strong>students:</strong> Vladimir Klimov, Vladimir Molchanov<br /><strong></strong><strong>scientific adviser:</strong> Ekaterina Chernyaeva</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr">Due to the high epidemiological rate of Mycobacterium tuberculosis and its constantly updated genomic data the problem of genome data analysis and systematization becomes extremely significant. For this reason our project was devoted to extend Genome-based Mycobacterium tuberculosis Variation (GMTV) Database which was developed by the researchers of Theodosius Dobzhansky Center for Genome Bioinformatics (Chernayeva et al., 2014). <br /><br /> In this study we performed an analysis of 999 M. tuberculosis strains which was isolated from patients in Malawi Republic. To achieve this results we designed a pipeline aimed at single nucleotide polymorphisms (SNPs) and insertions/deletions (InDels) identification from M. tuberculosis whole genome sequencing data. <br /><br />This pipeline based on BWA-mem and GATK programs which are widely used in such kind of investigations. Considering big amount of NGS data we suggested simple and rapid method to visualize or estimate quality control results performed by FastQC program using python3 regular expression and plotting in R. All variant calls (.vcf files) was uploaded on database, in future this data could be used for clade-specific annotation, which gives a possibility to identify strains without NGS methods. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/klimov_molchanov.pdf" style="" target="_blank">Slides</a></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec41448073" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Comparative analysis of natural selection effects across human populations</strong> | <br />Bioinformatics Institute<br /><span style="font-size: 20px;"><br /><strong>student:</strong> Julia Kornienko<br /><strong></strong><strong>scientific adviser:</strong> Yury Barbitoff</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">In this project we aimed to estimate the natural selection effects across human populations based on the Genome Aggregation Database (gnomAD) dataset which contains information about sequence variants in 123136 human exomes and 15,496 genomes. To this end we calculated the amount of protein truncating variants (PTV) both (i) per individual genes (based on GENCODE v.19); and (ii) per gene sets (hallmarks and canonical pathways obtained from the MSigDB Collections) for six different populations (European, South Asian, Latino American, East Asian, African and Finnish). <br /><br /> We estimated the selective coefficients of heterozygous PTVs for different human populations from the constructed dataset in the same way as it was done by Cassa et al. (2017) and found that distribution of selective coefficients both per individual genes and gene sets is dependent on the population size. Taking this into account, we evaluated the difference in distribution of PTV allele counts among the populations and found that for 2040 of 12367 analyzed genes and for 746 of 1379 of analyzed gene sets selective effects were significantly population-dependent. Thus it is possible to conclude that selective effects for some genes do vary across the populations. <br /><br /> Interestingly, we discovered significant enrichment of PTV alleles in the immune system-related pathways (IL-10, IL-13 and IFNG signaling) in the individuals of South Asian ancestry (SAS), with more than half of all PTVs discovered in the corresponding genes belonging to the SAS population. These results are concordant with some previous findings and emphasize the natural heterogeneity of selective effects. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/kornienko_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec41448090" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>De novo cdr3 annotation in VDJ rna sequences</strong> | Center for Algorithmic Biotechnology, St. Petersburg State University<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Kristina Krivonosova<br /><strong></strong><strong>scientific advisers:</strong> Andrey Slabodkin, Maria Chernigovskaya</span></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"> The project relates to the construction and analysis of the repertoire of antibodies. In order to build a repertoire, we look for mutations in antibodies with the help of a germline by aligning the variable part of the immunoglobulin gene with a special base of V-, D-, and J-genes (germline). With this alignment, we can annotate the sequence: we mark the boundaries of the V-gene and the J-gene, as well as the boundaries of the three regions that determine the specificity of the antibody to the antigen (CDRs). In practice the third region (CDR3) is the most variable part of the immunoglobulin gene so its borders are of the greatest interest. Unfortunately, on some data (for instance, in case of lymphoma) the level of mutations goes off the scale and we can not build an alignment on the germline for that data. <br /><br /> In this work we develop a heuristic for VDJ sequence annotation that does not use alignments. This new heuristic is based on searching conserved regions in the source sequence to identify CDR3 regions. In practice this approach produces satisfying results with accuracy rate of 95% when applied to verified data sets. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/krivonosova_final_16122017.pdf" style="font-size: 18px;" target="_blank">Slides</a><br /></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec41448185" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Forming a panel of markers for the molecular-genetic diagnosis of congenital<br />metabolic disorders </strong>| Parseq Lab<br /><br /><span style="font-size: 20px;"><strong></strong><strong>student:</strong> Ekaterina Nebozhatko<br /><strong></strong><strong>scientific advisers:</strong> Tamara Simakova, Anton Bragin</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Predicting the deleterious effects of mutation on protein function is one of the main tasks of genetics. Often researchers use predictive tools for this. The main problem with the use of predictive tools is not enough high sensitivity and specificity of classifiers. On average, the sensitivity is 80%, which means that 20% of the possible pathogenic mutations may go unnoticed. This can adversely affect the success of treatment. Another approach is to use open databases in which information on pathogenic mutations for certain genes is collected and verified. <br /><br />The company Parseq Lab is working on a large project to create a panel of markers. It included 37 genes associated with 35 different diseases and 36 external sources. In this paper, a part of the project with single database PNDdb and three genes GCH1, QDPR and PTS is presented. In the course of the work, a tool was implemented that exports the necessary data from the site and presents them in the VCF format. The sensitivity of predictive tools of SIFT and PolyPhen was also assessed. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/nebozhatko_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec41448198" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Khazars heritage in the world genomes</strong> | University of La Verne<br /><span style="font-size: 20px;"><br /><strong>student:</strong> Yury Orlov<br /><strong></strong><strong>scientific adviser:</strong> Tatiana Tatarinova</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr">Khazars are a semi-nomadic ethnic group that lived in the second part of the 1 st millennium, occupying a large area north of Caucasus between Black and <br /> Caspian seas. At the end of the 10 th century their state was destroyed and the Khazar Khanate disappeared as suddenly as it rose, without leaving any legacy <br /> except their own funerary mounds. At all times there were pretty much theories and guesses about the Khazars origin and their descendants but it was impossible to make solid conclusions about them. The aim of this project is to find the answers on the questions of Khazars origin and genetic legacy by <br /> analysing ancient DNA (aDNA) extracted from remains of three Khazar representatives. <br /><br /> In course of the project, aDNA sequencing data was processed according to its specific library preparation and degraded nature of aDNA. Reads were <br /> mapped to the HG38 reference genome. It was found that bacterial contamination was more than 75% (typical to aDNA), and by detecting significant amount of C-T transitions on the read ends it was shown that studied DNA is indeed of ancient origin. Using the GATK package we performed the SNP-calling procedure on obtained data and with Admixture tool figured out that the Khazars are a mix of North East Asians, Northern European, Mediterranean and South West Asian populations, as it is expected of a well-mixed group of semi-nomadic people. <br /><br /> As continuation of the work we are collecting different ancient and modern genomes to compare obtained data to them and finally draw a conclusion about the Khazars origin and their descendants in the modern world. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/orlov_final_16122017.pdf" style="" target="_blank">Slides</a></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec41448245" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Russian Exomes. Part 1</strong>. |Bioinformatics Institute<br /><br /><span style="font-size: 20px;"><strong></strong><strong>students:</strong> Olga Poleshchuk, Ekaterina Izmailova<br /><strong></strong><strong>scientific adviser:</strong> Yury Barbitoff</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Mutations in protein-coding part of the genome are a cause of numerous different pathologies. Thus, whole exome sequencing (WES) is a commonly used alternative to whole-genome sequencing in medical genetics and health-related studies. Some environmental adaptations also likely arose from changes in protein-coding regions, making exome sequencing a valuable tool for population genetics studies. Several large sequencing consortia (e.g., Exome Aggregation Consostium (ExAC)) have collected data from hundreds of thousands samples of western population, and a lot of research was done using this data. The goal of our project was to develop a pipeline for variant analysis in Russian population, and apply it to ~570 WES samples. <br /><br /> Firstly we created pipeline using Snakemake as one of possible tools for creating workflows. The pipeline receives raw FASTQ files as input and outputs a combined annotated VCF for all samples in a batch. We successfully tested our pipeline, and performed preliminary analysis of variants in a dataset of 570 samples of Russian and CIS ancestry. We observed many novel variants common to samples included in the study, with most of such variants classified as missense mutations, intronic variants, and synonimous substitutions. Thus, we made a very first and preliminary steps in assessing the exome-wide genetic structure of Russian population. Further data aggregation and analysis will help to completely fill the biggest gap on the genetic map of the world. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/poleschuk_izmaylova_161217.pdf" style="" target="_blank">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec41448254" class="r t-rec t-rec_pb_0" style="padding-bottom:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Speed-efficient data structures for cloudSPAdes</strong> |Center for Algorithmic Biotechnology, St. Petersburg State University<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Evgen Polevikov<br /><strong></strong><strong>scientific advisers:</strong> Anton Bankevich, Ivan Tolstoganov </span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">GemCode technology that was recently introduced by 10X Genomics company is rapidly becoming essential for variant calling, diploid genome assembly and read alignment. The cloudSPAdes algorithm was recently developed in the Center for Algorithmic Biotechnology. The algorithm uses GemCode data to improve metagenome assembly quality. Currently, cloudSPAdes consumes a large amount of computational resources for assembly of complex metagenomes. CloudSPAdes' assembly procedure consists of several stages: on the first stage it constructs assembly graph using procedures which were implemented in already existing metaSPAdes pipeline. Then barcoded reads are aligned to the edges of the graph such that for every edge we get a particular set of barcodes. Every set is represented as a sorted array. Intersection of these sets is computed in order to estimate genome distance between long edges and determine their true ordering. <br /><br /> In this work we adapted probabilistic data structure called containment min hash that allows to improve current procedure of computing of edge intersection. In order to estimate intersection of two sets of barcodes A and B (assume that size of A less than size of B) we first create a bloom filter from the set of a larger size B. Then we take a random sample S from the set of a smaller size A and test every element of S for membership in B using a bloom filter. By that we estimate an intersection of A and B. <br /><br /> In order to benchmark containment min hash against the original sorted array data structure we constructed assembly graph from GemCode library which was sequenced from a mixture of 5 known bacterial species. We selected a set of 1492 edges longer than 5,000 bp from the assembly graph and found an intersection for every ordered pair of these edges using containment min hash and compared it with initial edge intersecting procedure. Our analysis have shown that new algorithm works approximately 6 times faster. Also we have managed to decrease memory consumption: now it is enough to store about 60% of data that initial procedure uses. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/polevikov_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec41448273" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Protein digestion patterns</strong>|University of North Carolina at Chapel Hill<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Natalia Rodina<br /><strong></strong><strong>scientific advisers:</strong> Popov Konstantin, Dokholyan Nikolay </span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"> <span style="font-size: 18px;">Digestion of the proteins by proteasomes and proteases in cells results in producing a specific repertoire of peptides that can potentially bind to MHC 1 complex 1 and used for triggering immune response against specific cancer cells. Thus, creating a tool for prediction the "peptide profiles" produced by protease cleavage in different types of tissues in normal cells and primary tumor became the aim of the present project. <br /> In the first step, microarray expression data in 19 types of tissues types for normal cells (726 arrays) and primary tumor (1.460 arrays) was collected from the MERAV database. PCA analyzes of the preprocessed expression data showed differences in normal tissue and primary tumor for every tissue type were shown. For every tissue type, expressed genes were selected (values higher then median of quintile normalized data) and only genes expressed in the primary tumor and not presented in the normal tissues were taken into account. For selected genes, the amino acid sequences were parsed from the NCBI protein database. <br /> Information about cleavage sites of human cell proteases was downloaded from the Merops database and for every type of tissue only expressed proteases were selected. <br /> In the next step, a tool for prediction of the peptide profiles was created. As input, the tool takes a selected type of the tissue. Then, from inbuilt database, amino acid sequences of all genes and information about cleavage sites of all proteases expressed in the selected tissue are taken. The tool finds all possible cleavage sites in every protein for every protease and provides information about all peptides created by the cleavage of all proteins in the tissue. <br /> The created tool will provide the possibility to predict peptide profiles for all tissue types and identify peptides that are specific for the cancer cells and can be used for targeted immune therapy. </span><br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/rodina_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec41448282" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Ti plasmid evolution and horizontal gene transfer</strong> | St. Petersburg State University<br /><br /><span style="font-size: 20px;"><strong></strong><strong>students:</strong> Shikov Anton , Zorin Evgeniy<br /><strong></strong><strong>scientific advisers:</strong> Alexandr Tkachenko, Mikhail Rayko</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Agrobacterium species contain special sequence named T-DNA in Ti- and Ri-plasmids which can be inserted into plant genome. This feature is widely used in plant biotechnology. However, this insertion can become a stable part of plant genome, thus, Agrobacterium species are able to implement horizontal gene transfer to plant organism that happens quite rare in plant realm. <br /><br />The aim of our work was to detect of new examples of horizontal gene transfer in plants and reconstructing phylogeny and evolution of Ti- and Ri-plasmids. To achieve this goal, we used hmmer tools and analyzed available plant genomes and proteomes. In total, 66 proteomes and 45 genomes were scanned. Extracted hits were further utilized for making multiple alignments and building approximately 700 trees. <br /><br />Unfortunately, we didn't detect any explicit clusterization of plant and bacterial sequences. Nevertheless, during analysis we successfully revealed a brand-new example of horizontal gene transfer in Nicotiana tabacum that has not been described in literature before. Bacterial protein riORF20 from Agrobacterium rhizogenes has two plant homologs. Interestingly, this two proteins are homologous to C- and N-ends of riORF20 respectively. For this reason, we propose DNA recombination in N. tabacum after T-DNA insertion. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/shikov_zorin_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec41448285" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Enhancement of Export Option for a Genome Mappability Score Estimator</strong> | Bioinformatics Institute<br /><br /><span style="font-size: 20px;"><strong></strong><strong>student:</strong> Skalon Elizaveta<br /><strong></strong><strong>scientific adviser:</strong> Bakin Evgeniy</span></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Mappability is a genome-wide function that indicates whether it is possible for any read to be unambiguously mapped to a given position. Mappability information can be crucial for an interpretation of such experiments as ChIPSeq, SNP-calling etc., where quantitative estimates or confident identification of variations are performed. There is a special metric called Genome Mappability Score (GMS), which quantifies the mappability. GMS measures a weighted probability of mapping certainty in a given place. If the GMS is zero in a given position, many identical reads from different loci may be equally mapped to this region. Otherwise, if the GMS is 100, a read mapped to this position is unique. <br /><br /> In this work, we extended the functionality of fast and sufficiently accurate instrument for the GMS calculation, developed by in Bionformatics Institute in 2016. Firstly, a possibility to get output records in many various formats was provided. It allowed not only Wig and BigWig, but also BED, BigBed and TDF output formats to be supported. Secondly, the runtime of data export was reduced by an implementation of a multiprocessing mode, geometric expansion of arrays and export of GMS track directly to BigWig without wigToBigWig converter. <br /> These improvements made the GMS computation even more convenient and friendly for its users. <br /> <br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/skalon_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec41448293" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><span data-redactor-style="font-size: 24px;" style="font-size: 24px;"><strong data-redactor-tag="strong">Retention time for identification of natural products</strong> | Carnegie Mellon University</span><br /><br /><span style="font-size: 20px;"><strong>student:</strong> Vladimir Sukhov<br /><strong></strong><strong>scientific adviser:</strong> Alexey Gurevich, Husein Mohimani</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"> Natural Products (NPs) play an important role in pharmacology: many antibiotics, antiviral and antitumor agents are NPs. Thus, it is crucial to have methods for accurate discovery of new NP. In the process of searching for new NP, false positive identifications may occur. To reduce their number, we use retention time (RT) as an additional correctness check for discovered NP. <br /><br /> In this work, we applied machine learning methods for determining possible RT range for peptides. The multiple regression method was chosen as the primary technique. As a model for machine learning, we considered the amino acid composition of a peptide, where each amino acid adds its own weight to the final RT value. <br /><br /> As the result, the model was trained and tested. Model benchmarking demonstrated high accuracy of RT prediction and its potential for a significant reduction of false positive identifications. <br /><br />Slides</div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec41448302" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Analysis of VH-replacement statistical properties based on public datasets</strong> | Center for Algorithmic Biotechnology, St. Petersburg State University and Pavlov First St. Petersburg State Medical University<br /><br /><span style="font-size: 20px;"><strong>students:</strong> Adel Gazizova, Anastasia Vinogradova <br /><strong></strong><strong>scientific adviser:</strong> Andrey Slabodkin, Maria Chernigovskaya, Oksana Stanevich</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">During a construction, immunoglobulin H locus (IgH) undergo a process named VDJ-recombination, during which is random gene segments from IgH germline are set into resulting gene sequence. It provides a primary specificity to antigens. However, infrequently, the existing V-gene can be partly replaced by a new one, and this process is called VH-replacement. There are various hypotheses regarding the contribution that VH-replacement makes to antibody functionality. <br /><br /> In our work, we created a pipeline, which allows to identify VH-replacement in human antibody sequences. First we downloaded the data from Genbank, parsed files and extracted titles, that contained all the information about each sequence. Before starting the search, we divided antibody sequences into clonal families, because our data must contain only clonal-independent sequences in order to exclude a false-positive result. Then by means of developed script we made an exact and inexact (with one possible mismatch) search of VH-replacement's footprints in sequences of people with different phenotypes. We analyzed results and found, that VH-replacement frequency significantly increases for subjects infected with HIV-1, as well as for ones vaccinated against pneumococcus. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/vinogradova_gazizova_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec41448306" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Web bot development for automation of requests to IMGT / V-quest</strong> | Bioinformatics Institute<br /><br /><span style="font-size: 20px;"><strong></strong><strong>students:</strong> Andrey Zolotarev, Alexandr Cheblokov <br /><strong></strong><strong>scientific adviser:</strong> Evgeniy Bakin</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">There are a lot of different web-services that give the user an opportunity to work with integrated databases and research tools, which are necessary for a number of scientific areas. <br /><br /> One of them is IMGT® (the international ImMunoGeneTics information system®) – high-quality knowledge resource in immunogenetics and <br /> immunoinformatics that specifically provides data about immunoglobulin or antibodies, T-cell receptors, major histocompatibility (MH) of human and other vertebrate species, immunoglobulin superfamily, MH superfamily and related proteins of the immune system of vertebrates and invertebrates. Unfortunately, this service is difficult to use for implementation statistical analysis due to limitation of loadable <br /> sequences. IMGT allows the user to load only 50 sequences by one request, and the task become further complicated by the need to configure multiple query parameters. <br /><br /> The problem for statistical research is obvious, the scientist must spend a huge amount of time to process even one thousand sequence dataset. <br /><br /> The solution we developed is web-bot that allows the researcher to automate the processing of large amounts of data subject to the limitation given above. We came to the conclusion that a suitable basis for our objective is Selenium Web-Driver. <br /> Selenium is a software library with the open source code, which is widely represented for a number of the most popular web browsers and compatible with such popular programming languages as C#, Python, JavaScript and others. This module emulates user behavior on the site, what allows to set parameters once and then implement the repetition of their setting by Selenium API. <br /> As a result of our work we present the program that automates requests to IMGT for huge datasets and includes an interface for configuring the search parameters that are specific to a particular task. Result of program execution is a table in CSV-format that contains data required for the researcher. <br /> <br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/cheblokov_zolotarev_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec41448374" class="r t-rec t-rec_pb_0" style="padding-bottom:0px; " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Using approximate calculations to speed up the peak calling </strong><strong>procedure </strong>|Bioinformatics institute<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Viacheslav Borovitskiy<br /><strong></strong><strong>scientific adviser:</strong> Evgeniy Bakin</span></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Peak calling is a computational procedure used to identify areas in a genome that have been enriched with aligned reads primarily as a consequence of performing a ChIP-sequencing experiment. There are several popular pieces of software which perform this procedure (most of them require substantial computational resources and time). Each one has its own set of parameters requiring adjustment to the particular experiment. <br /><br /> In this work, we try to address the issue of time costs of the process of parameters adjustment for the peak calling procedure. We present a prototype of a tool that uses some fast machine learning / digital signal processing methods to approximately obtain the result of a peak calling procedure for a given caller with a given set of parameters in a matter of no time. <br /><br /> At first, we use given caller with a given set of parameters on a small piece of data. We then use the results of the previous step to train a linear <br /> classifier (some fast time-series optimized version of logistic regression). <br /><br /> Finally, we apply our trained classifier (followed by some threshold transformation) to the rest of the data to obtain an approximation of the result. <br /><br /> We test our tool against some data sets from the Encyclopedia of DNA Elements (ENCODE). On "good" data we have precision/recall scores at about 0.85/0.85. On "bad" data we have precision/recall scores at about 0.20/0.20. Tests give impression that we never overfit, meaning that precision/recall scores on the train set determine those on the test set. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/borovitskiy_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec41448377" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Regulatory network modeling based on analysis of ATAC-seq data from cancer cells</strong> | Institut Cochin, Laboratory "Computational Epigenetics of Cancer"<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Anastasia Danchurova<br /><strong>scientific adviser:</strong> Valentina Boeva</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">The tumor cell state is governed by complicated interplay between transcription factors that regulate gene expression and thus define cell fate. The concept of core, or master, transcription factors comprising Oct4, Sox2, Nanog (also known as Yamanaka's factor family) postulates that small number of transcription factors control the more numerous auxiliary transcription factors and play an essential role in determining of cell fate. Recent data showed that these core transcription factors play a regulative role in different types of cancer. <br /><br /> Because cancer is a disease associated with aberrant gene expression patterns, transcription factors, which serve as the convergence points of oncogenic signaling and are functionally altered in many cancers, hold great therapeutic promise. The more personal this therapy will be the more efficient result it will achieve.<br /> <br /> That is why in this project exactly ATAC-seq data is used. Related to DNAse-seq and MNAse-seq methods, ATAC-seq compares favorably in library preparation simplicity, speed and amount of required cells (500-50 000 cells), what in total makes it the appropriate for clinical usage. <br /><br /> In this project, we create a tool, which combines ATAC-seq data with human genome annotation and several databases, determines interactions between transcription factors and active promoters and enhancers. As a result, we are expecting to construct a graph that will represent all detected interactions. Analysis of such graph is intended to help to determine the main transcription factors that may become effective potential targets for anti-cancer therapy. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/danchurova_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec41463854" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Regulatory network modeling based on analysis of ATAC-seq data from cancer cells</strong> |<br /> All-Russia Institute for Agricultural Microbiology<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Yury Malovichko<br /><strong>scientific adviser:</strong> Evgeniy Andronov</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Sinorhizobium meliloti is one of the so-called Rhizobia, a group of α- and β-proteobacteria known for their capability of interacting with legume plants that results in stable mutualistic symbiosis where bacteria provide plants with atmosphere nitrogen reduced to ammonium in exchange for organic carbon. The genome of Rhizobia differs from that of Escherichia coli and other model prokaryotes and comprises of one major chromosome and one or more symbiotic plasmids that determine bacterium's host range and symbiosis efficacy. However, Rhizobia genome is also known for its flexibility, with symbiotic genes rearranged with plasmids, between them or even between plasmids and chromosome. <br /><br /> In this study, we aimed to prove a suggestion based on RFLP and other molecular marker analyses that two distinct genetic lines exist with S. meliloti species discriminated by linkage of particular alleles of leu and betCB genes. We used MLST approach with 10 loci suggested previously for genomic clustering of this species (see Reference) ad Bayesian Inference algorithm to build a tree that would show actual phylogeny of 12 isolates with 6 isolates for both supposed genomic lines, respectively. However, we gained ambiguous results showing that suggested loci are evidently not universal in their use for MLST of S. meliloti. For now, we seek for more informative loci that will shed the light on true phylogeny of <br /> these isolates and existence of these two genomic lines. <br /><br /> 1. Berkum P. Van, Elia P., Eardly B.D. Multilocus sequence typing as an approach for population analysis <br /> of Medicago-nodulating rhizobia // J. Bacteriol. 2006. Т. 188. № 15. С. 5570–5577. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/malovichko_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec51524981" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>NGS-based metagenomic pathogen viruses and bacteria identification system</strong> |<br /> Saint-Petersburg Pasteur Institute<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Alexandr Bebyakov<br /><strong>scientific adviser:</strong> Alexandr Semenov</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Большинство методов микробиологической диагностики занимают продолжительное время и неприменимы для обнаружения некультивируемых форм патогенных агентов. Предполагается возможным на основе данных секвенирования нуклеотидных последовательностей смешанного образца определять наличие возбудителей особо опасных инфекций и свойственных им факторов патогенности и, таким образом, ускорять процесс принятия решений о мерах противодействия возможным эпидемиям.<br /><br /><a href="https://github.com/ABapmath/2018_spring_SEM" style="" target="_blank">GitHub<br /></a><a href="http://bioinformaticsinstitute.ru/sites/default/files/bebyakov_final_26052018.pdf" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec51526125" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Applying state-of-the-art neural network architectures for predicting protein-binding sites</strong> |<br /> ITMO University<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Viacheslav Borovitskiy<br /><strong>scientific adviser:</strong> Tatiana Malygina</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">This is a project aiming to improve an approach proposed in <a href="https://www.ncbi.nlm.nih.gov/pubmed/28575181" style="nofollow">the paper</a> by using some of the modern neural network architectures.<br /><br />GitHub<br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/borovitskiy_26052018.pdf" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec51526629" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Study and development of a macrophage metabolic model</strong>|ITMO University<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Natalia Rodina, Alexandr Cheblokov<br /><strong>scientific adviser:</strong> Gainullina Anastasia, Sergushichev Alexey</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Macrophages are cells of the first line of immune protection: destroy pathogens (M1), maintain tissue homeostasis (M2). Using the metabolic FBA model allows you to see the coordination between metabolic pathways at the level of the whole cell. However, the existing FBA model of macrophage metabolism has a number of inaccuracies, and therefore it does not reflect the latest ideas about their M1 activation, formulated during molecular biological experiments. The purpose of this project is to detect and correct inaccuracies of the macrophage FBA model of metabolism.<br /><br /><a href="https://github.com/chebaleksandr/MACROPHAGE_BI_2018" style="" target="_blank">GitHub<br /></a><a href="http://bioinformaticsinstitute.ru/sites/default/files/cheblokov_rodina_26052018.pdf" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec51526982" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Biogeography of arabidopsis</strong>|University of La Verne<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Anton Eliseev, Kristina Krivonosova <br /><strong>scientific adviser:</strong> Tatiana Tatarinova</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Цель проекта - с помощью геномов медикаго и арабидопсиса определить корреляции между генетикой, климатом, почвой и прочей географией. Построить модель связывающую окружающую среду и геном.<br /><br /><a href="https://github.com/AntonCoon/biogeography" style="" target="_blank">GitHub<br /></a><a href="http://bioinformaticsinstitute.ru/sites/default/files/eliseev_krivonosova_26052018.pdf" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec51539987" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Statistical analysis of annotated genomes</strong>|University of La Verne<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Poleshchuk Olga, Danchurova Anastasia <br /><strong>scientific adviser:</strong> Tatiana Tatarinova</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"> Find correlation between sequence features and functional regions in different genomes <br /><br /> <ol> <li> Plot sequence features such as TFBS, SNPs, methylation, RNA-seq coverage </li> <li> Map it on promoter regions </li> <li> Find correlation </li> <li> Consider outcomes for promoter prediction for complex and not annotated genomes </li></ol><br /><a href="https://github.com/danchurova/Genomes_project" style="" target="_blank">GitHub</a><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/poleshchuk-danchurova-final-260518.pdf" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec51527481" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Finding novel variations of germline Immunoglobulin genes using WGS data</strong>|University of California San Diego<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Alexandr Ilin<br /><strong>scientific adviser:</strong> Yana Safonova</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Variety of immunoglobulin germline genes (V, D, and J) is a key component of the antibody repertoire diversity. Highly repetitive structure of Ig loci and a lack of natural selection result in elevated polymorphism rate of immunoglobulin germline genes. In this project, we want to analyze variations of Ig loci in several human populations and describe differences between them.<br /><br /><a href="https://github.com/Serfentum/bioinf_vsegments" style="" target="_blank">GitHub</a><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/ilin_final_26052018.pdf" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec51547404" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Long read mapping improvements for Flye assembler</strong>|University of California San Diego<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Evgeny Polevikov<br /><strong>scientific adviser:</strong> Mikhail Kolmogorov</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"> minimap2 is a versatile pairwise aligner for genomic and spliced nucleotide sequences written in C. The goal of this project is to write C++ wrapper for this tool in order to incorporate it into <a href="https://github.com/fenderglass/Flye">Flye</a>. <br /><br /> In <a href="https://github.com/EvgenyPolevikov/mm2wrapper/blob/master/src/example.cpp">src/example.cpp</a> you can find an usage example of minimap2 API with C++ interface. The example shows how to build an index and how to use this index to find overlaps for pacbio reads. <br /><br />We have recently released Flye assembler for long and noisy reads (PacBio, Oxford Nanopores). The <a href="https://www.biorxiv.org/content/early/2018/01/12/247148" target="_blank" style="">assembly results</a> seems to be very promising in comparison with the current state-of-the-art approaches.<br /><br />As a successor of the ABruijn assembler, Flye uses solid k-mer based approach to find overlaps between noisy reads, which is (relatively) fast, but might be not optimal in terms of memory usage and the parameter choice flexibility. On the other hand, <a href="https://github.com/lh3/minimap2" target="_blank" style="">minimap2</a> seems to be very memory efficient, while showing the best sensitivity/specificity among the other long read aligners. As minimap2 also has C++ API, we want to explore the possibility of replacing our solid k-mer approach with minimap2. We expect that this change will significantly reduce the memory usage bottleneck, while also improving the assembly accuracy.<br /><br /><a href="https://github.com/EvgenyPolevikov/mm2wrapper" style="" target="_blank">GitHub<br /></a><a href="http://bioinformaticsinstitute.ru/sites/default/files/polevikov_final_26052018.pdf" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec51538213" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Mediation of effects of persistent chemicals on the human sperm epigenome</strong>|A.N. Belozersky Research Institute of Physico-Chemical Biology, Moscow State University, Institute of Bioengineering, Research Center of Biotechnology RAS <br /><br /><span style="font-size: 20px;"><strong>student:</strong> Julia Kornienko<br /><strong>scientific adviser:</strong> Oleg Sergeev, Yulia Medvedeva</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"> Процессы сперматогенеза и созревания сперматозоидов включают в себя каскад эпигенетических изменений (Wu et al., 2015). Изучение эпигенома сперматозоидов представляет очень перспективное направление по нескольким причинам. Во-первых, явно недостаточно изучено специфическое воздействие разнообразных факторов среды, включая химические факторы, на эпигенетические маркеры. Во-вторых, выявляемые эпигенетические изменения связаны с качеством и количеством сперматозоидов. И, в-третьих, что может быть особенно важно - репрограммирование эпигенома половых клеток может быть передано следующему поколению, что может привести к нарушению развития потомства, как на этапе развития эмбриона, так и в последующей жизни.<br /><br />- Объект заявляемого молекулярно-эпидемиологического исследования – данные метилирования ДНК сперматозоидов (WGBS и RRBS) и данные факторов окружающей среды, полового развития и образа жизни, полученные в ходе родительского проспективного когортного исследования «Russian Children's Health. Male Reproductive SubStudy», начавшегося в 2003 году.<br /><br />- Что известно из родительского исследования?<br />Воздействие таких стойких токсикантов, как диоксины, в течение полового развития (пубертата) влияет как на метилирование ДНК сперматозоидов (Pilsner et al., 2018), так и на снижение качества семени (Minguez-Alarcon et al., 2017) в возрасте 18 лет.<br /><br />- Что неизвестно?<br />Какой вклад вносят другие факторы окружающий среды, потенциально негативно влияющие в процессе сперматогенеза на эпигеном сперматозоидов и качество семени, в частности курение.<br />Какой вклад в изменение эпигенома и качества сперматозоидов вносят различные варианты полового развития (ускоренное, нормальное, замедленное)?<br /><br />- Цель исследования:<br />Изучить роль курения и темпов полового созревания в качестве медиаторов влияния диоксинов на метилирование ДНК сперматозоидов.<br /><br /><a href="https://github.com/yukornienko/mediation_analysis" style="" target="_blank">GitHub<br /></a><a href="http://bioinformaticsinstitute.ru/sites/default/files/kornienko_final_26052018.pdf" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec51538666" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Role of protein dimerization</strong>|Department of Biochemistry and Biophysics, University of North Carolina at Chapel Hill <br /><br /><span style="font-size: 20px;"><strong>student:</strong> Orlov Iurii<br /><strong>scientific adviser:</strong> Nikolay Dokholyan</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"> We would like to understand why nature evolves proteins to function as dimers. To understand whether the oligomeric structure of protein is more evolutionary preferable than monomeric.<br /><br /> General plan: <br /> <ol> <li> detect core residues (CR) responsible for structure formation </li> <li> determine how number of CR grows with protein length </li> <li> compare obtained results with dimeric proteins </li> <li> go further for larger oligomers (n-mers) to find the most preferable n </li></ol><br /><a href="https://github.com/troublov/BI_project_spring18" style="" target="_blank">GitHub<br /></a><a href="http://bioinformaticsinstitute.ru/sites/default/files/orlov_31032018.pdf#overlay-context=node/1521" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec51539268" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Model of the N1 zone formation in human antibodies dimerization</strong>|Bioinformatics institute<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Elena Pazhenkova<br /><strong>scientific adviser:</strong> Evgeniy Bakin, Oksana Stanevich</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"> The N1-zone is a variable region of human antibodies DNA, formed as a result of VDJ-recombination and providing diversity of antigen binding regions. N1-zone generation is a complicated process including formation of palindroms on 5' and 3' ends and addind up to 20 random nucleotides to 5' and 3' ends with following non-homologous end joining. Thus, length of N1-zone depends of several random events. However, the N1-zone sometimes contains so-called footprints, appeared as a result of VH-replacement and recent studies showed that the length of CDR3 (including V3', N1, D, N2 and J5') is correlated with number of footprints (Meng et al., 2014). In this project we want to figure out whether VH-replacement is a random event by fitting of statistical model of N1-zone formation and estimating its parameters using Maximum Likelihood method.<br /><br /><a href="https://github.com/PazhenkovaEA/VH-replacement-analysis" style="" target="_blank">GitHub<br /></a><a href="http://bioinformaticsinstitute.ru/sites/default/files/pazhenkova_25052018.pdf" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec51796966" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Search for multiple associations in GWAS data</strong>|Bioinformatics institute<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Shikov Anton<br /><strong>scientific adviser:</strong> Yury Barbitoff</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"> В 2017 году Биобанк Великобритании сделал крупнейший релиз генетических данных в истории (500,000 человек). Группа Бенджамина Нила произвела быстрый массовый анализ ассоциаций (GWAS) с более чем 2,000 фенотипов, результаты данного анализа были выложены в открытый доступ. С тех пор, огромное количество препринтов, посвященных поиску интересного сигнала в этих данных, появились на сервере bioRxiv. Данный проект посвящен поиску маркеров, обладающих множественными ассоциированными фенотипами, и изучению механизмов, опосредствующих данные эффекты.<br /><br /><a href="https://github.com/anton-shikov/GWAS_project" style="" target="_blank">GitHub<br /></a><a href="http://bioinformaticsinstitute.ru/sites/default/files/shikov_26052018.pdf" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec51797277" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Identifying differentially expressed transposons across four life-cycle stages of Fasciola hepatica</strong>|Institute of Cytology RAS<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Elisaveta Scalon<br /><strong>scientific adviser:</strong> Anna Soloveva, Nikolay Panyushev</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"> Проект предполагает поиск мобильных элементов, чей уровень экспрессии варьирует в зависимости от стадии жизненного цикла Fasciola hepatica.<br />У F.hepatica секвенированы транскриптомы всех стадий жизненного цикла, данные доступны в Sequence read archive. Планируется собрать транскриптомы по референсу, выявить последовательности мобильных элементов<br />и определить их уровни транскрипции. На выходе будут получены данные о наличии или отсутствии мажорных транскриптов мобильных элементов, специфичных к отдельным стадиям жизненного цикла F.hepatica.<br /><br /><a href="https://github.com/NickPanyushev/IB_Fasciola" style="" target="_blank">GitHub<br /></a><a href="http://bioinformaticsinstitute.ru/sites/default/files/skalon_final_26052018.pdf" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec51798023" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"> <div class="t-container"> <div class="t-col t-col_12 "> <div class="t260__accordion" style="border-bottom: 1px solid #adadba;"> <div class="t260__wrapper"> <div class="t260__header" style="border-top: 1px solid #adadba;"> <div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Evolution analysis of genes associated with apomixis in Brassicaceae family</strong>|CAB SPbU<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Rostislav Skitchenko<br /><strong>scientific adviser:</strong> Mike Raiko</span><br /></span></div></div> <div class="t260__icon"> <div class="t260__lines"> <div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div> <div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div> </div> <div class="t260__circle" style="background: #ffffff"></div> </div> </div> <div class="t260__content"> <div class="t260__textwrapper"> <div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Goals: <br /> <ul> <li> Perform a comparative phylogenetic assay of the genomes of seven plants.<br /> </li> <li> Find the patterns between specific genes and apomixis plant-forms.<br /> </li> <li> Find orthologous genes in other representatives of the Brassicaceae family.<br /> </li> <li> Build the trees of genes of interest.<br /> </li></ul><br /><a href="https://github.com/rostkick/Boechera" style="" target="_blank">GitHub<br /></a><a href="http://bioinformaticsinstitute.ru/sites/default/files/skithenko_final_260518.pdf" target="_blank" style="">Slides</a></div></div> </div> </div> </div> </div> </div> </div></div><script type="text/javascript"> $(document).ready(function(){ t260_init(); });</script></div><div id="rec41128863" class="r t-rec" style=" " data-animationappear="off" data-record-type="407" ><!-- T407 --><div class="t407"> <div class="t-container_100 "> <div class="t407__wrapper"> <div class="t407__col t407__col_notempty" style="background-color: #fafafa;"> <a href="/popularization" target=""> <div class="t407__textwrapper"> <div class="t407__uptitle t-uptitle t-uptitle_xs" style="color: #3a81e8;color:#3a81e8;" field="subtitle" ><span style="font-weight: 700;">Previous</span></div> <div class="t407__title t-name t-name_lg" style="color: #3a81e8;" field="title" ><span style="font-weight: 300;">Популяризация</span></div> </div> </a> </div> <div class="t407__col t407__col_notempty" style="background-color: #f7f7f7;"> <a href="/research" target=""> <div class="t407__textwrapper t-align_right"> <div class="t407__uptitle t-uptitle t-uptitle_xs" style="color: #3a81e8;color:#3a81e8;" field="subtitle2" ><span style="font-weight: 700;">Next</span></div> <div class="t407__title t-name t-name_lg" style="color: #3a81e8;" field="title2" ><span style="font-weight: 300;">Наука</span></div> </div> </a> </div> </div> </div></div></div><div id="rec90576803" class="r t-rec t-rec_pt_75 t-rec_pb_90" style="padding-top:75px;padding-bottom:90px;background-color:#ffffff; " data-animationappear="off" data-record-type="420" data-bg-color="#ffffff"><!-- T420 --><div class="t420"> <div class="t-container t-align_left"> <div class="t420__col t-col t-col_3"> <a href="#" style="color:#ffffff;font-size:20px;"><img src="https://static.tildacdn.com/tild6665-3836-4636-b630-346264663537/bi_logo.png" class="t420__logo" imgfield="img" style="max-width: 60px;" alt="Об институте"></a> <div class="t-sociallinks"> <div class="t-sociallinks__wrapper"> <div class="t-sociallinks__item"> <a href="https://www.facebook.com/bioinforussia" target="_blank"> <svg class="t-sociallinks__svg" version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="25px" height="25px" viewBox="0 0 48 48" enable-background="new 0 0 48 48" xml:space="preserve"><desc>Facebook</desc><path style="fill:#3c3c50;" d="M47.761,24c0,13.121-10.638,23.76-23.758,23.76C10.877,47.76,0.239,37.121,0.239,24c0-13.124,10.638-23.76,23.764-23.76C37.123,0.24,47.761,10.876,47.761,24 M20.033,38.85H26.2V24.01h4.163l0.539-5.242H26.2v-3.083c0-1.156,0.769-1.427,1.308-1.427h3.318V9.168L26.258,9.15c-5.072,0-6.225,3.796-6.225,6.224v3.394H17.1v5.242h2.933V38.85z"/></svg> </a> </div> <div class="t-sociallinks__item"> <a href="https://twitter.com/bioinforussia" target="_blank"> <svg class="t-sociallinks__svg" version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="25px" height="25px" viewBox="0 0 48 48" enable-background="new 0 0 48 48" xml:space="preserve"><desc>Twitter</desc><path style="fill:#3c3c50;" d="M47.762,24c0,13.121-10.639,23.76-23.761,23.76S0.24,37.121,0.24,24c0-13.124,10.639-23.76,23.761-23.76 S47.762,10.876,47.762,24 M38.031,12.375c-1.177,0.7-2.481,1.208-3.87,1.481c-1.11-1.186-2.694-1.926-4.455-1.926 c-3.364,0-6.093,2.729-6.093,6.095c0,0.478,0.054,0.941,0.156,1.388c-5.063-0.255-9.554-2.68-12.559-6.367 c-0.524,0.898-0.825,1.947-0.825,3.064c0,2.113,1.076,3.978,2.711,5.07c-0.998-0.031-1.939-0.306-2.761-0.762v0.077 c0,2.951,2.1,5.414,4.889,5.975c-0.512,0.14-1.05,0.215-1.606,0.215c-0.393,0-0.775-0.039-1.146-0.109 c0.777,2.42,3.026,4.182,5.692,4.232c-2.086,1.634-4.712,2.607-7.567,2.607c-0.492,0-0.977-0.027-1.453-0.084 c2.696,1.729,5.899,2.736,9.34,2.736c11.209,0,17.337-9.283,17.337-17.337c0-0.263-0.004-0.527-0.017-0.789 c1.19-0.858,2.224-1.932,3.039-3.152c-1.091,0.485-2.266,0.811-3.498,0.958C36.609,14.994,37.576,13.8,38.031,12.375"/></svg> </a> </div> <div class="t-sociallinks__item"> <a href="https://vk.com/bioinf" target="_blank"> <svg class="t-sociallinks__svg" version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="25px" height="25px" viewBox="0 0 48 48" enable-background="new 0 0 48 48" xml:space="preserve"><desc>VK</desc><path style="fill:#3c3c50;" d="M47.761,24c0,13.121-10.639,23.76-23.76,23.76C10.878,47.76,0.239,37.121,0.239,24c0-13.123,10.639-23.76,23.762-23.76C37.122,0.24,47.761,10.877,47.761,24 M35.259,28.999c-2.621-2.433-2.271-2.041,0.89-6.25c1.923-2.562,2.696-4.126,2.45-4.796c-0.227-0.639-1.64-0.469-1.64-0.469l-4.71,0.029c0,0-0.351-0.048-0.609,0.106c-0.249,0.151-0.414,0.505-0.414,0.505s-0.742,1.982-1.734,3.669c-2.094,3.559-2.935,3.747-3.277,3.524c-0.796-0.516-0.597-2.068-0.597-3.171c0-3.449,0.522-4.887-1.02-5.259c-0.511-0.124-0.887-0.205-2.195-0.219c-1.678-0.016-3.101,0.007-3.904,0.398c-0.536,0.263-0.949,0.847-0.697,0.88c0.31,0.041,1.016,0.192,1.388,0.699c0.484,0.656,0.464,2.131,0.464,2.131s0.282,4.056-0.646,4.561c-0.632,0.347-1.503-0.36-3.37-3.588c-0.958-1.652-1.68-3.481-1.68-3.481s-0.14-0.344-0.392-0.527c-0.299-0.222-0.722-0.298-0.722-0.298l-4.469,0.018c0,0-0.674-0.003-0.919,0.289c-0.219,0.259-0.018,0.752-0.018,0.752s3.499,8.104,7.573,12.23c3.638,3.784,7.764,3.36,7.764,3.36h1.867c0,0,0.566,0.113,0.854-0.189c0.265-0.288,0.256-0.646,0.256-0.646s-0.034-2.512,1.129-2.883c1.15-0.36,2.624,2.429,4.188,3.497c1.182,0.812,2.079,0.633,2.079,0.633l4.181-0.056c0,0,2.186-0.136,1.149-1.858C38.281,32.451,37.763,31.321,35.259,28.999"/></svg> </a> </div> <div class="t-sociallinks__item"> <a href="https://www.youtube.com/user/bioinforussia" target="_blank"> <svg class="t-sociallinks__svg" version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="25px" height="25px" viewBox="0 0 48 48" enable-background="new 0 0 48 48" xml:space="preserve"><desc>Youtube</desc><path style="fill:#3c3c50;" d="M24 0.0130005C37.248 0.0130005 47.987 10.753 47.987 24C47.987 37.247 37.247 47.987 24 47.987C10.753 47.987 0.0130005 37.247 0.0130005 24C0.0130005 10.753 10.752 0.0130005 24 0.0130005ZM35.815 18.093C35.565 16.756 34.452 15.758 33.173 15.635C30.119 15.439 27.054 15.28 23.995 15.278C20.936 15.276 17.882 15.432 14.828 15.625C13.544 15.749 12.431 16.742 12.182 18.084C11.898 20.017 11.756 21.969 11.756 23.92C11.756 25.871 11.898 27.823 12.182 29.756C12.431 31.098 13.544 32.21 14.828 32.333C17.883 32.526 20.935 32.723 23.995 32.723C27.053 32.723 30.121 32.551 33.173 32.353C34.452 32.229 35.565 31.084 35.815 29.747C36.101 27.817 36.244 25.868 36.244 23.919C36.244 21.971 36.101 20.023 35.815 18.093ZM21.224 27.435V20.32L27.851 23.878L21.224 27.435Z"/></svg> </a> </div> <div class="t-sociallinks__item"> <a href="https://www.instagram.com/bioinforussia/" target="_blank"> <svg class="t-sociallinks__svg" version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="25px" height="25px" viewBox="0 0 30 30" xml:space="preserve"><desc>Instagram</desc><path style="fill:#3c3c50;" d="M15,11.014 C12.801,11.014 11.015,12.797 11.015,15 C11.015,17.202 12.802,18.987 15,18.987 C17.199,18.987 18.987,17.202 18.987,15 C18.987,12.797 17.199,11.014 15,11.014 L15,11.014 Z M15,17.606 C13.556,17.606 12.393,16.439 12.393,15 C12.393,13.561 13.556,12.394 15,12.394 C16.429,12.394 17.607,13.561 17.607,15 C17.607,16.439 16.444,17.606 15,17.606 L15,17.606 Z"></path><path style="fill:#3c3c50;" d="M19.385,9.556 C18.872,9.556 18.465,9.964 18.465,10.477 C18.465,10.989 18.872,11.396 19.385,11.396 C19.898,11.396 20.306,10.989 20.306,10.477 C20.306,9.964 19.897,9.556 19.385,9.556 L19.385,9.556 Z"></path><path style="fill:#3c3c50;" d="M15.002,0.15 C6.798,0.15 0.149,6.797 0.149,15 C0.149,23.201 6.798,29.85 15.002,29.85 C23.201,29.85 29.852,23.202 29.852,15 C29.852,6.797 23.201,0.15 15.002,0.15 L15.002,0.15 Z M22.666,18.265 C22.666,20.688 20.687,22.666 18.25,22.666 L11.75,22.666 C9.312,22.666 7.333,20.687 7.333,18.28 L7.333,11.734 C7.333,9.312 9.311,7.334 11.75,7.334 L18.25,7.334 C20.688,7.334 22.666,9.312 22.666,11.734 L22.666,18.265 L22.666,18.265 Z"></path></svg> </a> </div> </div></div> <div class="t420__text t-descr t-descr_xxs" style="" field="text">© Bioinformatics Institute</div> </div> <div class="t420__col t-col t-col_3"> <div class="t420__title t-name t-name_xs t420__title_uppercase" field="title" style="color: #3c3c50;">Об институте</div> <div class="t420__descr t-descr t-descr_xxs" field="descr" style="color: #3c3c50;"><ul> <li><a href="http://bioinf.me/admission">Как поступить</a></li><li><a href="https://bioinf.me/jobs" style="">Вакансии</a></li><li><a href="http://bioinformaticsinstitute.ru/about" target="_blank" style="">История</a></li> <li><a href="http://bioinformaticsinstitute.ru/media" target="_blank" style="">СМИ о нас</a></li> <li><a href="http://bioinformaticsinstitute.ru/press" target="_blank" style="">Фирменный стиль</a></li></ul></div> </div> <div class="t420__floatbeaker_lr3"></div> <div> <div class="t420__col t-col t-col_3"> <div class="t420__title t-name t-name_xs t420__title_uppercase" field="title2" style="color: #3c3c50;">направления</div> <div class="t420__descr t-descr t-descr_xxs" field="descr2" style="color: #3c3c50;"><ul><li><a href="/education" style="">Образование</a></li><li><a href="/research" style="">Наука</a></li><li><a href="/popularization" style="">Популяризация</a></li><li><a href="/community" style="">Сообщество</a></li></ul></div> </div> <div class="t420__col t-col t-col_3"> <div class="t420__title t-name t-name_xs t420__title_uppercase" field="title3" style="color: #3c3c50;">Мы в сети</div> <div class="t420__descr t-descr t-descr_xxs" field="descr3" style="color: #3c3c50;"><ul><li><a href="https://vk.com/bioinf" target="_blank" style="">ВКонтакте</a></li><li><a href="https://www.facebook.com/bioinforussia" target="_blank" style="">Facebook</a></li><li><a href="https://twitter.com/bioinforussia" target="_blank" style="">Twitter</a></li><li><a href="https://www.youtube.com/user/bioinforussia" target="_blank" style="">Youtube</a></li><li><a href="https://www.instagram.com/bioinforussia/" target="_blank" style="">Instagram</a></li></ul></div> </div> </div> </div></div></div></div><!--/allrecords--><!-- Stat --><script type="text/javascript">(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)	})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');	ga('create', 'UA-25060708-1', 'auto');	ga('send', 'pageview');	window.mainTracker = 'user';	if (! window.mainTracker) { window.mainTracker = 'tilda'; }	(function (d, w, k, o, g) { var n=d.getElementsByTagName(o)[0],s=d.createElement(o),f=function(){n.parentNode.insertBefore(s,n);}; s.type = "text/javascript"; s.async = true; s.key = k; s.id = "tildastatscript"; s.src=g; if (w.opera=="[object Opera]") {d.addEventListener("DOMContentLoaded", f, false);} else { f(); } })(document, window, '90b11afd574e5a7ae3a15cf961ded7e3','script','https://stat.tildacdn.com/js/tildastat-0.2.min.js');</script></body></html>