<!DOCTYPE html><html><head><meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta name="viewport" content="width=device-width, initial-scale=1.0" /> <!--metatextblock--><title>2017/18</title><meta name="description" content="Институт биоинформатики" /> <meta property="og:url" content="https://bioinf.me/rp1718" /><meta property="og:title" content="2017/18" /><meta property="og:description" content="Институт биоинформатики" /><meta property="og:type" content="website" /><meta property="og:image" content="https://static.tildacdn.com/tild3964-3333-4333-b436-323131343763/qX9T8vKOqi8.jpg" /><link rel="canonical" href="https://bioinf.me/rp1718"><!--/metatextblock--><meta name="format-detection" content="telephone=no" /><meta http-equiv="x-dns-prefetch-control" content="on"><link rel="dns-prefetch" href="https://ws.tildacdn.com"><link rel="dns-prefetch" href="https://static.tildacdn.com"><link rel="shortcut icon" href="https://static.tildacdn.com/tild6265-3030-4832-b937-623232356261/favicon.ico" type="image/x-icon" /><!-- Assets --><script src="https://neo.tildacdn.com/js/tilda-fallback-1.0.min.js" charset="utf-8" async></script><link rel="stylesheet" href="https://static.tildacdn.com/css/tilda-grid-3.0.min.css" type="text/css" media="all" onerror="this.loaderr='y';"/><link rel="stylesheet" href="https://ws.tildacdn.com/project29135/tilda-blocks-page33428671.min.css?t=1675620754" type="text/css" media="all" onerror="this.loaderr='y';" /><link rel="preconnect" href="https://fonts.gstatic.com"><link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;500;600;700&subset=latin,cyrillic" rel="stylesheet"><link rel="stylesheet" href="https://static.tildacdn.com/css/tilda-cover-1.0.min.css" type="text/css" media="all" onerror="this.loaderr='y';" /><script type="text/javascript">(function (d) {
if (!d.visibilityState) {
var s = d.createElement('script');
s.src = 'https://static.tildacdn.com/js/tilda-polyfill-1.0.min.js';
d.getElementsByTagName('head')[0].appendChild(s);
}
})(document);
function t_onReady(func) {
if (document.readyState != 'loading') {
func();
} else {
document.addEventListener('DOMContentLoaded', func);
}
}
function t_onFuncLoad(funcName, okFunc, time) {
if (typeof window[funcName] === 'function') {
okFunc();
} else {
setTimeout(function() {
t_onFuncLoad(funcName, okFunc, time);
},(time || 100));
}
}</script><script src="https://static.tildacdn.com/js/tilda-scripts-3.0.min.js" charset="utf-8" defer onerror="this.loaderr='y';"></script><script src="https://ws.tildacdn.com/project29135/tilda-blocks-page33428671.min.js?t=1675620754" charset="utf-8" async onerror="this.loaderr='y';"></script><script src="https://static.tildacdn.com/js/lazyload-1.3.min.js" charset="utf-8" async onerror="this.loaderr='y';"></script><script src="https://static.tildacdn.com/js/tilda-menu-1.0.min.js" charset="utf-8" async onerror="this.loaderr='y';"></script><script src="https://static.tildacdn.com/js/tilda-cover-1.0.min.js" charset="utf-8" async onerror="this.loaderr='y';"></script><script src="https://static.tildacdn.com/js/jquery-1.10.2.min.js" charset="utf-8" onerror="this.loaderr='y';"></script><script src="https://static.tildacdn.com/js/tilda-events-1.0.min.js" charset="utf-8" async onerror="this.loaderr='y';"></script><script type="text/javascript">window.dataLayer = window.dataLayer || [];</script><script type="text/javascript">(function () {
if((/bot|google|yandex|baidu|bing|msn|duckduckbot|teoma|slurp|crawler|spider|robot|crawling|facebook/i.test(navigator.userAgent))===false && typeof(sessionStorage)!='undefined' && sessionStorage.getItem('visited')!=='y' && document.visibilityState){
var style=document.createElement('style');
style.type='text/css';
style.innerHTML='@media screen and (min-width: 980px) {.t-records {opacity: 0;}.t-records_animated {-webkit-transition: opacity ease-in-out .2s;-moz-transition: opacity ease-in-out .2s;-o-transition: opacity ease-in-out .2s;transition: opacity ease-in-out .2s;}.t-records.t-records_visible {opacity: 1;}}';
document.getElementsByTagName('head')[0].appendChild(style);
function t_setvisRecs(){
var alr=document.querySelectorAll('.t-records');
Array.prototype.forEach.call(alr, function(el) {
el.classList.add("t-records_animated");
});
setTimeout(function () {
Array.prototype.forEach.call(alr, function(el) {
el.classList.add("t-records_visible");
});
sessionStorage.setItem("visited", "y");
}, 400);
} 
document.addEventListener('DOMContentLoaded', t_setvisRecs);
}
})();</script></head><body class="t-body" style="margin:0;"><!--allrecords--><div id="allrecords" class="t-records" data-hook="blocks-collection-content-node" data-tilda-project-id="29135" data-tilda-page-id="33428671" data-tilda-page-alias="rp1718" data-tilda-formskey="f0696df0d2ceecbcfdc18140c73b1757" data-tilda-lazy="yes"><div id="rec540317941" class="r t-rec" style="background-color:#ffffff; " data-animationappear="off" data-record-type="230" data-bg-color="#ffffff"><div id="nav540317941" class="t199__header t199__js__header " style="background-color:#ffffff;" data-menu="yes"><div class="t199__holder"><a class="t199__logo" href="http://bioinf.me" style=""><img class="t199__logo-image-one" 
src="https://static.tildacdn.com/tild6464-3064-4237-a433-383539613333/bi_logo.png" 
imgfield="img"
style="max-width: 60px; width: 100%; height: auto;" alt="" role="presentation"><img class="t199__logo-image-two" 
src="https://static.tildacdn.com/tild3261-3534-4537-a238-653361653739/bi_logo_white.png" 
imgfield="img2"
style="max-width: 60px; width: 100%; height: auto;" alt="" role="presentation"><div class="t199__logo-text-mobile t-title" field="title" style=""></div> </a><a class="t199__mmenu-toggler t199__js__menu-toggler" href="#"><div class="t-menuburger t-menuburger_first "><span style="background-color:#fff;"></span><span style="background-color:#fff;"></span><span style="background-color:#fff;"></span><span style="background-color:#fff;"></span></div><script>function t_menuburger_init(recid) {
var rec = document.querySelector('#rec' + recid);
if (!rec) return;
var burger = rec.querySelector('.t-menuburger');
if (!burger) return;
var isSecondStyle = burger.classList.contains('t-menuburger_second');
if (isSecondStyle && !window.isMobile && !('ontouchend' in document)) {
burger.addEventListener('mouseenter', function() {
if (burger.classList.contains('t-menuburger-opened')) return;
burger.classList.remove('t-menuburger-unhovered');
burger.classList.add('t-menuburger-hovered');
});
burger.addEventListener('mouseleave', function() {
if (burger.classList.contains('t-menuburger-opened')) return;
burger.classList.remove('t-menuburger-hovered');
burger.classList.add('t-menuburger-unhovered');
setTimeout(function() {
burger.classList.remove('t-menuburger-unhovered');
}, 300);
});
}
burger.addEventListener('click', function() {
if (!burger.closest('.tmenu-mobile') &&
!burger.closest('.t450__burger_container') &&
!burger.closest('.t466__container') &&
!burger.closest('.t204__burger')) {
burger.classList.toggle('t-menuburger-opened');
burger.classList.remove('t-menuburger-unhovered');
}
});
var menu = rec.querySelector('[data-menu="yes"]');
if (!menu) return;
var menuLinks = menu.querySelectorAll('.t-menu__link-item');
var submenuClassList = ['t978__menu-link_hook', 't978__tm-link', 't966__tm-link', 't794__tm-link', 't-menusub__target-link'];
Array.prototype.forEach.call(menuLinks, function (link) {
link.addEventListener('click', function () {
var isSubmenuHook = submenuClassList.some(function (submenuClass) {
return link.classList.contains(submenuClass);
});
if (isSubmenuHook) return;
burger.classList.remove('t-menuburger-opened');
});
});
}
t_onReady(function() {
t_onFuncLoad('t_menuburger_init', function(){t_menuburger_init('540317941');});
});</script><style>.t-menuburger {
position: relative;
flex-shrink: 0;
width: 28px;
height: 20px;
-webkit-transform: rotate(0deg);
transform: rotate(0deg);
transition: .5s ease-in-out;
cursor: pointer;
z-index: 999;
}
/*---menu burger lines---*/
.t-menuburger span {
display: block;
position: absolute;
width: 100%;
opacity: 1;
left: 0;
-webkit-transform: rotate(0deg);
transform: rotate(0deg);
transition: .25s ease-in-out;
height: 3px;
background-color: #000;
}
.t-menuburger span:nth-child(1) {
top: 0px;
}
.t-menuburger span:nth-child(2),
.t-menuburger span:nth-child(3) {
top: 8px;
}
.t-menuburger span:nth-child(4) {
top: 16px;
}
/*menu burger big*/
.t-menuburger__big {
width: 42px;
height: 32px;
}
.t-menuburger__big span {
height: 5px;
}
.t-menuburger__big span:nth-child(2),
.t-menuburger__big span:nth-child(3) {
top: 13px;
}
.t-menuburger__big span:nth-child(4) {
top: 26px;
}
/*menu burger small*/
.t-menuburger__small {
width: 22px;
height: 14px;
}
.t-menuburger__small span {
height: 2px;
}
.t-menuburger__small span:nth-child(2),
.t-menuburger__small span:nth-child(3) {
top: 6px;
}
.t-menuburger__small span:nth-child(4) {
top: 12px;
}
/*menu burger opened*/
.t-menuburger-opened span:nth-child(1) {
top: 8px;
width: 0%;
left: 50%;
}
.t-menuburger-opened span:nth-child(2) {
-webkit-transform: rotate(45deg);
transform: rotate(45deg);
}
.t-menuburger-opened span:nth-child(3) {
-webkit-transform: rotate(-45deg);
transform: rotate(-45deg);
}
.t-menuburger-opened span:nth-child(4) {
top: 8px;
width: 0%;
left: 50%;
}
.t-menuburger-opened.t-menuburger__big span:nth-child(1) {
top: 6px;
}
.t-menuburger-opened.t-menuburger__big span:nth-child(4) {
top: 18px;
}
.t-menuburger-opened.t-menuburger__small span:nth-child(1),
.t-menuburger-opened.t-menuburger__small span:nth-child(4) {
top: 6px;
}
/*---menu burger first style---*/
@media (hover), (min-width:0\0) {
.t-menuburger_first:hover span:nth-child(1) {
transform: translateY(1px);
}
.t-menuburger_first:hover span:nth-child(4) {
transform: translateY(-1px);
}
.t-menuburger_first.t-menuburger__big:hover span:nth-child(1) {
transform: translateY(3px);
}
.t-menuburger_first.t-menuburger__big:hover span:nth-child(4) {
transform: translateY(-3px);
}
}
/*---menu burger second style---*/
.t-menuburger_second span:nth-child(2),
.t-menuburger_second span:nth-child(3) {
width: 80%;
left: 20%;
right: 0;
}
@media (hover), (min-width:0\0) {
.t-menuburger_second.t-menuburger-hovered span:nth-child(2),
.t-menuburger_second.t-menuburger-hovered span:nth-child(3) {
animation: t-menuburger-anim 0.3s ease-out normal forwards;
}
.t-menuburger_second.t-menuburger-unhovered span:nth-child(2),
.t-menuburger_second.t-menuburger-unhovered span:nth-child(3) {
animation: t-menuburger-anim2 0.3s ease-out normal forwards;
}
}
.t-menuburger_second.t-menuburger-opened span:nth-child(2),
.t-menuburger_second.t-menuburger-opened span:nth-child(3){
left: 0;
right: 0;
width: 100%!important;
}
/*---menu burger third style---*/
.t-menuburger_third span:nth-child(4) {
width: 70%;
left: unset;
right: 0;
}
@media (hover), (min-width:0\0) {
.t-menuburger_third:not(.t-menuburger-opened):hover span:nth-child(4) {
width: 100%;
}
}
.t-menuburger_third.t-menuburger-opened span:nth-child(4) {
width: 0!important;
right: 50%;
}
/*---menu burger fourth style---*/
.t-menuburger_fourth {
height: 12px;
}
.t-menuburger_fourth.t-menuburger__small {
height: 8px;
}
.t-menuburger_fourth.t-menuburger__big {
height: 18px;
}
.t-menuburger_fourth span:nth-child(2),
.t-menuburger_fourth span:nth-child(3) {
top: 4px;
opacity: 0;
}
.t-menuburger_fourth span:nth-child(4) {
top: 8px;
}
.t-menuburger_fourth.t-menuburger__small span:nth-child(2),
.t-menuburger_fourth.t-menuburger__small span:nth-child(3) {
top: 3px;
}
.t-menuburger_fourth.t-menuburger__small span:nth-child(4) {
top: 6px;
}
.t-menuburger_fourth.t-menuburger__small span:nth-child(2),
.t-menuburger_fourth.t-menuburger__small span:nth-child(3) {
top: 3px;
}
.t-menuburger_fourth.t-menuburger__small span:nth-child(4) {
top: 6px;
}
.t-menuburger_fourth.t-menuburger__big span:nth-child(2),
.t-menuburger_fourth.t-menuburger__big span:nth-child(3) {
top: 6px;
}
.t-menuburger_fourth.t-menuburger__big span:nth-child(4) {
top: 12px;
}
@media (hover), (min-width:0\0) {
.t-menuburger_fourth:not(.t-menuburger-opened):hover span:nth-child(1) {
transform: translateY(1px);
}
.t-menuburger_fourth:not(.t-menuburger-opened):hover span:nth-child(4) {
transform: translateY(-1px);
}
.t-menuburger_fourth.t-menuburger__big:not(.t-menuburger-opened):hover span:nth-child(1) {
transform: translateY(3px);
}
.t-menuburger_fourth.t-menuburger__big:not(.t-menuburger-opened):hover span:nth-child(4) {
transform: translateY(-3px);
}
}
.t-menuburger_fourth.t-menuburger-opened span:nth-child(1),
.t-menuburger_fourth.t-menuburger-opened span:nth-child(4) {
top: 4px;
}
.t-menuburger_fourth.t-menuburger-opened span:nth-child(2),
.t-menuburger_fourth.t-menuburger-opened span:nth-child(3) {
opacity: 1;
}
/*---menu burger animations---*/
@keyframes t-menuburger-anim {
0% {
width: 80%;
left: 20%;
right: 0;
}
50% {
width: 100%;
left: 0;
right: 0;
}
100% {
width: 80%;
left: 0;
right: 20%;
}
}
@keyframes t-menuburger-anim2 {
0% {
width: 80%;
left: 0;
}
50% {
width: 100%;
right: 0;
left: 0;
}
100% {
width: 80%;
left: 20%;
right: 0;
}
}</style> </a><div class="t199__mmenu t199__js__menu"><nav class="t199__menu" 
aria-label="Основные разделы на странице"><ul class="t199__menu-items" 
role="menubar" aria-label="Основные разделы на странице"><li class="t199__menu-item-wrap" role="none"><a class="t199__menu-item t-title t-menu__link-item" 
href="/education/program#details"
role="menuitem" tabindex="0" data-menu-submenu-hook="" 
>ПРОГРАММЫ</a></li><li class="t199__menu-item-wrap" role="none"><a class="t199__menu-item t-title t-menu__link-item" 
href="/education/program#timeline"
role="menuitem" tabindex="0" data-menu-submenu-hook="" 
>ПОСТУПЛЕНИЕ</a></li><li class="t199__menu-item-wrap" role="none"><a class="t199__menu-item t-title t-menu__link-item" 
href="/education/program#courses"
role="menuitem" tabindex="0" data-menu-submenu-hook="" 
>ПРЕДМЕТЫ</a></li><li class="t199__menu-item-wrap" role="none"><a class="t199__menu-item t-title t-menu__link-item" 
href="/education/program#instructors"
role="menuitem" tabindex="0" data-menu-submenu-hook="" 
>ПРЕПОДАВАТЕЛИ</a></li><li class="t199__menu-item-wrap" role="none"><a class="t199__menu-item t-title t-menu__link-item" 
href="/education/projects"
target="_blank" 
role="menuitem" tabindex="0" data-menu-submenu-hook="" 
>ПРОЕКТЫ</a></li><li class="t199__menu-item-wrap" role="none"><a class="t199__menu-item t-title t-menu__link-item" 
href="/community/graduates"
target="_blank" 
role="menuitem" tabindex="0" data-menu-submenu-hook="" 
>ВЫПУСКНИКИ</a></li><li class="t199__menu-item-wrap" role="none"><a class="t199__menu-item t-title t-menu__link-item" 
href="/education/program/materials"
target="_blank" 
role="menuitem" tabindex="0" data-menu-submenu-hook="" 
>МАТЕРИАЛЫ</a></li><li class="t199__menu-item-wrap" role="none"><a class="t199__menu-item t-title t-menu__link-item" 
href="/education/faq"
target="_blank" 
role="menuitem" tabindex="0" data-menu-submenu-hook="" 
>FAQ</a></li></ul></nav><div class="t199__lang"><a class="t199__lang-item" style="" href="https://bioinf.me/en/education/">En</a></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t199_showMenu', function () {
t199_showMenu('540317941');
});
t_onFuncLoad('t199_positionHeader', function () {
t199_positionHeader('540317941');
});
t_onFuncLoad('t_menu__highlightActiveLinks', function () {
t_menu__highlightActiveLinks(".t199__menu-item");
});
t_onFuncLoad('t_menu__findAnchorLinks', function () {
t_menu__findAnchorLinks('540317941', '.t199__menu-item');
});
});</script><style>@media screen and (max-width: 1024px) {
}</style><style>#rec540317941 .t-menu__link-item{
-webkit-transition: color 0.3s ease-in-out, opacity 0.3s ease-in-out;
transition: color 0.3s ease-in-out, opacity 0.3s ease-in-out; 
outline: none;
}
/* #rec540317941 .t-menu__link-item:not(:focus-visible){
outline: none;
} */
#rec540317941 .t-menu__link-item.t-active:not(.t978__menu-link){
color:#3a81e8 !important; }
#rec540317941 .t-menu__link-item:not(.t-active):not(.tooltipstered):hover{
opacity: 0.9 !important;}
@supports (overflow:-webkit-marquee) and (justify-content:inherit)
{
#rec540317941 .t-menu__link-item,
#rec540317941 .t-menu__link-item.t-active {
opacity: 1 !important;
}
}</style></div><div id="rec540332544" class="r t-rec" style=" " data-animationappear="off" data-record-type="205" ><!-- cover --><div class="t-cover"
id="recorddiv540332544"
bgimgfield="img"
style="height:70vh; background-image:url('https://static.tildacdn.com/tild3666-6661-4537-b938-326162386331/-/resize/20x/residuals.png');"
><div class="t-cover__carrier" id="coverCarry540332544"data-content-cover-id="540332544"data-content-cover-bg="https://static.tildacdn.com/tild3666-6661-4537-b938-326162386331/residuals.png"data-display-changed="true"data-content-cover-height="70vh"data-content-cover-parallax=""style="height:70vh;background-attachment:scroll; "itemscope itemtype="http://schema.org/ImageObject"><meta itemprop="image" content="https://static.tildacdn.com/tild3666-6661-4537-b938-326162386331/residuals.png"></div><div class="t-cover__filter" style="height:70vh;background-image: -moz-linear-gradient(top, rgba(0,0,0,0.70), rgba(0,0,0,0.80));background-image: -webkit-linear-gradient(top, rgba(0,0,0,0.70), rgba(0,0,0,0.80));background-image: -o-linear-gradient(top, rgba(0,0,0,0.70), rgba(0,0,0,0.80));background-image: -ms-linear-gradient(top, rgba(0,0,0,0.70), rgba(0,0,0,0.80));background-image: linear-gradient(top, rgba(0,0,0,0.70), rgba(0,0,0,0.80));filter: progid:DXImageTransform.Microsoft.gradient(startColorStr='#4c000000', endColorstr='#33000000');"></div><div class="t-container"><div class="t-width t-width_10" style="margin:0 auto;"><div class="t-cover__wrapper t-valign_middle" style="height:70vh; position: relative;z-index:1;"><div class="t182"><div data-hook-content="covercontent"><div class="t182__wrapper"><div class="t182__title t-title t-title_xl" style="" field="title">Research projects</div><div class="t182__descr t-descr t-descr_lg" style="" field="descr">Academic year 2017/18</div></div></div></div></div></div></div></div></div><div id="rec540318196" class="r t-rec t-rec_pt_30 t-rec_pb_30" style="padding-top:30px;padding-bottom:30px; " data-record-type="397" ><!-- t397 --><div class="t397"><div class="t-container"><div class="t397__col t-width t-width_6"><ul class="t397__wrapper t-align_center" 
role="tablist" data-tab-current="1"><li role="presentation" 
class="t397__tab t397__tab_active t397__width_50" data-tab-rec-ids="41447892,41447935,41447953,41447995,41448007,41448026,41448068,41448073,41448090,41448185,41448198,41448245,41448254,41448273,41448282,41448285,41448293,41448302,41448306,41448374,41448377,41463854,69562162" data-tab-number="1" 
style="border-bottom: 1px solid #eee;;"><button 
type="button" 
class="t397__title t-name t-name_xs" 
style="font-weight:400;" 
id="tab1_540318196" 
role="tab" 
aria-selected="false" 
aria-controls="rec41447892" 
tabindex="-1" 
field="title">Осень 2017</button></li><li role="presentation" 
class="t397__tab t397__width_50" data-tab-rec-ids="51524981,51526125,51526629,51526982,51539987,51527481,51547404,51538213,51538666,51539268,51796966,51797277,51798023,51799445,51798887,51529168" data-tab-number="2" 
style="border-bottom: 1px solid #eee;;"><button 
type="button" 
class="t397__title t-name t-name_xs" 
style="font-weight:400;" 
id="tab2_540318196" 
role="tab" 
aria-selected="false" 
aria-controls="rec51524981" 
tabindex="-1" 
field="title">Весна 2018</button></li></ul><div class="t397__wrapper_mobile"><div class="t397__firefoxfix"></div><select autocomplete="off" class="t397__select t-name"><option value="41447892,41447935,41447953,41447995,41448007,41448026,41448068,41448073,41448090,41448185,41448198,41448245,41448254,41448273,41448282,41448285,41448293,41448302,41448306,41448374,41448377,41463854,69562162">Осень 2017</option><option value="51524981,51526125,51526629,51526982,51539987,51527481,51547404,51538213,51538666,51539268,51796966,51797277,51798023,51799445,51798887,51529168">Весна 2018</option></select></div></div></div></div><style>#rec540318196 .t397__tab_active {
border-bottom-color: #222 !important;
}
#rec540318196 .t397__tab_active .t397__title {
color: #222 !important;
font-weight: 600 !important;
}
#rec540318196 .t397__select {
border: 1px solid #222;
color: #222;
}
#rec540318196 .t397__wrapper_mobile:after {
border-color: #222 transparent transparent transparent;
}
#rec540318196 .t397__firefoxfix {
top: 1px;
bottom: 1px;
right: 1px;
}</style><script>t_onReady(function () {
t_onFuncLoad('t397_init', function () {
t397_init('540318196');
});
});</script></div><div id="rec540318197" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Phylogenetic networks comparison</strong> | ITMO University<br /><br /><span style="font-size: 20px;"><strong>students:</strong> Anton Eliseev, Natalia Klimenko, Elena Pazhenkova<br /><strong>scientific adviser:</strong> Nikita Alekseev</span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Phylogenetic networks are used to visualize evolutionary relationships that reflect any reticulations (such as hybridization). The amount of reticulation edges is a widely used criterion of networks, however, this measure is often identical in different topologies. We propose to use the number of possible convex colorings as metrics to distinguish networks with equal number of hybridizations. The number of convex colorings shows how many homoplasy-free characters are possible within this phylogeny. <br /><br /> Six species of Heliconius butterflies was chosen as model group to test our algorithm. A peculiar trait of genus Heliconius is the prevalence of interspecific hybridization, which reflects on phylogenetic networks as reticulation events. As suggested earlier, H. heurippa and H. elevatus have resulted from hybrid speciation [1, 2]. We analyze 20 nuclear genes, obtain NJ trees for each gene, compare these trees using pairwise Branch Score Distances, concatenate genes providing the most similar trees (distances up to 0.015), calculate hybridization networks and estimate numbers of convex colorings for each network. The network with the largest count of convex colorings is congruent with the hypothesis of hybrid origin of heurippa and elevatus species. <br /><br /> Another part of the study concerned the phylogeny of potatoes. 420 potato plants classified earlier as 29 species (7 cultivated and 22 wild) were analyzed by 15 plastid SSR-markers. As genomes were plastid, no hybridization was observed. We concentrated on building the most accurate phylogenetic tree for this data. Dendrograms were based on the Manhattan distance matrix. Cultivated and wild species of potato are clearly distinguishable. The idea of dividing Solanum tuberosum into Andigenum group and Chilotanum groups (according to [3]) is correct. Results of molecular analysis don't correspond to classification based on morphological features. <br /><br /> References: <br /> 1.Kronforst M.R., Papa R. The Functional Basis of Wing Patterning in Heliconius Butterflies: The Molecules Behind Mimicry. GENETICS. 2015. 200(1): 1-19 <br />2.Salazar C., Baxter S.W., Pardo-Diaz C., Wu G., Surridge A., Linares M., Bermingham E., Jiggins C.D. Genetic Evidence for Hybrid Trait Speciation in Heliconius Butterflies. PLoS Genet. 2010. 6(4): e1000930. <br /> 3.Spooner D.M., Ghislain M., Simon R., Jansky S.H., Gavrilenko T. Systematics, Diversity, Genetics, and Evolution of Wild and Cultivated Potatoes. Bot. Rev. 2014. 80: 283–383 <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/eliseev_klimenko_pazhenkova_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318198" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Optimization of spectral network parameters</strong> | EPAM Lifescience<br /><br /><strong></strong><span style="font-size: 20px;"><strong>students:</strong> Evgenia Fedotova , Rostislav Skitchenko , Ksenia Cherenkova <br /><strong><strong data-redactor-tag="strong">scientific adviser:</strong></strong> Gennadiy Zacharov</span></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">A pipeline for exome and target-sequencing analyses was developed. It's results could be used by physicians for diagnosis refinement. Such problems as pipeline deployment, it's utilities versions and dependencies control was solved by using Docker software. Pipeline quality control was obtained for NA12848 GIAB sample: Precision 0.95 and Sensitivity 0.78. <br /><br /> We've analyzed sequence results of cardyo-panel for families, whose members had diagnosis cardiomyopathy. Dependencies between variations and clinical diagnosis cardiomyopathy was found. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/skitchenko_cherenkova_fedotova_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318199" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Construction of RNA fragment database</strong> | University of North Carolina at Chapel Hill<br /><span style="font-size: 20px;"><br /><strong>student:</strong> Alexandr Ilin<br /><strong></strong><strong>scientific advisers:</strong> J. Wang, N. Dokholyan<br /></span></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">RNA plays significant role in regulation of gene expression at transcriptional and translational levels. This is achieved because of appropriate spatial structure of RNA molecule (i.e. motif), which is obtained after folding. Ability to predict 3-dimensional structure given sequence of RNA oligonucleotide is very important due to possibility to make use of this information in construction molecules with predefined structure – thus with known properties and targets to interact. Therefore, it supports design of new RNAs, which can be used as medications against wide spectrum of diseases caused by consequences of problems with gene product abundance. <br /><br /> In this work we developed RNA secondary structure decomposition algorithm to decompose an integrated RNA into many motifs. According to the RNA secondary structure decomposition algorithm, an RNA 3D motifs database was built by decomposing all the RNA 3D structures downloaded from PDB. We devised an algorithm to analyze and compare the base interactions networks between different RNA 3D motifs. We classified and clustered all the RNA 3D motifs in the database by using the network comparison algorithm. We utilized the supervised machine learning method to learn the relationship between sequences and base interactions networks of clustered RNA 3D motifs. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/ilin_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318200" class="r t-rec t-rec_pt_0 t-rec_pb_0" style="padding-top:0px;padding-bottom:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="font-size:24px;color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong></strong><strong>Assembly of mammalian genomes using GemCode data </strong>| Center for Algorithmic Biotechnology, St. Petersburg State University<br /><br /><strong>student:</strong> Angira Kekteeva<br /><strong></strong><strong>scientific advisers:</strong> Ivan Tolstoganov, Anton Bankevich</span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">GemCode technology that was developed by 10X Genomics Company is actively used for assembly of mammalian genomes. CloudSPAdes is a genome assembly algorithm which was designed for metagenome assembly. However, algorithms in this tool, that were developed for resolving repetitions in the assembly graph, can be successfully used for assembly of mammalian genomes. <br /><br /> In this work we've examined exisiting metagenome assembly algorithms and analysed the disadvatages of using them for large genomes. Our analysis has shown that the average number of close edges in a human genome graph is more, than in metagenomes assembly graph, so it requires additional methods for sequencing long edges in the genomes of mammals. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/kekteeva_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318201" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Searching for molecular markers of chromosome bands</strong> | Bioinformatics Institute<br /><br /><span style="font-size: 20px;"><strong></strong><strong>student:</strong> Alexandra Klimina<br /><strong>scientific advisers: </strong>Yury Barbitoff</span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Giemsa staining produces specific bands on metaphase chromosomes that have coloring of different intensity (G-bands). There are known correlations between the intensity of coloring and the degree of chromatin condensation, GC-content, and replication time. However, little is known about molecular markers of such banding pattern. <br /><br /> Main purpose of this project was to develop a tool for analysis of genome-wide correlation between different genomic features. We implemented the tool in Java with possibility to work on Spark-cluster for distributed computitions. <br /><br /> We chose previously described Projection test and Jaccard test to analyze the dependence between the reference (e.g., chromosome bands) and query feature of interest. We estimated the significance of correlation by sampling 1000 sets of randomly distributed intervals of the same length as the query feature followed by Kolmogorov-Smirnov normality test and one-sample t-test to obtain the p-value of association. <br /><br /> We tested our tool by analyzing correlation between chromosome banding pattern and such features as CpG-islands, microsatellite repeats and DNAse hypersensitivity regions. Expectedly, we showed that G-positive bands are positively correlated with microsattelite repeats, and negatively - with CpG-islands and open chromatin, DNAse hypersensitive regions. Thus, our tool can be used to further analyze genome-wide correlations between banding pattern and diverse molecular features. <br /><br /> <a href="http://bioinformaticsinstitute.ru/sites/default/files/klimina_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318202" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-size: 20px;"><strong></strong></span><span style="font-weight: 100;"><strong>Association of methylation level CpG-islands and IQ-level</strong> | University of Houston<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Daria Krytskaya<br /><strong></strong><strong>scientific adviser: </strong>Olga Naumova</span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Methylation is an epigenomics modification of DNA. It change the activity of a DNA segment without changing the sequence. The most methylated region is region riched guanine and cytosine called CpG-islands. <br /><br /> In this work, we evaluate the methylation level of all known 26640 CpG-islands as average value of this region. Than we made a correlation test with using the Benjamini-Hochberg procedure for decreases the false discovery rate. <br /><br /> Analysis of biological role of this region was make with UCSC Genome Browser on Human Feb. 2009 (GRCh37/hg19) Assembly and GeneCards. <br /> Among our results are predicted transcription and translation region and known protein. For example, on 1 chromosome the most significant region is (38059428, 38063740) contained: prediction region are ENST00000373062, ENST00000463351, ENST00000488496 and gene of known proteins GLN2 – Homo sapiens guanine nucleotide binding protein-like 2 (nucleolar). For region (54951893, 54957287) of 1 chromosome are not predictions. For 2 chromosome the most significant region is (65213598, 65219212) contained gene of known proteins SLC1A4 Gene – Solute Carrier Family 1 Member 4. It is a transporter of alanine, serine, cysteine, threonine. Predicts a transport of a glucose by this protein. Disorders associated with this protein are spastic tetraplegia, thin corpus callosum, progressive microcephaly, microcephaly. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/krytskaya_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318203" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Genome structure of Mycobacterium tuberculosis strains in different world regions </strong>|<br />Theodosius Dobzhansky Center for Genome Bioinformatics, St. Petersburg State University<br /><br /><span style="font-size: 20px;"><strong>students:</strong> Vladimir Klimov, Vladimir Molchanov<br /><strong></strong><strong>scientific adviser:</strong> Ekaterina Chernyaeva</span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr">Due to the high epidemiological rate of Mycobacterium tuberculosis and its constantly updated genomic data the problem of genome data analysis and systematization becomes extremely significant. For this reason our project was devoted to extend Genome-based Mycobacterium tuberculosis Variation (GMTV) Database which was developed by the researchers of Theodosius Dobzhansky Center for Genome Bioinformatics (Chernayeva et al., 2014). <br /><br /> In this study we performed an analysis of 999 M. tuberculosis strains which was isolated from patients in Malawi Republic. To achieve this results we designed a pipeline aimed at single nucleotide polymorphisms (SNPs) and insertions/deletions (InDels) identification from M. tuberculosis whole genome sequencing data. <br /><br />This pipeline based on BWA-mem and GATK programs which are widely used in such kind of investigations. Considering big amount of NGS data we suggested simple and rapid method to visualize or estimate quality control results performed by FastQC program using python3 regular expression and plotting in R. All variant calls (.vcf files) was uploaded on database, in future this data could be used for clade-specific annotation, which gives a possibility to identify strains without NGS methods. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/klimov_molchanov.pdf" style="" target="_blank">Slides</a></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318204" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Comparative analysis of natural selection effects across human populations</strong> | <br />Bioinformatics Institute<br /><span style="font-size: 20px;"><br /><strong>student:</strong> Julia Kornienko<br /><strong></strong><strong>scientific adviser:</strong> Yury Barbitoff</span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">In this project we aimed to estimate the natural selection effects across human populations based on the Genome Aggregation Database (gnomAD) dataset which contains information about sequence variants in 123136 human exomes and 15,496 genomes. To this end we calculated the amount of protein truncating variants (PTV) both (i) per individual genes (based on GENCODE v.19); and (ii) per gene sets (hallmarks and canonical pathways obtained from the MSigDB Collections) for six different populations (European, South Asian, Latino American, East Asian, African and Finnish). <br /><br /> We estimated the selective coefficients of heterozygous PTVs for different human populations from the constructed dataset in the same way as it was done by Cassa et al. (2017) and found that distribution of selective coefficients both per individual genes and gene sets is dependent on the population size. Taking this into account, we evaluated the difference in distribution of PTV allele counts among the populations and found that for 2040 of 12367 analyzed genes and for 746 of 1379 of analyzed gene sets selective effects were significantly population-dependent. Thus it is possible to conclude that selective effects for some genes do vary across the populations. <br /><br /> Interestingly, we discovered significant enrichment of PTV alleles in the immune system-related pathways (IL-10, IL-13 and IFNG signaling) in the individuals of South Asian ancestry (SAS), with more than half of all PTVs discovered in the corresponding genes belonging to the SAS population. These results are concordant with some previous findings and emphasize the natural heterogeneity of selective effects. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/kornienko_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318205" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>De novo cdr3 annotation in VDJ rna sequences</strong> | Center for Algorithmic Biotechnology, St. Petersburg State University<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Kristina Krivonosova<br /><strong></strong><strong>scientific advisers:</strong> Andrey Slabodkin, Maria Chernigovskaya</span></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"> The project relates to the construction and analysis of the repertoire of antibodies. In order to build a repertoire, we look for mutations in antibodies with the help of a germline by aligning the variable part of the immunoglobulin gene with a special base of V-, D-, and J-genes (germline). With this alignment, we can annotate the sequence: we mark the boundaries of the V-gene and the J-gene, as well as the boundaries of the three regions that determine the specificity of the antibody to the antigen (CDRs). In practice the third region (CDR3) is the most variable part of the immunoglobulin gene so its borders are of the greatest interest. Unfortunately, on some data (for instance, in case of lymphoma) the level of mutations goes off the scale and we can not build an alignment on the germline for that data. <br /><br /> In this work we develop a heuristic for VDJ sequence annotation that does not use alignments. This new heuristic is based on searching conserved regions in the source sequence to identify CDR3 regions. In practice this approach produces satisfying results with accuracy rate of 95% when applied to verified data sets. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/krivonosova_final_16122017.pdf" style="font-size: 18px;" target="_blank">Slides</a><br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318206" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Forming a panel of markers for the molecular-genetic diagnosis of congenital<br />metabolic disorders </strong>| Parseq Lab<br /><br /><span style="font-size: 20px;"><strong></strong><strong>student:</strong> Ekaterina Nebozhatko<br /><strong></strong><strong>scientific advisers:</strong> Tamara Simakova, Anton Bragin</span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Predicting the deleterious effects of mutation on protein function is one of the main tasks of genetics. Often researchers use predictive tools for this. The main problem with the use of predictive tools is not enough high sensitivity and specificity of classifiers. On average, the sensitivity is 80%, which means that 20% of the possible pathogenic mutations may go unnoticed. This can adversely affect the success of treatment. Another approach is to use open databases in which information on pathogenic mutations for certain genes is collected and verified. <br /><br />The company Parseq Lab is working on a large project to create a panel of markers. It included 37 genes associated with 35 different diseases and 36 external sources. In this paper, a part of the project with single database PNDdb and three genes GCH1, QDPR and PTS is presented. In the course of the work, a tool was implemented that exports the necessary data from the site and presents them in the VCF format. The sensitivity of predictive tools of SIFT and PolyPhen was also assessed. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/nebozhatko_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318207" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Khazars heritage in the world genomes</strong> | University of La Verne<br /><span style="font-size: 20px;"><br /><strong>student:</strong> Yury Orlov<br /><strong></strong><strong>scientific adviser:</strong> Tatiana Tatarinova</span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr">Khazars are a semi-nomadic ethnic group that lived in the second part of the 1 st millennium, occupying a large area north of Caucasus between Black and <br /> Caspian seas. At the end of the 10 th century their state was destroyed and the Khazar Khanate disappeared as suddenly as it rose, without leaving any legacy <br /> except their own funerary mounds. At all times there were pretty much theories and guesses about the Khazars origin and their descendants but it was impossible to make solid conclusions about them. The aim of this project is to find the answers on the questions of Khazars origin and genetic legacy by <br /> analysing ancient DNA (aDNA) extracted from remains of three Khazar representatives. <br /><br /> In course of the project, aDNA sequencing data was processed according to its specific library preparation and degraded nature of aDNA. Reads were <br /> mapped to the HG38 reference genome. It was found that bacterial contamination was more than 75% (typical to aDNA), and by detecting significant amount of C-T transitions on the read ends it was shown that studied DNA is indeed of ancient origin. Using the GATK package we performed the SNP-calling procedure on obtained data and with Admixture tool figured out that the Khazars are a mix of North East Asians, Northern European, Mediterranean and South West Asian populations, as it is expected of a well-mixed group of semi-nomadic people. <br /><br /> As continuation of the work we are collecting different ancient and modern genomes to compare obtained data to them and finally draw a conclusion about the Khazars origin and their descendants in the modern world. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/orlov_final_16122017.pdf" style="" target="_blank">Slides</a></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318208" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Russian Exomes. Part 1</strong>. |Bioinformatics Institute<br /><br /><span style="font-size: 20px;"><strong></strong><strong>students:</strong> Olga Poleshchuk, Ekaterina Izmailova<br /><strong></strong><strong>scientific adviser:</strong> Yury Barbitoff</span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Mutations in protein-coding part of the genome are a cause of numerous different pathologies. Thus, whole exome sequencing (WES) is a commonly used alternative to whole-genome sequencing in medical genetics and health-related studies. Some environmental adaptations also likely arose from changes in protein-coding regions, making exome sequencing a valuable tool for population genetics studies. Several large sequencing consortia (e.g., Exome Aggregation Consostium (ExAC)) have collected data from hundreds of thousands samples of western population, and a lot of research was done using this data. The goal of our project was to develop a pipeline for variant analysis in Russian population, and apply it to ~570 WES samples. <br /><br /> Firstly we created pipeline using Snakemake as one of possible tools for creating workflows. The pipeline receives raw FASTQ files as input and outputs a combined annotated VCF for all samples in a batch. We successfully tested our pipeline, and performed preliminary analysis of variants in a dataset of 570 samples of Russian and CIS ancestry. We observed many novel variants common to samples included in the study, with most of such variants classified as missense mutations, intronic variants, and synonimous substitutions. Thus, we made a very first and preliminary steps in assessing the exome-wide genetic structure of Russian population. Further data aggregation and analysis will help to completely fill the biggest gap on the genetic map of the world. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/poleschuk_izmaylova_161217.pdf" style="" target="_blank">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318209" class="r t-rec t-rec_pb_0" style="padding-bottom:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Speed-efficient data structures for cloudSPAdes</strong> |Center for Algorithmic Biotechnology, St. Petersburg State University<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Evgen Polevikov<br /><strong></strong><strong>scientific advisers:</strong> Anton Bankevich, Ivan Tolstoganov </span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">GemCode technology that was recently introduced by 10X Genomics company is rapidly becoming essential for variant calling, diploid genome assembly and read alignment. The cloudSPAdes algorithm was recently developed in the Center for Algorithmic Biotechnology. The algorithm uses GemCode data to improve metagenome assembly quality. Currently, cloudSPAdes consumes a large amount of computational resources for assembly of complex metagenomes. CloudSPAdes' assembly procedure consists of several stages: on the first stage it constructs assembly graph using procedures which were implemented in already existing metaSPAdes pipeline. Then barcoded reads are aligned to the edges of the graph such that for every edge we get a particular set of barcodes. Every set is represented as a sorted array. Intersection of these sets is computed in order to estimate genome distance between long edges and determine their true ordering. <br /><br /> In this work we adapted probabilistic data structure called containment min hash that allows to improve current procedure of computing of edge intersection. In order to estimate intersection of two sets of barcodes A and B (assume that size of A less than size of B) we first create a bloom filter from the set of a larger size B. Then we take a random sample S from the set of a smaller size A and test every element of S for membership in B using a bloom filter. By that we estimate an intersection of A and B. <br /><br /> In order to benchmark containment min hash against the original sorted array data structure we constructed assembly graph from GemCode library which was sequenced from a mixture of 5 known bacterial species. We selected a set of 1492 edges longer than 5,000 bp from the assembly graph and found an intersection for every ordered pair of these edges using containment min hash and compared it with initial edge intersecting procedure. Our analysis have shown that new algorithm works approximately 6 times faster. Also we have managed to decrease memory consumption: now it is enough to store about 60% of data that initial procedure uses. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/polevikov_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318210" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Protein digestion patterns</strong>|University of North Carolina at Chapel Hill<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Natalia Rodina<br /><strong></strong><strong>scientific advisers:</strong> Popov Konstantin, Dokholyan Nikolay </span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"> <span style="font-size: 18px;">Digestion of the proteins by proteasomes and proteases in cells results in producing a specific repertoire of peptides that can potentially bind to MHC 1 complex 1 and used for triggering immune response against specific cancer cells. Thus, creating a tool for prediction the "peptide profiles" produced by protease cleavage in different types of tissues in normal cells and primary tumor became the aim of the present project. <br /> In the first step, microarray expression data in 19 types of tissues types for normal cells (726 arrays) and primary tumor (1.460 arrays) was collected from the MERAV database. PCA analyzes of the preprocessed expression data showed differences in normal tissue and primary tumor for every tissue type were shown. For every tissue type, expressed genes were selected (values higher then median of quintile normalized data) and only genes expressed in the primary tumor and not presented in the normal tissues were taken into account. For selected genes, the amino acid sequences were parsed from the NCBI protein database. <br /> Information about cleavage sites of human cell proteases was downloaded from the Merops database and for every type of tissue only expressed proteases were selected. <br /> In the next step, a tool for prediction of the peptide profiles was created. As input, the tool takes a selected type of the tissue. Then, from inbuilt database, amino acid sequences of all genes and information about cleavage sites of all proteases expressed in the selected tissue are taken. The tool finds all possible cleavage sites in every protein for every protease and provides information about all peptides created by the cleavage of all proteins in the tissue. <br /> The created tool will provide the possibility to predict peptide profiles for all tissue types and identify peptides that are specific for the cancer cells and can be used for targeted immune therapy. </span><br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/rodina_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318211" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Ti plasmid evolution and horizontal gene transfer</strong> | St. Petersburg State University<br /><br /><span style="font-size: 20px;"><strong></strong><strong>students:</strong> Shikov Anton , Zorin Evgeniy<br /><strong></strong><strong>scientific advisers:</strong> Alexandr Tkachenko, Mikhail Rayko</span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Agrobacterium species contain special sequence named T-DNA in Ti- and Ri-plasmids which can be inserted into plant genome. This feature is widely used in plant biotechnology. However, this insertion can become a stable part of plant genome, thus, Agrobacterium species are able to implement horizontal gene transfer to plant organism that happens quite rare in plant realm. <br /><br />The aim of our work was to detect of new examples of horizontal gene transfer in plants and reconstructing phylogeny and evolution of Ti- and Ri-plasmids. To achieve this goal, we used hmmer tools and analyzed available plant genomes and proteomes. In total, 66 proteomes and 45 genomes were scanned. Extracted hits were further utilized for making multiple alignments and building approximately 700 trees. <br /><br />Unfortunately, we didn't detect any explicit clusterization of plant and bacterial sequences. Nevertheless, during analysis we successfully revealed a brand-new example of horizontal gene transfer in Nicotiana tabacum that has not been described in literature before. Bacterial protein riORF20 from Agrobacterium rhizogenes has two plant homologs. Interestingly, this two proteins are homologous to C- and N-ends of riORF20 respectively. For this reason, we propose DNA recombination in N. tabacum after T-DNA insertion. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/shikov_zorin_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318212" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Enhancement of Export Option for a Genome Mappability Score Estimator</strong> | Bioinformatics Institute<br /><br /><span style="font-size: 20px;"><strong></strong><strong>student:</strong> Skalon Elizaveta<br /><strong></strong><strong>scientific adviser:</strong> Bakin Evgeniy</span></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Mappability is a genome-wide function that indicates whether it is possible for any read to be unambiguously mapped to a given position. Mappability information can be crucial for an interpretation of such experiments as ChIPSeq, SNP-calling etc., where quantitative estimates or confident identification of variations are performed. There is a special metric called Genome Mappability Score (GMS), which quantifies the mappability. GMS measures a weighted probability of mapping certainty in a given place. If the GMS is zero in a given position, many identical reads from different loci may be equally mapped to this region. Otherwise, if the GMS is 100, a read mapped to this position is unique. <br /><br /> In this work, we extended the functionality of fast and sufficiently accurate instrument for the GMS calculation, developed by in Bionformatics Institute in 2016. Firstly, a possibility to get output records in many various formats was provided. It allowed not only Wig and BigWig, but also BED, BigBed and TDF output formats to be supported. Secondly, the runtime of data export was reduced by an implementation of a multiprocessing mode, geometric expansion of arrays and export of GMS track directly to BigWig without wigToBigWig converter. <br /> These improvements made the GMS computation even more convenient and friendly for its users. <br /> <br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/skalon_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318213" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><span data-redactor-style="font-size: 24px;" style="font-size: 24px;"><strong data-redactor-tag="strong">Retention time for identification of natural products</strong> | Carnegie Mellon University</span><br /><br /><span style="font-size: 20px;"><strong>student:</strong> Vladimir Sukhov<br /><strong></strong><strong>scientific adviser:</strong> Alexey Gurevich, Husein Mohimani</span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"> Natural Products (NPs) play an important role in pharmacology: many antibiotics, antiviral and antitumor agents are NPs. Thus, it is crucial to have methods for accurate discovery of new NP. In the process of searching for new NP, false positive identifications may occur. To reduce their number, we use retention time (RT) as an additional correctness check for discovered NP. <br /><br /> In this work, we applied machine learning methods for determining possible RT range for peptides. The multiple regression method was chosen as the primary technique. As a model for machine learning, we considered the amino acid composition of a peptide, where each amino acid adds its own weight to the final RT value. <br /><br /> As the result, the model was trained and tested. Model benchmarking demonstrated high accuracy of RT prediction and its potential for a significant reduction of false positive identifications. <br /><br />Slides</div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318214" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Analysis of VH-replacement statistical properties based on public datasets</strong> | Center for Algorithmic Biotechnology, St. Petersburg State University and Pavlov First St. Petersburg State Medical University<br /><br /><span style="font-size: 20px;"><strong>students:</strong> Adel Gazizova, Anastasia Vinogradova <br /><strong></strong><strong>scientific adviser:</strong> Andrey Slabodkin, Maria Chernigovskaya, Oksana Stanevich</span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">During a construction, immunoglobulin H locus (IgH) undergo a process named VDJ-recombination, during which is random gene segments from IgH germline are set into resulting gene sequence. It provides a primary specificity to antigens. However, infrequently, the existing V-gene can be partly replaced by a new one, and this process is called VH-replacement. There are various hypotheses regarding the contribution that VH-replacement makes to antibody functionality. <br /><br /> In our work, we created a pipeline, which allows to identify VH-replacement in human antibody sequences. First we downloaded the data from Genbank, parsed files and extracted titles, that contained all the information about each sequence. Before starting the search, we divided antibody sequences into clonal families, because our data must contain only clonal-independent sequences in order to exclude a false-positive result. Then by means of developed script we made an exact and inexact (with one possible mismatch) search of VH-replacement's footprints in sequences of people with different phenotypes. We analyzed results and found, that VH-replacement frequency significantly increases for subjects infected with HIV-1, as well as for ones vaccinated against pneumococcus. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/vinogradova_gazizova_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318215" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Web bot development for automation of requests to IMGT / V-quest</strong> | Bioinformatics Institute<br /><br /><span style="font-size: 20px;"><strong></strong><strong>students:</strong> Andrey Zolotarev, Alexandr Cheblokov <br /><strong></strong><strong>scientific adviser:</strong> Evgeniy Bakin</span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">There are a lot of different web-services that give the user an opportunity to work with integrated databases and research tools, which are necessary for a number of scientific areas. <br /><br /> One of them is IMGT® (the international ImMunoGeneTics information system®) – high-quality knowledge resource in immunogenetics and <br /> immunoinformatics that specifically provides data about immunoglobulin or antibodies, T-cell receptors, major histocompatibility (MH) of human and other vertebrate species, immunoglobulin superfamily, MH superfamily and related proteins of the immune system of vertebrates and invertebrates. Unfortunately, this service is difficult to use for implementation statistical analysis due to limitation of loadable <br /> sequences. IMGT allows the user to load only 50 sequences by one request, and the task become further complicated by the need to configure multiple query parameters. <br /><br /> The problem for statistical research is obvious, the scientist must spend a huge amount of time to process even one thousand sequence dataset. <br /><br /> The solution we developed is web-bot that allows the researcher to automate the processing of large amounts of data subject to the limitation given above. We came to the conclusion that a suitable basis for our objective is Selenium Web-Driver. <br /> Selenium is a software library with the open source code, which is widely represented for a number of the most popular web browsers and compatible with such popular programming languages as C#, Python, JavaScript and others. This module emulates user behavior on the site, what allows to set parameters once and then implement the repetition of their setting by Selenium API. <br /> As a result of our work we present the program that automates requests to IMGT for huge datasets and includes an interface for configuring the search parameters that are specific to a particular task. Result of program execution is a table in CSV-format that contains data required for the researcher. <br /> <br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/cheblokov_zolotarev_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318217" class="r t-rec t-rec_pb_0" style="padding-bottom:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Using approximate calculations to speed up the peak calling </strong><strong>procedure </strong>|Bioinformatics institute<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Viacheslav Borovitskiy<br /><strong></strong><strong>scientific adviser:</strong> Evgeniy Bakin</span></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Peak calling is a computational procedure used to identify areas in a genome that have been enriched with aligned reads primarily as a consequence of performing a ChIP-sequencing experiment. There are several popular pieces of software which perform this procedure (most of them require substantial computational resources and time). Each one has its own set of parameters requiring adjustment to the particular experiment. <br /><br /> In this work, we try to address the issue of time costs of the process of parameters adjustment for the peak calling procedure. We present a prototype of a tool that uses some fast machine learning / digital signal processing methods to approximately obtain the result of a peak calling procedure for a given caller with a given set of parameters in a matter of no time. <br /><br /> At first, we use given caller with a given set of parameters on a small piece of data. We then use the results of the previous step to train a linear <br /> classifier (some fast time-series optimized version of logistic regression). <br /><br /> Finally, we apply our trained classifier (followed by some threshold transformation) to the rest of the data to obtain an approximation of the result. <br /><br /> We test our tool against some data sets from the Encyclopedia of DNA Elements (ENCODE). On "good" data we have precision/recall scores at about 0.85/0.85. On "bad" data we have precision/recall scores at about 0.20/0.20. Tests give impression that we never overfit, meaning that precision/recall scores on the train set determine those on the test set. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/borovitskiy_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318218" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Regulatory network modeling based on analysis of ATAC-seq data from cancer cells</strong> | Institut Cochin, Laboratory "Computational Epigenetics of Cancer"<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Anastasia Danchurova<br /><strong>scientific adviser:</strong> Valentina Boeva</span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">The tumor cell state is governed by complicated interplay between transcription factors that regulate gene expression and thus define cell fate. The concept of core, or master, transcription factors comprising Oct4, Sox2, Nanog (also known as Yamanaka's factor family) postulates that small number of transcription factors control the more numerous auxiliary transcription factors and play an essential role in determining of cell fate. Recent data showed that these core transcription factors play a regulative role in different types of cancer. <br /><br /> Because cancer is a disease associated with aberrant gene expression patterns, transcription factors, which serve as the convergence points of oncogenic signaling and are functionally altered in many cancers, hold great therapeutic promise. The more personal this therapy will be the more efficient result it will achieve.<br /> <br /> That is why in this project exactly ATAC-seq data is used. Related to DNAse-seq and MNAse-seq methods, ATAC-seq compares favorably in library preparation simplicity, speed and amount of required cells (500-50 000 cells), what in total makes it the appropriate for clinical usage. <br /><br /> In this project, we create a tool, which combines ATAC-seq data with human genome annotation and several databases, determines interactions between transcription factors and active promoters and enhancers. As a result, we are expecting to construct a graph that will represent all detected interactions. Analysis of such graph is intended to help to determine the main transcription factors that may become effective potential targets for anti-cancer therapy. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/danchurova_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318219" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Regulatory network modeling based on analysis of ATAC-seq data from cancer cells</strong> |<br /> All-Russia Institute for Agricultural Microbiology<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Yury Malovichko<br /><strong>scientific adviser:</strong> Evgeniy Andronov</span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Sinorhizobium meliloti is one of the so-called Rhizobia, a group of α- and β-proteobacteria known for their capability of interacting with legume plants that results in stable mutualistic symbiosis where bacteria provide plants with atmosphere nitrogen reduced to ammonium in exchange for organic carbon. The genome of Rhizobia differs from that of Escherichia coli and other model prokaryotes and comprises of one major chromosome and one or more symbiotic plasmids that determine bacterium's host range and symbiosis efficacy. However, Rhizobia genome is also known for its flexibility, with symbiotic genes rearranged with plasmids, between them or even between plasmids and chromosome. <br /><br /> In this study, we aimed to prove a suggestion based on RFLP and other molecular marker analyses that two distinct genetic lines exist with S. meliloti species discriminated by linkage of particular alleles of leu and betCB genes. We used MLST approach with 10 loci suggested previously for genomic clustering of this species (see Reference) ad Bayesian Inference algorithm to build a tree that would show actual phylogeny of 12 isolates with 6 isolates for both supposed genomic lines, respectively. However, we gained ambiguous results showing that suggested loci are evidently not universal in their use for MLST of S. meliloti. For now, we seek for more informative loci that will shed the light on true phylogeny of <br /> these isolates and existence of these two genomic lines. <br /><br /> 1. Berkum P. Van, Elia P., Eardly B.D. Multilocus sequence typing as an approach for population analysis <br /> of Medicago-nodulating rhizobia // J. Bacteriol. 2006. Т. 188. № 15. С. 5570–5577. <br /><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/malovichko_final_16122017.pdf" style="" target="_blank">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318220" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>NGS-based metagenomic pathogen viruses and bacteria identification system</strong> |<br /> Saint-Petersburg Pasteur Institute<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Alexandr Bebyakov<br /><strong>scientific adviser:</strong> Alexandr Semenov</span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Большинство методов микробиологической диагностики занимают продолжительное время и неприменимы для обнаружения некультивируемых форм патогенных агентов. Предполагается возможным на основе данных секвенирования нуклеотидных последовательностей смешанного образца определять наличие возбудителей особо опасных инфекций и свойственных им факторов патогенности и, таким образом, ускорять процесс принятия решений о мерах противодействия возможным эпидемиям.<br /><br /><a href="https://github.com/ABapmath/2018_spring_SEM" style="" target="_blank">GitHub<br /></a><a href="http://bioinformaticsinstitute.ru/sites/default/files/bebyakov_final_26052018.pdf" target="_blank" style="">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318221" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Applying state-of-the-art neural network architectures for predicting protein-binding sites</strong> |<br /> ITMO University<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Viacheslav Borovitskiy<br /><strong>scientific adviser:</strong> Tatiana Malygina</span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">This is a project aiming to improve an approach proposed in <a href="https://www.ncbi.nlm.nih.gov/pubmed/28575181" style="nofollow">the paper</a> by using some of the modern neural network architectures.<br /><br />GitHub<br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/borovitskiy_26052018.pdf" target="_blank" style="">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318222" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Study and development of a macrophage metabolic model</strong>|ITMO University<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Natalia Rodina, Alexandr Cheblokov<br /><strong>scientific adviser:</strong> Gainullina Anastasia, Sergushichev Alexey</span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Macrophages are cells of the first line of immune protection: destroy pathogens (M1), maintain tissue homeostasis (M2). Using the metabolic FBA model allows you to see the coordination between metabolic pathways at the level of the whole cell. However, the existing FBA model of macrophage metabolism has a number of inaccuracies, and therefore it does not reflect the latest ideas about their M1 activation, formulated during molecular biological experiments. The purpose of this project is to detect and correct inaccuracies of the macrophage FBA model of metabolism.<br /><br /><a href="https://github.com/chebaleksandr/MACROPHAGE_BI_2018" style="" target="_blank">GitHub<br /></a><a href="http://bioinformaticsinstitute.ru/sites/default/files/cheblokov_rodina_26052018.pdf" target="_blank" style="">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318223" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Biogeography of arabidopsis</strong>|University of La Verne<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Anton Eliseev, Kristina Krivonosova <br /><strong>scientific adviser:</strong> Tatiana Tatarinova</span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Цель проекта - с помощью геномов медикаго и арабидопсиса определить корреляции между генетикой, климатом, почвой и прочей географией. Построить модель связывающую окружающую среду и геном.<br /><br /><a href="https://github.com/AntonCoon/biogeography" style="" target="_blank">GitHub<br /></a><a href="http://bioinformaticsinstitute.ru/sites/default/files/eliseev_krivonosova_26052018.pdf" target="_blank" style="">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318224" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Statistical analysis of annotated genomes</strong>|University of La Verne<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Poleshchuk Olga, Danchurova Anastasia <br /><strong>scientific adviser:</strong> Tatiana Tatarinova</span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"> Find correlation between sequence features and functional regions in different genomes <br /><br /> <ol> <li> Plot sequence features such as TFBS, SNPs, methylation, RNA-seq coverage </li> <li> Map it on promoter regions </li> <li> Find correlation </li> <li> Consider outcomes for promoter prediction for complex and not annotated genomes </li></ol><br /><a href="https://github.com/danchurova/Genomes_project" style="" target="_blank">GitHub</a><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/poleshchuk-danchurova-final-260518.pdf" target="_blank" style="">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318225" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Finding novel variations of germline Immunoglobulin genes using WGS data</strong>|University of California San Diego<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Alexandr Ilin<br /><strong>scientific adviser:</strong> Yana Safonova</span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Variety of immunoglobulin germline genes (V, D, and J) is a key component of the antibody repertoire diversity. Highly repetitive structure of Ig loci and a lack of natural selection result in elevated polymorphism rate of immunoglobulin germline genes. In this project, we want to analyze variations of Ig loci in several human populations and describe differences between them.<br /><br /><a href="https://github.com/Serfentum/bioinf_vsegments" style="" target="_blank">GitHub</a><br /><a href="http://bioinformaticsinstitute.ru/sites/default/files/ilin_final_26052018.pdf" target="_blank" style="">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318226" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Long read mapping improvements for Flye assembler</strong>|University of California San Diego<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Evgeny Polevikov<br /><strong>scientific adviser:</strong> Mikhail Kolmogorov</span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"> minimap2 is a versatile pairwise aligner for genomic and spliced nucleotide sequences written in C. The goal of this project is to write C++ wrapper for this tool in order to incorporate it into <a href="https://github.com/fenderglass/Flye">Flye</a>. <br /><br /> In <a href="https://github.com/EvgenyPolevikov/mm2wrapper/blob/master/src/example.cpp">src/example.cpp</a> you can find an usage example of minimap2 API with C++ interface. The example shows how to build an index and how to use this index to find overlaps for pacbio reads. <br /><br />We have recently released Flye assembler for long and noisy reads (PacBio, Oxford Nanopores). The <a href="https://www.biorxiv.org/content/early/2018/01/12/247148" target="_blank" style="">assembly results</a> seems to be very promising in comparison with the current state-of-the-art approaches.<br /><br />As a successor of the ABruijn assembler, Flye uses solid k-mer based approach to find overlaps between noisy reads, which is (relatively) fast, but might be not optimal in terms of memory usage and the parameter choice flexibility. On the other hand, <a href="https://github.com/lh3/minimap2" target="_blank" style="">minimap2</a> seems to be very memory efficient, while showing the best sensitivity/specificity among the other long read aligners. As minimap2 also has C++ API, we want to explore the possibility of replacing our solid k-mer approach with minimap2. We expect that this change will significantly reduce the memory usage bottleneck, while also improving the assembly accuracy.<br /><br /><a href="https://github.com/EvgenyPolevikov/mm2wrapper" style="" target="_blank">GitHub<br /></a><a href="http://bioinformaticsinstitute.ru/sites/default/files/polevikov_final_26052018.pdf" target="_blank" style="">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318227" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Mediation of effects of persistent chemicals on the human sperm epigenome</strong>|A.N. Belozersky Research Institute of Physico-Chemical Biology, Moscow State University, Institute of Bioengineering, Research Center of Biotechnology RAS <br /><br /><span style="font-size: 20px;"><strong>student:</strong> Julia Kornienko<br /><strong>scientific adviser:</strong> Oleg Sergeev, Yulia Medvedeva</span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"> Процессы сперматогенеза и созревания сперматозоидов включают в себя каскад эпигенетических изменений (Wu et al., 2015). Изучение эпигенома сперматозоидов представляет очень перспективное направление по нескольким причинам. Во-первых, явно недостаточно изучено специфическое воздействие разнообразных факторов среды, включая химические факторы, на эпигенетические маркеры. Во-вторых, выявляемые эпигенетические изменения связаны с качеством и количеством сперматозоидов. И, в-третьих, что может быть особенно важно - репрограммирование эпигенома половых клеток может быть передано следующему поколению, что может привести к нарушению развития потомства, как на этапе развития эмбриона, так и в последующей жизни.<br /><br />- Объект заявляемого молекулярно-эпидемиологического исследования – данные метилирования ДНК сперматозоидов (WGBS и RRBS) и данные факторов окружающей среды, полового развития и образа жизни, полученные в ходе родительского проспективного когортного исследования «Russian Children's Health. Male Reproductive SubStudy», начавшегося в 2003 году.<br /><br />- Что известно из родительского исследования?<br />Воздействие таких стойких токсикантов, как диоксины, в течение полового развития (пубертата) влияет как на метилирование ДНК сперматозоидов (Pilsner et al., 2018), так и на снижение качества семени (Minguez-Alarcon et al., 2017) в возрасте 18 лет.<br /><br />- Что неизвестно?<br />Какой вклад вносят другие факторы окружающий среды, потенциально негативно влияющие в процессе сперматогенеза на эпигеном сперматозоидов и качество семени, в частности курение.<br />Какой вклад в изменение эпигенома и качества сперматозоидов вносят различные варианты полового развития (ускоренное, нормальное, замедленное)?<br /><br />- Цель исследования:<br />Изучить роль курения и темпов полового созревания в качестве медиаторов влияния диоксинов на метилирование ДНК сперматозоидов.<br /><br /><a href="https://github.com/yukornienko/mediation_analysis" style="" target="_blank">GitHub<br /></a><a href="http://bioinformaticsinstitute.ru/sites/default/files/kornienko_final_26052018.pdf" target="_blank" style="">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318228" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Role of protein dimerization</strong>|Department of Biochemistry and Biophysics, University of North Carolina at Chapel Hill <br /><br /><span style="font-size: 20px;"><strong>student:</strong> Orlov Iurii<br /><strong>scientific adviser:</strong> Nikolay Dokholyan</span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"> We would like to understand why nature evolves proteins to function as dimers. To understand whether the oligomeric structure of protein is more evolutionary preferable than monomeric.<br /><br /> General plan: <br /> <ol> <li> detect core residues (CR) responsible for structure formation </li> <li> determine how number of CR grows with protein length </li> <li> compare obtained results with dimeric proteins </li> <li> go further for larger oligomers (n-mers) to find the most preferable n </li></ol><br /><a href="https://github.com/troublov/BI_project_spring18" style="" target="_blank">GitHub<br /></a><a href="http://bioinformaticsinstitute.ru/sites/default/files/orlov_31032018.pdf#overlay-context=node/1521" target="_blank" style="">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318229" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Model of the N1 zone formation in human antibodies dimerization</strong>|Bioinformatics institute<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Elena Pazhenkova<br /><strong>scientific adviser:</strong> Evgeniy Bakin, Oksana Stanevich</span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"> The N1-zone is a variable region of human antibodies DNA, formed as a result of VDJ-recombination and providing diversity of antigen binding regions. N1-zone generation is a complicated process including formation of palindroms on 5' and 3' ends and addind up to 20 random nucleotides to 5' and 3' ends with following non-homologous end joining. Thus, length of N1-zone depends of several random events. However, the N1-zone sometimes contains so-called footprints, appeared as a result of VH-replacement and recent studies showed that the length of CDR3 (including V3', N1, D, N2 and J5') is correlated with number of footprints (Meng et al., 2014). In this project we want to figure out whether VH-replacement is a random event by fitting of statistical model of N1-zone formation and estimating its parameters using Maximum Likelihood method.<br /><br /><a href="https://github.com/PazhenkovaEA/VH-replacement-analysis" style="" target="_blank">GitHub<br /></a><a href="http://bioinformaticsinstitute.ru/sites/default/files/pazhenkova_25052018.pdf" target="_blank" style="">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318230" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Search for multiple associations in GWAS data</strong>|Bioinformatics institute<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Shikov Anton<br /><strong>scientific adviser:</strong> Yury Barbitoff</span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"> В 2017 году Биобанк Великобритании сделал крупнейший релиз генетических данных в истории (500,000 человек). Группа Бенджамина Нила произвела быстрый массовый анализ ассоциаций (GWAS) с более чем 2,000 фенотипов, результаты данного анализа были выложены в открытый доступ. С тех пор, огромное количество препринтов, посвященных поиску интересного сигнала в этих данных, появились на сервере bioRxiv. Данный проект посвящен поиску маркеров, обладающих множественными ассоциированными фенотипами, и изучению механизмов, опосредствующих данные эффекты.<br /><br /><a href="https://github.com/anton-shikov/GWAS_project" style="" target="_blank">GitHub<br /></a><a href="http://bioinformaticsinstitute.ru/sites/default/files/shikov_26052018.pdf" target="_blank" style="">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318231" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Identifying differentially expressed transposons across four life-cycle stages of Fasciola hepatica</strong>|Institute of Cytology RAS<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Elisaveta Scalon<br /><strong>scientific adviser:</strong> Anna Soloveva, Nikolay Panyushev</span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"> Проект предполагает поиск мобильных элементов, чей уровень экспрессии варьирует в зависимости от стадии жизненного цикла Fasciola hepatica.<br />У F.hepatica секвенированы транскриптомы всех стадий жизненного цикла, данные доступны в Sequence read archive. Планируется собрать транскриптомы по референсу, выявить последовательности мобильных элементов<br />и определить их уровни транскрипции. На выходе будут получены данные о наличии или отсутствии мажорных транскриптов мобильных элементов, специфичных к отдельным стадиям жизненного цикла F.hepatica.<br /><br /><a href="https://github.com/NickPanyushev/IB_Fasciola" style="" target="_blank">GitHub<br /></a><a href="http://bioinformaticsinstitute.ru/sites/default/files/skalon_final_26052018.pdf" target="_blank" style="">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318232" class="r t-rec" style=" " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_12 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong>Evolution analysis of genes associated with apomixis in Brassicaceae family</strong>|CAB SPbU<br /><br /><span style="font-size: 20px;"><strong>student:</strong> Rostislav Skitchenko<br /><strong>scientific adviser:</strong> Mike Raiko</span><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Goals: <br /> <ul> <li> Perform a comparative phylogenetic assay of the genomes of seven plants.<br /> </li> <li> Find the patterns between specific genes and apomixis plant-forms.<br /> </li> <li> Find orthologous genes in other representatives of the Brassicaceae family.<br /> </li> <li> Build the trees of genes of interest.<br /> </li></ul><br /><a href="https://github.com/rostkick/Boechera" style="" target="_blank">GitHub<br /></a><a href="http://bioinformaticsinstitute.ru/sites/default/files/skithenko_final_260518.pdf" target="_blank" style="">Slides</a></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec540318236" class="r t-rec" style=" " data-animationappear="off" data-record-type="407" ><!-- T407 --><div class="t407"><div class="t-container_100 "><div class="t407__wrapper"><div class="t407__col t407__col_notempty" style="background-color: #fafafa;"><a href="/popularization" target=""> <div class="t407__textwrapper"><div class="t407__uptitle t-uptitle t-uptitle_xs" style="color: #3a81e8;color:#3a81e8;" field="subtitle" ><span style="font-weight: 700;">Previous</span></div> <div class="t407__title t-name t-name_lg" style="color: #3a81e8;" field="title" ><span style="font-weight: 300;">Популяризация</span></div> </div></a> </div><div class="t407__col t407__col_notempty" style="background-color: #f7f7f7;"><a href="/research" target=""> <div class="t407__textwrapper t-align_right"><div class="t407__uptitle t-uptitle t-uptitle_xs" style="color: #3a81e8;color:#3a81e8;" field="subtitle2" ><span style="font-weight: 700;">Next</span></div> <div class="t407__title t-name t-name_lg" style="color: #3a81e8;" field="title2" ><span style="font-weight: 300;">Наука</span></div> </div></a> </div></div></div></div></div><div id="rec540318237" class="r t-rec t-rec_pt_75 t-rec_pb_90" style="padding-top:75px;padding-bottom:90px;background-color:#ffffff; " data-animationappear="off" data-record-type="420" data-bg-color="#ffffff"><!-- T420 --><div class="t420"><div class="t-container t-align_left"><div class="t420__col t-col t-col_3"><a href="#" style="color:#ffffff;font-size:20px;text-transform:uppercase;"><img class="t420__logo t-img" 
src="https://static.tildacdn.com/tild6665-3836-4636-b630-346264663537/-/empty/bi_logo.png" data-original="https://static.tildacdn.com/tild6665-3836-4636-b630-346264663537/bi_logo.png" 
imgfield="img"
style="max-width: 60px;" alt="Об институте"
></a><div class="t-sociallinks"><div class="t-sociallinks__wrapper"><!-- new soclinks --><div class="t-sociallinks__item t-sociallinks__item_facebook"><a href="https://www.facebook.com/bioinforussia" target="_blank" rel="nofollow" style="width: 25px; height: 25px;"><svg class="t-sociallinks__svg" width=25px height=25px viewBox="0 0 100 100" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M50 100c27.6142 0 50-22.3858 50-50S77.6142 0 50 0 0 22.3858 0 50s22.3858 50 50 50Zm3.431-73.9854c-2.5161.0701-5.171.6758-7.0464 2.4577-1.5488 1.4326-2.329 3.5177-2.5044 5.602-.0534 1.4908-.0458 2.9855-.0382 4.4796.0058 1.1205.0115 2.2407-.0085 3.3587-.6888.005-1.3797.0036-2.0709.0021-.9218-.0019-1.8441-.0038-2.7626.0096 0 .8921.0013 1.7855.0026 2.6797.0026 1.791.0052 3.5853-.0026 5.3799.9185.0134 1.8409.0115 2.7627.0096.6912-.0015 1.382-.0029 2.0708.0021.0155 3.5565.0127 7.1128.0098 10.669-.0036 4.4452-.0072 8.8903.0252 13.3354 1.8903-.0134 3.7765-.0115 5.6633-.0095 1.4152.0014 2.8306.0028 4.2484-.0022.0117-4.0009.0088-7.9986.0058-11.9963-.0029-3.9979-.0058-7.9957.0059-11.9964.9533-.005 1.9067-.0036 2.86-.0021 1.2713.0019 2.5425.0038 3.8137-.0096.396-2.679.7335-5.3814.9198-8.0947-1.2576-.0058-2.5155-.0058-3.7734-.0058-1.2578 0-2.5157 0-3.7734-.0059 0-.4689-.0007-.9378-.0014-1.4066-.0022-1.4063-.0044-2.8123.0131-4.2188.198-1.0834 1.3158-1.9104 2.3992-1.8403h5.1476c.0117-2.8069.0117-5.602 0-8.4089-.6636 0-1.3273-.0007-1.9911-.0014-1.9915-.0022-3.9832-.0044-5.975.0131Z" fill="#3c3c50"/></svg></a></div><div class="t-sociallinks__item t-sociallinks__item_twitter"><a href="https://twitter.com/bioinforussia" target="_blank" rel="nofollow" style="width: 25px; height: 25px;"><svg class="t-sociallinks__svg" width=25px height=25px viewBox="0 0 100 100" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M50 100c27.614 0 50-22.386 50-50S77.614 0 50 0 0 22.386 0 50s22.386 50 50 50Zm19.091-63.636c2.046-.227 4.09-.682 5.909-1.591-1.364 2.045-2.954 3.863-5.228 5v1.364c0 13.636-10.227 29.09-29.09 29.09-5.909 0-11.137-1.59-15.682-4.545.909.227 1.59.227 2.5.227 4.773 0 9.318-1.59 12.727-4.318-4.545 0-8.181-2.955-9.545-7.046.455.227 1.137.227 1.818.227.907 0 1.814-.226 2.72-.453l.008-.002c-4.772-.909-8.182-5-8.182-10v-.227a8.196 8.196 0 0 0 4.546 1.364c-2.727-1.819-4.546-4.773-4.546-8.41 0-2.045.455-3.636 1.364-5.226 5.227 6.136 12.727 10.227 21.136 10.682-.227-.682-.227-1.364-.227-2.273A10.184 10.184 0 0 1 59.546 30c2.954 0 5.681 1.136 7.5 3.181 2.5-.454 4.545-1.362 6.59-2.5-.909 2.501-2.5 4.319-4.545 5.683Z" fill="#3c3c50"/></svg></a></div><div class="t-sociallinks__item t-sociallinks__item_vk"><a href="https://vk.com/bioinf " target="_blank" rel="nofollow" style="width: 25px; height: 25px;"><svg class="t-sociallinks__svg" width=25px height=25px viewBox="0 0 100 100" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M50 100c27.614 0 50-22.386 50-50S77.614 0 50 0 0 22.386 0 50s22.386 50 50 50ZM25 34c.406 19.488 10.15 31.2 27.233 31.2h.968V54.05c6.278.625 11.024 5.216 12.93 11.15H75c-2.436-8.87-8.838-13.773-12.836-15.647C66.162 47.242 71.783 41.62 73.126 34h-8.058c-1.749 6.184-6.932 11.805-11.867 12.336V34h-8.057v21.611C40.147 54.362 33.838 48.304 33.556 34H25Z" fill="#3c3c50"/></svg></a></div><div class="t-sociallinks__item t-sociallinks__item_instagram"><a href="https://instagram.com/bioinforussia" target="_blank" rel="nofollow" style="width: 25px; height: 25px;"><svg class="t-sociallinks__svg" width=25px height=25px viewBox="0 0 100 100" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M50 100C77.6142 100 100 77.6142 100 50C100 22.3858 77.6142 0 50 0C22.3858 0 0 22.3858 0 50C0 77.6142 22.3858 100 50 100ZM25 39.3918C25 31.4558 31.4566 25 39.3918 25H60.6082C68.5442 25 75 31.4566 75 39.3918V60.8028C75 68.738 68.5442 75.1946 60.6082 75.1946H39.3918C31.4558 75.1946 25 68.738 25 60.8028V39.3918ZM36.9883 50.0054C36.9883 42.8847 42.8438 37.0922 50.0397 37.0922C57.2356 37.0922 63.0911 42.8847 63.0911 50.0054C63.0911 57.1252 57.2356 62.9177 50.0397 62.9177C42.843 62.9177 36.9883 57.1252 36.9883 50.0054ZM41.7422 50.0054C41.7422 54.5033 45.4641 58.1638 50.0397 58.1638C54.6153 58.1638 58.3372 54.5041 58.3372 50.0054C58.3372 45.5066 54.6145 41.8469 50.0397 41.8469C45.4641 41.8469 41.7422 45.5066 41.7422 50.0054ZM63.3248 39.6355C65.0208 39.6355 66.3956 38.2606 66.3956 36.5646C66.3956 34.8687 65.0208 33.4938 63.3248 33.4938C61.6288 33.4938 60.2539 34.8687 60.2539 36.5646C60.2539 38.2606 61.6288 39.6355 63.3248 39.6355Z" fill="#3c3c50"/></svg></a></div><div class="t-sociallinks__item t-sociallinks__item_youtube"><a href="https://www.youtube.com/user/bioinforussia" target="_blank" rel="nofollow" style="width: 25px; height: 25px;"><svg class="t-sociallinks__svg" width=25px height=25px viewBox="0 0 100 100" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M50 100c27.614 0 50-22.386 50-50S77.614 0 50 0 0 22.386 0 50s22.386 50 50 50Zm17.9-67.374c3.838.346 6 2.695 6.474 6.438.332 2.612.626 6.352.626 10.375 0 7.064-.626 11.148-.626 11.148-.588 3.728-2.39 5.752-6.18 6.18-4.235.48-13.76.7-17.992.7-4.38 0-13.237-.184-17.66-.552-3.8-.317-6.394-2.44-6.916-6.218-.38-2.752-.626-6.022-.626-11.222 0-5.788.209-8.238.7-10.853.699-3.732 2.48-5.54 6.548-5.96C36.516 32.221 40.55 32 49.577 32c4.413 0 13.927.228 18.322.626Zm-23.216 9.761v14.374L58.37 49.5l-13.686-7.114Z" fill="#3c3c50"/></svg></a></div><!-- /new soclinks --><script>t_onReady(function () {
var rec = document.getElementById('rec540318237');
if (!rec) return;
var customImageList = rec.querySelectorAll('img.t-sociallinks__customimg');
Array.prototype.forEach.call(customImageList, function(img) {
var imgURL = img.getAttribute('src');
if (imgURL.indexOf('/lib__tildaicon__') !== -1 || imgURL.indexOf('/lib/tildaicon/') !== -1) {
var xhr = new XMLHttpRequest();
xhr.open('GET', imgURL);
xhr.responseType = 'document';
xhr.onload = function() {
if (xhr.status >= 200 && xhr.status < 400) {
var response = xhr.response;
var svg = response.querySelector('svg');
svg.style.width = '25px';
svg.style.height = '25px';
var figureList = svg.querySelectorAll('*:not(g):not(title):not(desc)');
for(var j = 0; j < figureList.length; j++) {
figureList[j].style.fill = '#3c3c50';
}
img.parentNode.append(svg);
img.remove();
} else {
img.style.visibility = 'visible';
}
}
xhr.onerror = function(error) {
img.style.visibility = 'visible';
}
xhr.send();
}
});
});</script></div></div><div class="t420__text t-descr t-descr_xxs" style="" field="text">© Bioinformatics Institute</div></div><div class="t420__col t-col t-col_3"><div class="t420__title t-name t-name_xs t420__title_uppercase" field="title" style="color: #3c3c50;">Об институте</div><div class="t420__descr t-descr t-descr_xxs" field="descr" style="color: #3c3c50;"><ul> <li><a href="http://bioinf.me/admission" target="_blank" style="">Как поступить</a></li><li><a href="https://bioinf.me/media" target="_blank" style="">Медиа о нас</a></li> <li><a href="http://bioinf.me/press" target="_blank" style="">Фирменный стиль</a></li><li><a href="https://bioinf.me/contacts" style="">Реквизиты</a></li><li><a href="https://bioinf.me/personaldata/gamechangers" style="">Условия конфиденциальности</a></li></ul></div></div><div class="t420__floatbeaker_lr3"></div><div><div class="t420__col t-col t-col_3"><div class="t420__title t-name t-name_xs t420__title_uppercase" field="title2" style="color: #3c3c50;">направления</div><div class="t420__descr t-descr t-descr_xxs" field="descr2" style="color: #3c3c50;"><ul><li><a href="/education" style="">Образование</a></li><li><a href="/research" style="">Наука</a></li><li><a href="/popularization" style="">Популяризация</a></li><li><a href="/community" style="">Сообщество</a></li><li><a href="https://bioinf.me/jobs" style="">Вакансии</a></li><li></li></ul></div></div><div class="t420__col t-col t-col_3"><div class="t420__title t-name t-name_xs t420__title_uppercase" field="title3" style="color: #3c3c50;">Мы в сети</div><div class="t420__descr t-descr t-descr_xxs" field="descr3" style="color: #3c3c50;"><ul><li><a href="https://vk.com/bioinf" target="_blank" style="">ВКонтакте</a></li><li><a href="https://www.facebook.com/bioinforussia" target="_blank" style="">Facebook</a></li><li><a href="https://twitter.com/bioinforussia" target="_blank" style="">Twitter</a></li><li><a href="https://www.youtube.com/user/bioinforussia" target="_blank" style="">Youtube</a></li><li><a href="https://instagram.com/bioinforussia" target="_blank" style="">Instagram</a></li></ul></div></div></div></div></div></div></div><!--/allrecords--><!-- Stat --><script type="text/javascript" data-tilda-cookie-type="analytics">setTimeout(function(){ (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
ga('create', 'UA-25060708-1', 'auto');
ga('send', 'pageview');
}, 2000); window.mainTracker = 'user';</script> <script type="text/javascript">if (! window.mainTracker) { window.mainTracker = 'tilda'; }
setTimeout(function(){ (function (d, w, k, o, g) { var n=d.getElementsByTagName(o)[0],s=d.createElement(o),f=function(){n.parentNode.insertBefore(s,n);}; s.type = "text/javascript"; s.async = true; s.key = k; s.id = "tildastatscript"; s.src=g; if (w.opera=="[object Opera]") {d.addEventListener("DOMContentLoaded", f, false);} else { f(); } })(document, window, '90b11afd574e5a7ae3a15cf961ded7e3','script','https://static.tildacdn.com/js/tilda-stat-1.0.min.js');
}, 2000); </script></body></html>