<!DOCTYPE html><html><head><meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta name="viewport" content="width=device-width, initial-scale=1.0" /> <!--metatextblock--><title>2021/22</title><meta name="description" content="Институт биоинформатики" /> <meta property="og:url" content="https://bioinf.me/rp2122" /><meta property="og:title" content="2021/22" /><meta property="og:description" content="Институт биоинформатики" /><meta property="og:type" content="website" /><meta property="og:image" content="https://static.tildacdn.com/tild3964-3333-4333-b436-323131343763/qX9T8vKOqi8.jpg" /><link rel="canonical" href="https://bioinf.me/rp2122"><!--/metatextblock--><meta name="format-detection" content="telephone=no" /><meta http-equiv="x-dns-prefetch-control" content="on"><link rel="dns-prefetch" href="https://ws.tildacdn.com"><link rel="dns-prefetch" href="https://static.tildacdn.com"><link rel="shortcut icon" href="https://static.tildacdn.com/tild6265-3030-4832-b937-623232356261/favicon.ico" type="image/x-icon" /><!-- Assets --><script src="https://neo.tildacdn.com/js/tilda-fallback-1.0.min.js" charset="utf-8" async></script><link rel="stylesheet" href="https://static.tildacdn.com/css/tilda-grid-3.0.min.css" type="text/css" media="all" onerror="this.loaderr='y';"/><link rel="stylesheet" href="https://ws.tildacdn.com/project29135/tilda-blocks-page33403298.min.css?t=1675802638" type="text/css" media="all" onerror="this.loaderr='y';" /><link rel="preconnect" href="https://fonts.gstatic.com"><link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;500;600;700&subset=latin,cyrillic" rel="stylesheet"><link rel="stylesheet" href="https://static.tildacdn.com/css/tilda-cover-1.0.min.css" type="text/css" media="all" onerror="this.loaderr='y';" /><script type="text/javascript">(function (d) {
if (!d.visibilityState) {
var s = d.createElement('script');
s.src = 'https://static.tildacdn.com/js/tilda-polyfill-1.0.min.js';
d.getElementsByTagName('head')[0].appendChild(s);
}
})(document);
function t_onReady(func) {
if (document.readyState != 'loading') {
func();
} else {
document.addEventListener('DOMContentLoaded', func);
}
}
function t_onFuncLoad(funcName, okFunc, time) {
if (typeof window[funcName] === 'function') {
okFunc();
} else {
setTimeout(function() {
t_onFuncLoad(funcName, okFunc, time);
},(time || 100));
}
}</script><script src="https://static.tildacdn.com/js/tilda-scripts-3.0.min.js" charset="utf-8" defer onerror="this.loaderr='y';"></script><script src="https://ws.tildacdn.com/project29135/tilda-blocks-page33403298.min.js?t=1675802638" charset="utf-8" async onerror="this.loaderr='y';"></script><script src="https://static.tildacdn.com/js/lazyload-1.3.min.js" charset="utf-8" async onerror="this.loaderr='y';"></script><script src="https://static.tildacdn.com/js/tilda-cover-1.0.min.js" charset="utf-8" async onerror="this.loaderr='y';"></script><script src="https://static.tildacdn.com/js/jquery-1.10.2.min.js" charset="utf-8" onerror="this.loaderr='y';"></script><script src="https://static.tildacdn.com/js/tilda-events-1.0.min.js" charset="utf-8" async onerror="this.loaderr='y';"></script><script type="text/javascript">window.dataLayer = window.dataLayer || [];</script><script type="text/javascript">(function () {
if((/bot|google|yandex|baidu|bing|msn|duckduckbot|teoma|slurp|crawler|spider|robot|crawling|facebook/i.test(navigator.userAgent))===false && typeof(sessionStorage)!='undefined' && sessionStorage.getItem('visited')!=='y' && document.visibilityState){
var style=document.createElement('style');
style.type='text/css';
style.innerHTML='@media screen and (min-width: 980px) {.t-records {opacity: 0;}.t-records_animated {-webkit-transition: opacity ease-in-out .2s;-moz-transition: opacity ease-in-out .2s;-o-transition: opacity ease-in-out .2s;transition: opacity ease-in-out .2s;}.t-records.t-records_visible {opacity: 1;}}';
document.getElementsByTagName('head')[0].appendChild(style);
function t_setvisRecs(){
var alr=document.querySelectorAll('.t-records');
Array.prototype.forEach.call(alr, function(el) {
el.classList.add("t-records_animated");
});
setTimeout(function () {
Array.prototype.forEach.call(alr, function(el) {
el.classList.add("t-records_visible");
});
sessionStorage.setItem("visited", "y");
}, 400);
} 
document.addEventListener('DOMContentLoaded', t_setvisRecs);
}
})();</script></head><body class="t-body" style="margin:0;"><!--allrecords--><div id="allrecords" class="t-records" data-hook="blocks-collection-content-node" data-tilda-project-id="29135" data-tilda-page-id="33403298" data-tilda-page-alias="rp2122" data-tilda-formskey="f0696df0d2ceecbcfdc18140c73b1757" data-tilda-lazy="yes"><div id="rec539947579" class="r t-rec" style=" " data-animationappear="off" data-record-type="205" ><!-- cover --><div class="t-cover"
id="recorddiv539947579"
bgimgfield="img"
style="height:70vh; background-image:url('https://static.tildacdn.com/tild6535-3133-4436-b736-396638623732/-/resize/20x/noroot.png');"
><div class="t-cover__carrier" id="coverCarry539947579"data-content-cover-id="539947579"data-content-cover-bg="https://static.tildacdn.com/tild6535-3133-4436-b736-396638623732/noroot.png"data-display-changed="true"data-content-cover-height="70vh"data-content-cover-parallax=""style="height:70vh;background-attachment:scroll; "itemscope itemtype="http://schema.org/ImageObject"><meta itemprop="image" content="https://static.tildacdn.com/tild6535-3133-4436-b736-396638623732/noroot.png"></div><div class="t-cover__filter" style="height:70vh;background-image: -moz-linear-gradient(top, rgba(0,0,0,0.70), rgba(0,0,0,0.80));background-image: -webkit-linear-gradient(top, rgba(0,0,0,0.70), rgba(0,0,0,0.80));background-image: -o-linear-gradient(top, rgba(0,0,0,0.70), rgba(0,0,0,0.80));background-image: -ms-linear-gradient(top, rgba(0,0,0,0.70), rgba(0,0,0,0.80));background-image: linear-gradient(top, rgba(0,0,0,0.70), rgba(0,0,0,0.80));filter: progid:DXImageTransform.Microsoft.gradient(startColorStr='#4c000000', endColorstr='#33000000');"></div><div class="t-container"><div class="t-width t-width_10" style="margin:0 auto;"><div class="t-cover__wrapper t-valign_middle" style="height:70vh; position: relative;z-index:1;"><div class="t182"><div data-hook-content="covercontent"><div class="t182__wrapper"><div class="t182__title t-title t-title_xl" style="" field="title">Research projects</div><div class="t182__descr t-descr t-descr_lg" style="" field="descr">Academic year 2021/22</div><div class="t182__buttons"><div class="t-btnwrapper t182__btnwrapper"><a href="https://drive.google.com/file/d/1UqUumV4fOSqpEl1YoOTp6PjIIF0mVzZd/view" target="_blank" class="t182__btn t-btn t-btn_md " style="color:#ffffff;background-color:#82a1e0;border-radius:30px; -moz-border-radius:30px; -webkit-border-radius:30px;text-transform:uppercase;"><table role="presentation" style="width:100%; height:100%;"><tr><td>Projects Abstracts (PDF)</td></tr></table></a></div><div class="t-btnwrapper t182__btnwrapper"><a href="https://www.elibrary.ru/item.asp?id=49591395" target="_blank" class="t182__btn t-btn t-btn_md " style="color:#000000;background-color:#ffffff;border-radius:30px; -moz-border-radius:30px; -webkit-border-radius:30px;text-transform:uppercase;"><table role="presentation" style="width:100%; height:100%;"><tr><td>RSCI</td></tr></table></a></div></div></div></div></div></div></div></div></div></div><div id="rec541286520" class="r t-rec t-rec_pt_60 t-rec_pb_60" style="padding-top:60px;padding-bottom:60px; " data-record-type="61" ><!-- T051 --><div class="t051"><div class="t-container"><div class="t-col t-col_12 "><div class="t051__text t-text t-text_md" style="" field="text">ISBN: 978-5-7422-7814-6</div></div></div></div></div><div id="rec539947588" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>A transcriptome assembly from fragments of the annelids Pygospio elegans (Spionidae, Annelida) and Arenicola marina (Arenicolidae, Annelida)</b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">Zoological Institute of the RAS</span><br /><br /><strong>student:</strong> <span style="font-weight: 300;">Aleksandr Chen</span><br /><strong><span class="Y2IQFc" data-redactor-tag="span">supervisor</span>: </strong><span style="font-weight: 300;">Elena Novikova<br /></span><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vRzdMFhYpN7J2o53HiMHz41aq5nskBy05DD9QK7DjHvFNDj8JLHTZsFGz-al-c2Rw/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a><a href="https://github.com/chensasha/TanscriptomeAssembly-Annelids" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Annelids, like many other invertebrate animals, replace lost body parts in a process called regeneration. In particular, polychaetes, a class of generally marine annelid worms, are capable of regenerating to some degree. The degree of regeneration varies widely across the taxon. For instance, Pygospio elegans (Spionidae, Annelida) is capable of regenerating both head and tail segments, whereas Arenicola marina (Arenicolidae, Annelida) does not regenerate lost segments. The aim of the project is to assemble transcriptomes of two polychaetes - Pygospio elegans and Arenicola marina, - and to prepare data for further investigation of genes responsible for gradient expression in the body of annelids.<br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947589" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>Transcriptome analysis of Platynereis dumerilii (Nereididae, Annelida) and Pygospio elegans (Spionidae, Annelida) at different stages of anterior and posterior regeneration</b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">Zoological Institute of the RAS</span><br /><br /><strong>student:</strong> <span style="font-weight: 300;">Anna Koroleva</span><br /><strong>supervisor: </strong><span style="font-weight: 300;">Elena Novikova<br /><br /></span><a href="https://docs.google.com/presentation/d/e/2PACX-1vQOkotvQtk_qonfsV5oO3Yg5B4MDzuGiS5E9626LLaGI20bLE6_cchAUQoZhMmDVw/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a><a href="https://github.com/Ann-Krlv/DE_analysis_anelida" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"><span class="Y2IQFc">In the course of the project, it will be necessary to identify genes that are activated and suppressed at each stage of regeneration in the tail and head sites of regeneration compared to point 0. It will also be necessary to determine the belonging of these genes to biological processes and build heat maps that reflect the dynamics of expression of selected conservative genes in the process of regeneration.</span><br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947590" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>"Split" repeat resolution for long reads</b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">Center for bioinformatics and algorithmic biotechnology, SPbSU</span><br /><br /><strong>student:</strong> <span style="font-weight: 300;">Grigoriy Bukley</span><br /><strong>supervisor: </strong><span style="font-weight: 300;">Dmitry Antipov<br /><br /></span><a href="https://docs.google.com/presentation/d/e/2PACX-1vTJUHiA1lwnfqwVh58pyItLt1tW22aLCBOLfSwfmK0C0CHhocGPg-2_n8z-tLTVXw/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a><a href="https://github.com/grgrbu/LJASplit" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">When assembling the genome, de Bruijn graphs built on the basis of reads are used. However, some of the information from the original reads remains unused in the graph. Due to inaccuracies of readings and imperfections of genome assemblers, unresolved repeats occur in the de Bruijn graph. This project implements some approaches for resolving repetitions for the LJA assembler based on Split methods.<br /><ul class="redactor-toolbar" id="redactor-toolbar-5"><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li></ul></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947592" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>Role of Drosophila Chromatin Remodeling Factor CHD1 in regulation of gene expression</b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">NRC "KI" - PNPI</span><br /><br /><strong>students:</strong> <span style="font-weight: 100;">Zhanna</span> <span style="font-weight: 300;">Repinskaia, Alexander Zhuravlev</span><br /><strong>supervisor: </strong><span style="font-weight: 100;">Alexander</span> <span style="font-weight: 300;">Konev<br /><br /></span><a href="https://docs.google.com/presentation/d/e/2PACX-1vTw0dpkDFHaiYkIcU_lECesY7tfz6_w3B83QpIxWzUHIXTumc1_SQXAytewTiiXjQ/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a><a href="https://github.com/ZReplicant/Dmel_CHD1_Spring2022" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">The aim of this project is to study the role of chromatin remodeling factor CHD1 (Chromo-ATPase/Helicase-DNA-binding protein 1) in the in the process of dosage compensation in Drosophila melanogaster. We have previously demonstrated CHD1 is unique among Drosophila chromatin-remodeling factors in terms of its specific recruitment to the male X chromosome suggesting the role in DC. To investigate the specific roles (if any) of the CHD1 in dosage compensation and assess additional functions in regulation of gene expression we sequenced (using an MGI platform) rRNA depleted total RNA from wildtype and Chd1 mutant male and female larvae.The main task of this project is to analyze the data acquired. We will need to perform quality control, alignment and analysis of gene differential expression in the X-chromosome and in autosomes of Drosophila males and females.<br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947593" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>Studying Salmonella gene expression dynamics in response to novobiocin</b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">ITMO University</span><br /><br /><strong>students:</strong> <span style="font-weight: 300;">Semyon Kupriyanov, Valeriia Ladyhina</span><br /><strong>supervisor: </strong><span style="font-weight: 300;">Aleksandr Tkachenko<br /><br /></span><a href="https://docs.google.com/presentation/d/e/2PACX-1vTN6LockOQVzVcYnK3Y6YgNzgZaHhJGKobZGtXrppW7ay8Zkj0qlzHDFqDHwJsGYg/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a><a href="https://github.com/ValeriiaLadyhina/BI_Project_analysis_of_effect_of-novobiocin_on_Salmonella" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">The object of the study is Salmonella enterica, bacterium largely resistant to novobiocin and DNA superspiralization alterations in general. Here we aim to analyze gene expression changes in Salmonella cultures grown on media with varying concentrations of antibiotics and to identify clusters of coexpressing genes. We need to analyze gene expression in several bacterial cultures (several timepoints and concentrations). Identify clusters of coexpressing genes. Characterize clusters form functional point of view.<br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947594" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>Developing best practices for semi-automatic single-cell data annotation</b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">ImmunoMind Inc.</span><br /><br /><strong>students:</strong> <span style="font-weight: 300;">Ivan Semenov, Anton Muromtsev, Vladimir Shitov</span><br /><strong>supervisors: </strong><span style="font-weight: 300;">Daniil Litvinov, Vasily Tsvetkov<br /><br /></span><a href="https://docs.google.com/presentation/d/e/2PACX-1vTlzaQKfkFK8z4lvgU6r75PycbmaJsdbh-onJYB00k4JOic-RSerGhih5m8oZgdiQ/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a><a href="https://github.com/ipsemenov/ImmunoMind_project" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Single-cell sequencing is paving the way for precision medicine. It is the next step towards making precision medicine more accurate. One of the most important step in single-cell data analysis is cell type labeling. This is a very time-consuming process, the automation of which is a task of current interest. The goal of this project is using modern machine learning approaches to build semi-automatic single-cell data annotation tool.<br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947595" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b></b><strong>Structure-based modeling of cysteine and serine disease variants of human proteome</strong> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">Skoltech</span><br /><br /><strong>student:</strong> <span style="font-weight: 300;">Dmitrii Podgalo</span><br /><strong>supervisor: </strong><span style="font-weight: 300;">Petr Popov<br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vSYhMT6_uanjEvvar_LV-NETxJpJI8WbS5HumM9ChdKPGGdLCEKx7fYTp3N7W9R2g/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a><a href="https://github.com/DmitriiPodgalo/POP" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a><br /></span><br /></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">The goal of this project is to model structures of human proteins with disease-associated amino acid substitutions. Two types of amino acid substitutions are selected: X to Cysteine or X to Serine (X is any amino acid residue) – these residues are often used as the attachment points for covalent drugs. <br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947596" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>Determining the effectiveness of momi2 for inferring demographic history in GADMA</b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">ITMO</span><br /><br /><strong>students:</strong> <span style="font-weight: 100;">Kseniya Struikhina, Valentin Mikhalchuk</span><br /><strong>supervisor: </strong><span style="font-weight: 100;">Ekaterina Noskova<br /><br /></span><a href="https://docs.google.com/presentation/d/e/2PACX-1vQqczoTU6dTIuGaScBso8NYBfnou9Lxf6nuHsP5D-_OQOqK2928dY_vkWC-BruPxA/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a><a href="https://github.com/ValyMihalchuk/momi2validation" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Restoration of parameters of demographic history by chromosomes and comparison with real data.<br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947597" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>Application of machine learning methods to approximate demographic history parameters from allele frequency spectrum</b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">ITMO</span><br /><br /><strong>student:</strong> <span style="font-weight: 100;">Elizaveta Gorelkina</span><br /><strong>supervisor: </strong><span style="font-weight: 100;">Ekaterina Noskova</span><br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vTucK0f_bhVhFoiCTuj5F3xZoqzSZQRd1FH8LtYg5NJ7XXsRsW_r48RvKTcg5s48g/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a><a href="https://github.com/lisosoma/ML-for-demographic-inference" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Full-fledged genetic data are not used to output demographic history parameters, as they require a lot of computing resources. Therefore, they use various statistics based on these data. One of these statistics is the allele-frequency spectrum. In the simplest case, it can be represented as a multidimensional tensor (matrix). Existing methods for deriving demographic history parameters (dadi, moments) use local optimization algorithms that work faster for given initial approximations of parameters close to optimal. In this project, it is proposed to apply the simplest machine learning methods for approximate prediction of the parameters of the demographic history of two populations. As machine learning algorithms, a choice is offered: random forest or convolutional neural networks. It is required to generate data, train and validate the selected method on them.<br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947598" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>Genome-wide association search (GWAS) and construction of polygene scales (PRS) for height and weight</b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">Genotek</span><br /><br /><strong>students:</strong> <span style="font-weight: 100;">Mark Zorin, Dmitrii Iliushchenko</span><br /><strong>supervisor: </strong><span style="font-weight: 300;">Alexander Rakitko<br /><br /></span><a href="https://docs.google.com/presentation/d/e/2PACX-1vT0TSC0muzHgL-hhWs55Pz0uIgI0K_IDMG-c-1SAMWKmHJJjChlM6ysEtzj8Sf6Tw/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a><a href="https://github.com/DIliushchenko/GWAS_IB" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Genome-wide association study (GWAS) is a search for single nucleotide polymorphisms (SNPs) on which the human phenotype directly depends. GWAS is often used to identify various diseases or risks of their occurrence. In our study we analyzed the genetic variants of a large cohort of Russians to identify SNPs with significant associations with changes in body mass index.<br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947599" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>In silico modeling of coverage profiles for multiplex target panels<em data-redactor-tag="em"></em></b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">ParSeq Lab</span><br /><br /><strong>student:</strong> <span style="font-weight: 100;">Anastasia Kislova</span><br /><strong>supervisor: </strong><span style="font-weight: 300;">Ivan Pyankov<br /><br /></span><a href="https://docs.google.com/presentation/d/e/2PACX-1vTAOIuS70486Svwm0K19Prm1FiEq3epPBpSzXYDk0lrVkwPbvZ4l8XJUMKrZNAHVA/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a><a href="https://github.com/rusanovaA/Sponge_symbionts" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">The development of multiplex target panels for polymerase chain reaction means that highly specific primers are designed to minimize the number of amplicons for target regions. The panels are obligatory in vitro validated, but in silico validation would improve the existing pipeline. The goal of this project was to adjust existing tool called DegenPrimer for in silico validation of designed target panels and check the output correlation with the real data.<br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947600" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>Pipeline for a targeted gene sequencing panel validation</b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">ParSeq Lab</span><br /><br /><strong>student:</strong> <span style="font-weight: 100;">Maria Lopatkina</span><br /><strong>supervisor: </strong><span style="font-weight: 100;">Tamara Simakova</span><br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vTWheJM7dazWfbzTEDIwt3GFcKUbyb6fpr75yDtt7kK1VKhlnvCW1Btvnm5KOLcSQ/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a><a href="https://github.com/tsimakova/panel-validation" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Targeted sequencing is a rapid and cost-effective way to detect known and novel variants and is widely applied in medicine. Targeted sequencing requires upfront selection and isolation of genes or regions of interest, typically by either PCR amplification or hybridization-based capture methods. It should be recalled that not all regions of interest are suitable for analysis. It is important to be aware of the analytical characteristics and limitations of panels. The goal of the project was to develop a pipeline for validation of targeted gene sequencing panels.<br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947601" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>Clustering Hi-C contact graphs using Graph Neural Networks</b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">Center for Bioinformatics and Algorithmic Biotechnology, SPbSU</span><br /><br /><strong>student:</strong> <span style="font-weight: 100;">Fyodor Velikonivtsev</span><br /><strong>supervisors: </strong>Ivan <span style="font-weight: 300;">Tolstoganov, Anton Korobeynikov<br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vRbx2UoCflNyE6FY8zJS3kUQagYVMft2BWUu51zoN6EDDEGnQzPj1vm0C5bLeBTiA/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a></span><a href="https://github.com/Abusagit/GNN_plus_HiC" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">A typical approach to binning using Hi-C data consists of two steps: constructing Hi-C contact graph, where nodes are contigs and edge weights are normalized number of Hi-C links between them, and Hi-C contact graph clusterization. Existing binning approaches based on Hi-C technology, such as bin3C and HiCBin, are based on community detection algorithms, such as Infomap and Leiden. Recent advances in genome binning approaches implemented in tools such as VAMB clearly shows the potential of neural networks in binning problem solutions. As a result we hope that recent advances in graph clusterization using Graph Neural Networks (GNN) might provide a more generalized and possibly more accurate way to clusterize Hi-C contact maps, which would in turn provide more accurate and complete MAGs and improve further metagenomic analysis.<br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947602" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>Analysis and construction of SARS-CoV-2 neutralizing ligands with extensive Spike binding</b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">Laboratory of Biomolecular NMR, Saint Petersburg State University</span><br /><br /><strong>students:</strong> <span style="font-weight: 100;">Aleksandr Kovalenko, Xenia Sukhanova</span><br /><strong>supervisors: </strong><span style="font-weight: 100;">Olga Lebedenko, Nikolai Skrynnikov<br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vRCMLM9nsX0uYWUcJymuAapilIE6ZTMXx2Zb_lF9uu6jBGku0eBhqlECQcM4w15xw/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a></span><a href="https://github.com/Aleksandr-biochem/covid-spike-binders-analysis" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Protein minibinders (MP1 and MP3) have been designed in silico against RBD of wild-type spike protein to prevent SARS-CoV-2 entry into cells. However, the emergence of new covid strains hinders MPs effective use. The main goal of this project is to develop a workflow for estimation of the MP1 and MP3 proteins binding affinity to RBD of the new SARS-CoV-2 variants (alpha, delta, delta+, omicron) and propose a way to optimize the MP1 and MP3 sequences for stronger interaction with new covid strains RBDs. As a result, the collection of python scripts is provided for structure manipulation, simulations, binding analyses and results processing. MP1/3 interaction with the new SARS-CoV-2 RBD variants is analysed. For MP1 no beneficial mutation were observed, while MP3(D37R) mutant with enhanced binding to delta+ strain RBD is proposed.<br /><br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947603" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>Differential expression analysis of macrophage RNA sequencing data using the Hobotnica tool</b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">Ivannikov Institute for System Programming of the RAS, Information Systems Department</span><br /><br /><strong>students:</strong> <span style="font-weight: 100;">Anton Zhelonkin, Alexandra Belyaeva</span><br /><strong>supervisor:</strong> <span style="font-weight: 100;">Evgeny Karpulevich<br /><br /></span><a href="https://docs.google.com/presentation/d/e/2PACX-1vTYxJW1VVQDUnw-DBIr1zZxEGEvuTgLWVRDhNV1FgSmpPt7f-_MHLUOPUQaSjH8-w/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a><a href="https://github.com/tony-zhelonkin/Hobotnica_baySeq_ed" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Macrophages are a central component of innate immunity and play an important role in host defense. In our work, we study the effect of lipopolysaccharide on macrophages derived from the two most polar (CD14+ and CD16+ monocytes), as well as an intermediate subset of blood monocytes from healthy donors, and evaluate what happens to the subset most prone to polarization at the transcriptomic level. Tools for calculating the differential expression of RNA sequencing data work on the basis of the apparatus of mathematical statistics. In order to choose the most appropriate tool for specific data, it is proposed to use the Hobotnitsa tool.<br /><br /><br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947604" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>Search for homologs of egg-cell specific genes, study of their expression patterns and regulatory elements for the creation of effective constructs for genetic engineering</b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">Skolkovo Institute of Science and Technology, Institute for Information Transmission Problems RAS</span><br /><br /><strong>students: </strong><span style="font-weight: 300;">Elena Grigoreva, Anna Toidze</span><br /><strong>supervisors: </strong><span style="font-weight: 300;">Maria Logacheva, Artem Kasianov<br /><br /></span><a href="https://docs.google.com/presentation/d/e/2PACX-1Fxu7L83m1qDUM84fvsrQN3iwEjaxeRLEy_aMaw5BW5sitjZlMhreWjaQn2wkgRZzWc0QlOc1A/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a><a href="https://github.com/AnnaToi01/EC_genes_BI_Project_2022" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Using germ line cells-specific promoters is effective approach in genome editing. <em>EC1.1</em> and <em>EC1.2</em> are <em>A. thaliana</em> genes from Egg Cell family that are specifically and highly expressed in egg cells. It was shown that using of promoters of these genes significantly improved genome editing. But no similar promoters are known for other plants. Knowing that homologous genes can have similar functions, we supposed that EC homologs could have similar expression patterns and using their promoters could also be effective. So the aim of our project is to find functional analogs of EC genes in different crops and model plants and explore their expression patterns and regulatory elements.<br /><br /><br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947605" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>Research of signaling pathways and transcriptional factors (TF) activity alteration associated with acute myeloid leukemia (AML)</b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">BostonGene</span><br /><br /><strong>students: </strong><span style="font-weight: 100;">Iuliia Ruzhenkova, Ekaterina Osintseva</span><br /><strong>supervisor: </strong><span style="font-weight: 300;">Eleonora Belykh<br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vQ_Rvj3RFtn1MH_LS5RCwP3EJZU5VIoTIr7perFELRDXX4FRUtnkLz9vEpADYmB4Q/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a></span><a href="https://github.com/theCastleBuilder/Acute-myeloid-leukemia/" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">In the bone marrow, transcriptional factors (TFs) control the genes important for normal hematopoiesis maintenance. Dysregulation of TFs activity can lead to hematological malignancies including acute myeloid leukemia (AML). Currently, TFs are considered as promising drug targets, and their research (as well as the signaling pathways responsible for their activation) is relevant for the development of new therapeutic strategies. In the present study, we analyze publicly available NGS data (RNA-seq and scRNA-seq) from the bone marrow of AML patients and healthy donors: after preprocessing the data, we run PROGENy and DoRothEA programs to determine the important molecular pathways and TFs activity alteration. To validate the obtained results the literature research, Kaplan-Meier survival analysis as well as TFs visualization single cells are performed.<br /><br /><br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947606" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>Correlation between DNA sequence and chromatin structure</b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">EPAM Systems</span><br /><br /><strong>students: </strong><span style="font-weight: 100;">Kirill Kirilenko, Ivan Kozlov</span><br /><strong>supervisor: </strong><span style="font-weight: 300;">Gennadii Zakharov<br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vQ8Vl0_C_zWjjrsjTIoHjsfIHC-19qAkCSHzv6cMYHoTOycaxHdcr2UC22Ijhe1vw/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a></span><a href="https://github.com/keyreallkeyrealenko/NAP" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">The goal is to determine whether DNA sequence itself can be a good predictor of 3D nuclear structure.<br /><br /><br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947608" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><span style="color: rgb(0, 0, 0);"><b></b><b>Сlassification of β-arches based on their 3D structure</b> | Saint-Petersburg State University<br /><br /><strong>students: </strong>Rustam Basyrov, Leonid Zhozhikov<br /><strong>supervisor: </strong><strong></strong></span><span style="font-weight: 300;"><span data-redactor-style="color: rgb(0, 0, 0);" style="color: rgb(0, 0, 0);">Stanislav Bondarev</span><br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vSBXPyU3pcyKZrnitmKap1TSBSdSEpZUKTq2rrmJb1Nx5DlDxUakKa-dv7CMvwrQw/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a></span><a href="https://bitbucket.org/stanislavspbgu/fibrils_3d/src/master/" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">β-arches are structural elements of proteins that include two β-strands united by a turn. They are present in the proteins of β-solenoids, as well as amyloid aggregates. In the original article in which the term was proposed, such structures were divided into groups based on the conformation of amino acids at the turn sites, as well as the number of amino acids in them. As part of the project, we analyzed the diversity of 3D β-arch organizations.<br /><br /><br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947609" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>Analysis of differential expression of genes involved in NO-signaling in synucleinopathies</b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">Saint-Petersburg State University</span><br /><br /><strong>students: </strong><span style="font-weight: 300;">Aleksandra Livanova, Anna Kapitonova</span><br /><strong>supervisor: </strong><span style="font-weight: 300;">Stanislav Bondarev<br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vSknKNNe1H0kDvSaJJUDe7Y2EfW3BFKaHrsS4FMvAEEYAIMXHdCaa9NNObskfp79Q/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a></span><a href="https://github.com/anna-kapitonova/RNA_seq_synucleinopathies_NO" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Synucleinopathies are neurodegenerative diseases characterized by aggregation of proteins, in particular, alpha-synuclein, in brain neurons. According to bioinformatic predictions, nitric oxide synthase 1 adaptor protein (NOS1AP) is also capable of forming protein aggregates in neurons. Moreover, it directly interacts with alpha-synuclein. Based on this, the hypothesis emerged, that NO signaling could be involved in the pathogenesis of synucleinopathies. The aim of the project was to evaluate changes in expression level of NOS1AP and other NO-signaling genes in brain samples from patients with synucleinopathies. Four open datasets with raw RNA-seq reads from different brain regions of patients with synucleinopathies were analyzed. Although NOS1AP expression did not change significantly, tissue-specific differential expression of other NO-signaling genes was demonstrated.<br /><br /><br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947610" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><span style="color: rgb(0, 0, 0);"><b></b><b>Analysis of variable evolutionary constraint within a single ORF</b><b></b> | Bioinformatics Institute, Research Center of Medical Genetics<br /><br /><strong>student: </strong>Oksana Kotovskaya<br /><strong>supervisors: </strong></span><span style="font-weight: 300;"><span data-redactor-style="color: rgb(0, 0, 0);" style="color: rgb(0, 0, 0);">Yury Barbitoff, Mikhail Skoblov</span><br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vQvHAH-dOY7xm28MR6K8r4h54l8QmgQCwN3ESbl5Ae2VyQslLavHcTqdiqBm_tq4g/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a></span><a href="https://github.com/ombystoma-young/evolutionary-constraint-within-orf" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"> Genetic variants leading to loss of function are not found in all genes. If a gene is found under selection pressure, protein truncation variants (PTV) are much less common in them (Cassa C., 2017). Most often, such genes have important functions, and a such catastrophic change in the protein leads to various diseases or death (Samocha K., 2014). In this work, we are interested in the case when the division of genes into conservative (that is, under selection) and non-conservative (that is, free from selection) becomes less unambiguous, namely, cases when non-conservative genes are found in relatively conservative genes. This work is devoted to implementation of algorithm to the search for such sequences.<br /> Goal: to estimate the evolutionary conservation of individual regions within single ORF.<br /> The key task to achieve this goal was to implement an algorithm based on the hidden Markov model (HMM), which allows to determine the conservativeness of individual regions of the protein-coding sequence (CDS). <br /><br /><div style="font-size:18px;" data-customstyle="yes"></div></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947611" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><span style="color: rgb(0, 0, 0);"><b></b><b>Analysis of the effects of combinations of single nucleotide polymorphisms within a single codon</b><b></b> | Bioinformatics Institute, Research Center of Medical Genetics<br /><br /><strong>student: </strong>Ekaterina Kravchuk<br /><strong>supervisors: </strong><strong></strong></span><span style="font-weight: 300;"><span data-redactor-style="color: #000000;" style="color: rgb(0, 0, 0);">Yury Barbitoff, Mikhail Skoblov</span><br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vQWOSFPKDWO6bCrtcvcZ_PgPJQc1IO7nTT-3uhIEvPdC2S8bjbRixZnWi29zRxs_Q/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a></span><a href="https://github.com/Kravchuk-Ekaterina/MNVFinder" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Creating a tool that correctly predicts the effects of polymorphisms within a single codon.<div style="font-size:18px;" data-customstyle="yes"></div></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947612" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>Evaluation of the evolutionary conservation of uORFs</b><b></b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">Bioinformatics Institute, Research Center of Medical Genetics</span><br /><br /><strong>student: </strong><span style="font-weight: 100;">Dmitrii Poliakov</span><br /><strong>supervisors: </strong><span style="font-weight: 300;">Yury Barbitoff, Mikhail Skoblov<br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vSSiRp3GlliVSVJ9i_Gzaw_M9vMyWCXrdu8YN11q76HpDgjPcacQeGDUwmqqX68bw/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a></span><a href="https://github.com/Plastron/BI_spring_2022_UTRs" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">The method highlights ORFs that encode functional proteins important for human survival. The idea of the current project is to try to apply the method to upstream open reading frames (uORFs) to find the ones encoding functional products. uORFs are the relatively short ORFS found upstream of the main ORF in eucaryotic mRNAs. image They were lacking attention for a long time because of the paradigm "one mRNA - one protein" for eucaryotic mRNA. Then ribosome profiling emerged, the method which allowed for the mapping of the translating ribosomes on mRNAs. And then it became clear that there are a lot of uORFs on which translation occurs. If the majority of products of the uORFs are functional is still a matter of discussion though. Our goal is to try to find uORFs encoding functional proteins.<div style="font-size:18px;" data-customstyle="yes"></div></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947613" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>Generation of possible single-nucleotide variants with a given effect on protein-coding sequence</b><b></b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">Bioinformatics Institute, Research Center of Medical Genetics</span><br /><br /><strong>student: </strong><span style="font-weight: 100;">Oxana Kolpakova</span><br /><strong>supervisors: </strong><span style="font-weight: 300;">Yury Barbitoff, Mikhail Skoblov<br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vQH-OYxxZfsryNi4bEXGF1c5cdQbLwHCUYQstbgRvJkZ6E6v5Bj-vP9RBAgXduk9w/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a></span><a href="https://github.com/OxanaKolpakova/new_SNP" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Creation of a tool to generate pathogenic and benign SNPs for OMIM genes by substitution of 1 codon nucleotide resulting in the same amino acid substitution.<div style="font-size:18px;" data-customstyle="yes"></div></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947614" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>Identification of genetic variants affecting branchpoints within human introns</b><b></b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">Bioinformatics Institute, Research Center of Medical Genetics</span><br /><br /><strong>students: </strong><span style="font-weight: 100;">Alisa Sergeeva, Irina Veretenenko</span><br /><strong>supervisors: </strong><span style="font-weight: 300;">Yury Barbitoff, Mikhail Skoblov<br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vT_m6Pc0Uc94SilI2Vb2m-j4bTeKKVw10YmiWLykL440LTsOSyvF4AYDavW04-1PA/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a></span><a href="https://github.com/aliserge/branchpoint_prediction" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">The aim of the project is to evaluate the effect of intronic variations on the position and strengh of branchpoints – the sites in introns which guide the splicing process. We used two existing tools: Branchpointer and BPP to predict branchpoint probability in reference human intrones and to identify, how inctonic variations change it. We compared pathogenic variations from ClinVar and high-frequent variations from gnomAD. We observed the similarity of BPP and Branchpointer tools in separating ClinVar and gnomAD databases but also their difference in predition probabilities. We created a new ML predictor which showed a good performance and quality metrics. Next we`re planning to verify this model on the ClinVar and gnomAD datasets and estimate the role of pathogenic variaions in the splicing process.<div style="font-size:18px;" data-customstyle="yes"></div></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947615" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>Dissecting the role of gene expression variability in complex traits</b><b></b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">Bioinformatics Institute</span><br /><br /><strong>student: </strong><span style="font-weight: 100;">Mikhail</span> <span style="font-weight: 100;">Slizen</span><br /><strong>supervisors: </strong><span style="font-weight: 300;">Yury Barbitoff<br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vR_r1b3_RPP_HjkiBk3R-ACmNuCt4U_c8pN3eHTUVtv86FGxd1vUerlDxCowpt8Ig/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a></span><a href="https://github.com/MShtol/expression_variability" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Genome-Wide Association Study (GWAS) is a technique used to look for genome sequence variations that affect the development of complex traits. In recent years, GWAS results have been published for thousands of different traits, including two of the world's largest datasets, UK Biobank and Finngen. It is known that changes in gene expression levels are one of the main mechanisms that determine the small effects of genetic variants detected during GWAS. In this project, we test the hypothesis that not only the level of gene expression, but also the degree of expression variability, is associated with the influence of a gene on complex human traits.<div style="font-size:18px;" data-customstyle="yes"></div></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947616" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>Borgs - new entities in archaeal genomes?</b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">Saint-Petersburg State University</span><br /><br /><strong>students: </strong><span style="font-weight: 100;">Vera Emelianenko, Alexandra Kolodyazhnaya</span><br /><strong>supervisors: </strong><span style="font-weight: 300;">Mikhail Rayko, Lavrentiy Danilov<br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vRwUmfz00hIxVrhPWcpM7WdUa8qpzfI8T78ierogL0Txn0KTSwxIjKVOpbko6d_pg/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a></span><a href="https://github.com/Vera-Emelianenko/BI_project_Borgs" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">In a preprint published on bioarchive in July 2021, authours describe misterious new entities in archeal genomes (Al-Shayeb et al., 2021.). Authours define them them the following way: "We infer that these are a new type of archaeal extrachromosomal element with a distinct evolutionary origin. Gene sequence similarity, phylogeny, and local divergence of sequence composition indicate that many of their genes were assimilated from methane-oxidizing Methanoperedens archaea. We refer to these elements as "Borgs"". We used the Borgs sequences published in this paper along with the open-source metagenomic data to find out what are Borgs, how can they be defined and whether they have representatived in metagenomic data.<br /><br /><br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947617" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>Systematics and classification of plasmids</b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">Bioinformatics Institute</span><br /><br /><strong>students: </strong><span style="font-weight: 100;">Ekaterina Vostokova, Pavel Vychik</span><br /><strong>supervisor: </strong><span style="font-weight: 300;">Mikhail Rayko<br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vT7UDipme_1G9Csk75CKjVVuvFAxWhKVrSnBLPpmaCSZz5JZmHoe0HjW6dcPRMCbg/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a></span><a href="https://github.com/p-vychik/plasmids_classification" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Investigate an approach for plasmid systematics based on Rep-protein sequence, and develop the automatized pipeline for newly sequenced plasmid classification.<br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947618" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>Molecular mechanisms behind the life cycle evolution and speciation in hydroids of the Arctic region</b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">Bionformatic institute, Saint Petersburg State University, Moscow State University, N.K. Koltsov Institute of Developmental Biology</span><br /><br /><strong>student: </strong><span style="font-weight: 100;">Polina Guro</span><br /><strong>supervisors: </strong><span style="font-weight: 100;">Lavrenty</span><strong> </strong><span style="font-weight: 300;">Danilov, Stanislav Kremnyov<br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vQ6dgq7vZHFPzV9kj8yErl7CxEH7SkkdDljEVvfCqKa68PZt8kB7qtOHfIsXUZ_Yw/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides</span></a></span><br /></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">The molecular mechanisms of speciation in hydroids have never been studied, neither has the relationship between the evolution of the life cycle and speciation ever been considered. The hydroid Sarsia lovenii was chosen as the object. Recently, in S. lovenii, breeding season polymorphism has been found to be associated with life cycle polymorphism. Colonies of one morph produce normally developed free-floating medusas, while colonies of the second morph produce attached gonophores - medusoids. The morphs identified represent phenological populations: in the example of S. lovenii, we can observe the initial stage of sympatric speciation. Thus, due to the object we have chosen, we can study the molecular mechanisms of speciation associated with the divergence of sympatric populations in breeding time and associated with the evolution of the life cycle.<br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947619" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>Search for human proteins capable to co-aggregate with SARS-CoV-2 proteins</b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">SPbSU</span><br /><br /><strong>student: </strong><span style="font-weight: 300;">Evgeniia Sevasteeva</span><br /><strong>supervisor: </strong><span style="font-weight: 100;">Stanislav Bondarev</span><br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vQ768AAh7H4GQFo6GdGWiZHQ1j5XRPe9QwPC12YjP5W6AgesQhYbf_T6shDMHCgCg/pub?start=true&amp;loop=true&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a><a href="https://github.com/Appolo-11/Amyloid-structures-analysis" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">To assess the probability of developing human amyloidosis as a result of coaggregation with SARS-CoV-2 proteins. To solve this problem, the AmyloComp program should be used, developed in SPbSU laboratory together with the A.V. Kayava group (University of Montpellier, France).<br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947620" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>Studying complex structural variations in cancer using long reads</b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">NIH / NCI</span><br /><br /><strong>student: </strong><span style="font-weight: 300;">Olga Kalinichenko</span><br /><strong>supervisor:</strong> <span style="font-weight: 100;">Mikhail Kolmogorov<br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vRtakNG6acCkarSoYxOTcJ0Q7tloCILk99fgZuGiJ-OVlbBfJzDo8KsbyT5PgYp7w/pub?start=true&amp;loop=true&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a></span><a href="https://github.com/madshuttlecock/structural-variations.git" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Cancer is driven by genomic changes. Small-scale mutations have been extensively catalogued across various types of cancer using short-read sequencing data. However, it is more difficult to detect large and complex structural rearrangements due to read mapping ambiguities. In this project, we will use long-read sequencing to explore the complex genomic changes in cancer genomes. The approach will build on our experience in computational genomics, graph algorithms and de novo assembly.<br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947621" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>Integration of the ADASTRA database as a novel annotator module in the OpenCRAVAT pipeline</b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">IITP RAS</span><br /><br /><strong>students: </strong><span style="font-weight: 100;">Stepan</span><strong> </strong><span style="font-weight: 300;">Kuznetsov, Mikhail Fofanov, Andrey Suponin</span><br /><strong>supervisor: </strong><span style="font-weight: 100;">Artem</span> <span style="font-weight: 300;">Kasyanov<br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vSUpygVOTj4EVoOwBo_wPRbRRlCnrwlzZHLjj0f8Ei6KQxfP2C3dYiXx8YdHPU8Ww/pub?start=true&amp;loop=true&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a></span><a href="https://github.com/gottalottarock/OpenCravat-Adastra" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Development of an annotator and widget and integration into the open-cravat system.<br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947622" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>Age in gene regulatory networks</b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">SciLifeLab, Stockholm University</span><br /><br /><strong>student: </strong><span style="font-weight: 100;">Yuliya Burankova</span><br /><strong>supervisor: </strong><span style="font-weight: 100;">Erik Zhivkoplias</span><span style="font-weight: 300;"><br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vQEyXzGSurJp1PSntosOnQ7z_MXT5pLBRf2-wqK_j10kL0WfivfCygwlMFgcT2l1g/pub?start=true&amp;loop=true&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a></span><a href="https://github.com/Freddsle/age_patterns" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">The wide availability of system-level gene expression datasets gave rise to a variety of reverse-engineering methods that aim to reconstruct the hidden regulatory gene-gene and gene–protein relationships. Such relationships form a gene regulatory network (GRN) that controls the organismic response to changes in the environment. The GRNs we know are the result of a long biological evolution. With the phylogenomic analysis, it is possible to classify genes based on the oldest species that carry orthologous gene. For protein-protein interaction networks in yeast and human, it was shown that proteins of the same age tend to interact more. The goal of this project is to explore if gene interaction preference for genes with similar age holds in gene regulatory networks, in particular in those that describe direct regulatory interaction (transcription factor-target gene).<br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947623" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><b>Analysis of 5'-isomiR targeting</b> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">National Research University HSE</span><br /><br /><strong>student: </strong><span style="font-weight: 100;">Alexandra Gorbonos</span><br /><strong>supervisor: </strong><span style="font-weight: 100;">Stepan Nersisyan</span><span style="font-weight: 300;"><br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vSj8UV-jqbF4WDaR2WhP5GyIIr8fbXymoXaooJz-CeGKPgBz4aj_rhrCy8PqqLk4A/pub?start=true&amp;loop=true&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a></span><a href="https://github.com/IlonaGA/Analysis-of-5-isomiR-targeting" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">MicroRNAs are short non-coding RNA molecules that post-transcriptionally regulate gene expression. MicroRNA expression has been shown to play an important role in various pathologies, including various types of cancer. Studies have shown that there is variability in the nucleotide sequences at the 5'- and 3'-ends of mature miRNAs - miRNA isoforms. The project analyzes targets of miRNA isoforms in 31 cancers based on mRNA and miRNA sequencing data from The Cancer Genome Atlas (TCGA) project, and proposes a method for assessing the activity of miRNA isoforms.<br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947624" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><strong>Analysis of RecQ involvement in primed adaptation in the type I-E CRISPR-Cas system of Escherichia coli</strong> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">Skoltech</span><br /><br /><strong>student: </strong><span style="font-weight: 100;">Anna Shiriaeva</span><br /><strong>supervisor: </strong><span style="font-weight: 100;">Konstantin Severinov</span><span style="font-weight: 300;"><br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vT_O5p1uGucQdkiz6QfsRasQ_4-9uYsai7uQ_RVDAzyY0Mpf0gXXY3_gsp68JVhZA/pub?start=true&amp;loop=true&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides</span></a></span><br /></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Analysis of primed adaptation efficiency and prespacer generation efficiency in cells with the recQ deletion.<br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947625" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><strong>Benchmark creation for drug-target interaction (DTI) prediction task</strong> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">JetBrains Research</span><br /><br /><strong>student: </strong><span style="font-weight: 100;">Dmitrii Traktirov</span><br /><strong>supervisor: </strong><span style="font-weight: 100;">Ellen Kartysheva</span><span style="font-weight: 300;"><br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vStFawFQxng7q6ystmbLKXMQbELLtWBW8EKRgijuHwaBUqA2FO5rISPrIQ3APZAFQ/pub?start=true&amp;loop=true&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a></span><a href="https://github.com/Lemonnik/BI_2021_JB_benchmark" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Drug-target interaction prediction (DTI) task plays an important role in the drug discovery process, which aims to identify new drugs for biological targets. Automation of prediction will speed up the process of creating new drugs. Now there are many machine learning models that solve this problem, however, due to the presence of a huge number of different datasets and testing protocols, it is difficult to compare different models with each other. And so one unified benchmark is needed.<br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947626" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><strong>Domain based burden analysis in PD genes</strong> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">McGill</span><br /><br /><strong>student: </strong><span style="font-weight: 100;">Artem Kosmin</span><br /><strong>supervisor: </strong><span style="font-weight: 100;">Konstantin Senkevich</span><span style="font-weight: 300;"><br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vTkjPJrhsPEid0NRdcWWTn0qXlFEeCmCbqqK_hNt4xX9Kd_YgIif9Gd_Bp5QoWwIw/pub?start=true&amp;loop=true&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a></span><a href="https://github.com/artemkosmin/Domain-based-burden-analysis-in-PD-genes" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">1) Prepare a list of domains with chromosome position-reference for each PD causing gene (Around 20 familiar plus 80 GWAS). 2) Write script for domain based burden analysis, QC script was prepared during the previous project 3) Project will be done using Terra cloud. Knowledge of R, awk, Linux will be helpful.<br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947627" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><strong>Improving Quality of Epitope Mapping by Deep Learning Methods</strong> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">JetBrains Research</span><br /><br /><strong>student: </strong><span style="font-weight: 100;">Simon Tsirikov</span><br /><strong>supervisor: </strong><span style="font-weight: 100;">Natalia Zenkova</span><span style="font-weight: 300;"><br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vSZBV8Jq6IngY_2je0c48_AJxXxqwh0D1CPnL4gy6FtTb4N3pjynNipeumvKntSjg/pub?start=true&amp;loop=true&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a></span><a href="https://github.com/SimonTsirikov/EpitopeMappingPredictor" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">The task of epitope mapping is to determine the region on the surface of the antigen – a specific type of protein recognized by the organism as malicious – to which an antibody – a special protein produced to destroy the antigen – will attach. To reduce the cost and increase the speed of drug development, computer modeling of this process is used. Existing models cope well with the task when the input is a complex of an antigen and an antibody, but if there are no assumptions about a possible antibody, there is need to work only with an antigen. For this formulation of the problem, previous researchers have maximized the value of the recall metric, while the proposed model, built using the Transformer architecture, gives a higher value of the precision metric, which is more relevant for applied tasks.<br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947628" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><strong>Metagenomic analysis of diversity and properties of bacterial communities of White Sea sponges</strong> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">IGB RAS, Skoltech</span><br /><br /><strong>student: </strong><span style="font-weight: 100;">Anastasiia Rusanova</span><br /><strong>supervisors: </strong><span style="font-weight: 100;">Dmitry Sutormin, Svetlana Dubiley</span><span style="font-weight: 300;"><br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vRCnWP3l9qzhGQ9LzaFpCBUFk0iHpavOGvQqgWrZL6MuBmMg1rLWJ4dvVDNZYTmtQ/pub?start=true&amp;loop=true&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a></span><a href="https://github.com/rusanovaA/Sponge_symbionts" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Sponges (phylum Porifera) form symbiotic relationship with the community of microorganisms. Sponges and their symbionts produce various pharmacologically active substances. These communities differ in taxonomic composition from those of the surrounding seawater. Metagenomic analysis of the microbiome allows to find out the taxonomic diversity and properties of the microbial community. We investigated potential bacterial symbionts of our sponges to found out their specific symbioyic features.<br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947629" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><strong>Potential cancer dependencies in the context of LKB1 loss in non-small cell lung cancer (NSCLC)</strong> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">Clarivate</span><br /><br /><strong>student: </strong><span style="font-weight: 100;">Tatiana Kikalova</span><br /><strong>supervisor: </strong><span style="font-weight: 100;">Aliaksei Holik</span><span style="font-weight: 300;"><br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vRyEHT2mvXNWeEIeTuPN1tpx-Rut_-HoLRpn4kEQjtq0T5l0sk4pgeRowx748rVyQ/pub?start=true&amp;loop=true&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a></span><a href="https://github.com/Tatiana-kik/NSCLC_dependencies_LKB1" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">One of the characteristics of NSCLC is a loss of the tumor-suppressor kinase LKB1 (liver kinase B1). LKB1 is known for its ability to induce apoptosis, regulate cell polarity and differentiation and suppress the growth, invasion, and metastases of tumor cells. Although the inhibition of tumor-suppressors, such as LKB1, gives an advantage in avoiding the apoptosis, is also affects the normal pathways and thus the tumor cells have to rely on alternative means (ways) of survival. This gives us an opportunity to identify effective targets in these alternative pathways that we can inhibit and by this affect only tumor cells without damaging normal tissues.<br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947630" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><strong>Prediction of pathogenicity of genetic variants in Kozak sequences</strong> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">Bioinformatics Institute, Research Center of Medical Genetics</span><br /><br /><strong>student: </strong><span style="font-weight: 100;">Marianna Baranovskaia</span><br /><strong>supervisors: </strong><span style="font-weight: 100;">Yury Barbitov, Michail Skoblov</span><span style="font-weight: 300;"><br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vTihcIqhOI9zNHWKed3BQAQ1gjIDChv7Hn_jvwChSjehNyegUVfhDFAczwmMf2JVA/pub?start=true&amp;loop=true&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a></span><a href="https://github.com/mbnext/BI_spring_Kozak_project" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">Kozak sequence is a consensus nucleotide environment of the start codon in the most of the eukaryotic mRNAs, involved in the translation initiation. Kozak sequence can be different in different mRNAs and is was reported that different Kozak sequences influenced the translation level differently. In 2014, collective of scientists has published the data of direct measurement of translation level for every possible Kozak sequence containing classic AUG start codon and computed the model of influence of the particular nucleotides on particular position in the Kozak sequence on the translation efficiency. It is known that human genome has a lot of variable positions and some of them are annotated as related with some diseases, some of them are referred as benign but the significance of some other genetic variants is uncertain for now. If the variant is located in the protein-coding secuence or other well studied sequence, it can be easy to predict the effect of such variant but if is is located in non-coding sequence the prediction becomes more unreliable. In this project we have tried to combine the data of Kozak sequence efficiency and genetic variants located in the Kozak sequences to predict possible pathogenicity of such variants to improve medical genetic analysis.<br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec539947631" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="color:#3c3c50;" data-customstyle="yes"><strong>Studying the alternative ORFs in genes associated with neurological and psychiatric diseases</strong> <span style="font-weight: 300;">|</span> <span style="font-weight: 300;">McGill</span><br /><br /><strong>student: </strong><span style="font-weight: 100;">Eduard Akhmetgaliev, Ekaterina Kershinskaya</span><br /><strong>supervisor: </strong><span style="font-weight: 100;">Konstantin Senkevich</span><span style="font-weight: 300;"><br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vStzjlH_CRhDguOwJjFRplTH-3qTvHQrtsomzrPb7hAk1TZ4x55Mz94NzPqifnPTg/pub?start=true&amp;loop=true&amp;delayms=3000" target="_blank" style=""><span style="font-weight: 400;">Slides<br /></span></a></span><a href="https://github.com/Elementarno9/AltORFStudy" target="_blank" style=""><span style="font-weight: 400;">GitHub</span></a></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes">OpenVar is the first tool for genomic variant annotation and functional effect prediction supporting deep open reading frame (ORF) annotation and polycistronic annotation of Human, Mouse, Rat and Fruit fly transcripts. OpenVar builds on the well-known and extensively used SNPEff tool (Cingolani et al., 2012), but also offers the possibility to predict variant effect in alternative ORFs as defined in OpenProt (Brunet et al., 2019). The aim of the project is to analyze data from OpenVar results for 16 VCF with GWAS SNPs associated with 7 neurological and 9 psychiatric disorders to make a list of SNPs that are nonpathogenic in the canonical ORF along with altering the alternative ORF with moderate/high impact and affecting expression in brain tissues as candidates for future functionality studies.<br /></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div></div><!--/allrecords--><!-- Stat --><script type="text/javascript" data-tilda-cookie-type="analytics">setTimeout(function(){ (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
ga('create', 'UA-25060708-1', 'auto');
ga('send', 'pageview');
}, 2000); window.mainTracker = 'user';</script> <script type="text/javascript">if (! window.mainTracker) { window.mainTracker = 'tilda'; }
setTimeout(function(){ (function (d, w, k, o, g) { var n=d.getElementsByTagName(o)[0],s=d.createElement(o),f=function(){n.parentNode.insertBefore(s,n);}; s.type = "text/javascript"; s.async = true; s.key = k; s.id = "tildastatscript"; s.src=g; if (w.opera=="[object Opera]") {d.addEventListener("DOMContentLoaded", f, false);} else { f(); } })(document, window, '90b11afd574e5a7ae3a15cf961ded7e3','script','https://static.tildacdn.com/js/tilda-stat-1.0.min.js');
}, 2000); </script></body></html>