<!DOCTYPE html><html><head><meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta name="viewport" content="width=device-width, initial-scale=1.0" /> <!--metatextblock--><title>2022/23</title><meta name="description" content="Институт биоинформатики" /> <meta property="og:url" content="https://bioinf.me/rp2223" /><meta property="og:title" content="2022/23" /><meta property="og:description" content="Институт биоинформатики" /><meta property="og:type" content="website" /><meta property="og:image" content="https://static.tildacdn.com/tild3964-3333-4333-b436-323131343763/qX9T8vKOqi8.jpg" /><link rel="canonical" href="https://bioinf.me/rp2223"><!--/metatextblock--><meta name="format-detection" content="telephone=no" /><meta http-equiv="x-dns-prefetch-control" content="on"><link rel="dns-prefetch" href="https://ws.tildacdn.com"><link rel="dns-prefetch" href="https://static.tildacdn.com"><link rel="shortcut icon" href="https://static.tildacdn.com/tild6265-3030-4832-b937-623232356261/favicon.ico" type="image/x-icon" /><!-- Assets --><script src="https://neo.tildacdn.com/js/tilda-fallback-1.0.min.js" async charset="utf-8"></script><link rel="stylesheet" href="https://static.tildacdn.com/css/tilda-grid-3.0.min.css" type="text/css" media="all" onerror="this.loaderr='y';"/><link rel="stylesheet" href="https://ws.tildacdn.com/project29135/tilda-blocks-page33431278.min.css?t=1696333055" type="text/css" media="all" onerror="this.loaderr='y';" /><link rel="preconnect" href="https://fonts.gstatic.com"><link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;500;600;700&subset=latin,cyrillic" rel="stylesheet"><link rel="stylesheet" href="https://static.tildacdn.com/css/tilda-cover-1.0.min.css" type="text/css" media="all" onerror="this.loaderr='y';" /><link rel="stylesheet" href="https://static.tildacdn.com/css/tilda-cards-1.0.min.css" type="text/css" media="all" onerror="this.loaderr='y';" /><script nomodule src="https://static.tildacdn.com/js/tilda-polyfill-1.0.min.js" charset="utf-8"></script><script type="text/javascript">function t_onReady(func) {
if (document.readyState != 'loading') {
func();
} else {
document.addEventListener('DOMContentLoaded', func);
}
}
function t_onFuncLoad(funcName, okFunc, time) {
if (typeof window[funcName] === 'function') {
okFunc();
} else {
setTimeout(function() {
t_onFuncLoad(funcName, okFunc, time);
},(time || 100));
}
}</script><script src="https://static.tildacdn.com/js/tilda-scripts-3.0.min.js" charset="utf-8" defer onerror="this.loaderr='y';"></script><script src="https://ws.tildacdn.com/project29135/tilda-blocks-page33431278.min.js?t=1696333055" charset="utf-8" async onerror="this.loaderr='y';"></script><script src="https://static.tildacdn.com/js/lazyload-1.3.min.js" charset="utf-8" async onerror="this.loaderr='y';"></script><script src="https://static.tildacdn.com/js/tilda-menu-1.0.min.js" charset="utf-8" async onerror="this.loaderr='y';"></script><script src="https://static.tildacdn.com/js/tilda-cover-1.0.min.js" charset="utf-8" async onerror="this.loaderr='y';"></script><script src="https://static.tildacdn.com/js/jquery-1.10.2.min.js" charset="utf-8" onerror="this.loaderr='y';"></script><script src="https://static.tildacdn.com/js/tilda-cards-1.0.min.js" charset="utf-8" async onerror="this.loaderr='y';"></script><script src="https://static.tildacdn.com/js/tilda-skiplink-1.0.min.js" charset="utf-8" async onerror="this.loaderr='y';"></script><script src="https://static.tildacdn.com/js/tilda-events-1.0.min.js" charset="utf-8" async onerror="this.loaderr='y';"></script><script type="text/javascript">window.dataLayer = window.dataLayer || [];</script><script type="text/javascript">(function () {
if((/bot|google|yandex|baidu|bing|msn|duckduckbot|teoma|slurp|crawler|spider|robot|crawling|facebook/i.test(navigator.userAgent))===false && typeof(sessionStorage)!='undefined' && sessionStorage.getItem('visited')!=='y' && document.visibilityState){
var style=document.createElement('style');
style.type='text/css';
style.innerHTML='@media screen and (min-width: 980px) {.t-records {opacity: 0;}.t-records_animated {-webkit-transition: opacity ease-in-out .2s;-moz-transition: opacity ease-in-out .2s;-o-transition: opacity ease-in-out .2s;transition: opacity ease-in-out .2s;}.t-records.t-records_visible {opacity: 1;}}';
document.getElementsByTagName('head')[0].appendChild(style);
function t_setvisRecs(){
var alr=document.querySelectorAll('.t-records');
Array.prototype.forEach.call(alr, function(el) {
el.classList.add("t-records_animated");
});
setTimeout(function () {
Array.prototype.forEach.call(alr, function(el) {
el.classList.add("t-records_visible");
});
sessionStorage.setItem("visited", "y");
}, 400);
} 
document.addEventListener('DOMContentLoaded', t_setvisRecs);
}
})();</script></head><body class="t-body" style="margin:0;"><!--allrecords--><div id="allrecords" class="t-records" data-hook="blocks-collection-content-node" data-tilda-project-id="29135" data-tilda-page-id="33431278" data-tilda-page-alias="rp2223" data-tilda-formskey="f0696df0d2ceecbcfdc18140c73b1757" data-tilda-lazy="yes" ><div id="rec540355735" class="r t-rec" style="background-color:#ffffff; " data-animationappear="off" data-record-type="230" data-bg-color="#ffffff"><div id="nav540355735" class="t199__header t199__js__header " style="background-color:#ffffff;" data-menu="yes"><div class="t199__holder"><a class="t199__logo" href="http://bioinf.me" style=""><img class="t199__logo-image-one" 
src="https://static.tildacdn.com/tild6464-3064-4237-a433-383539613333/bi_logo.png" 
imgfield="img"
style="max-width: 60px; width: 100%; height: auto;" alt=""><img class="t199__logo-image-two" 
src="https://static.tildacdn.com/tild3261-3534-4537-a238-653361653739/bi_logo_white.png" 
imgfield="img2"
style="max-width: 60px; width: 100%; height: auto;" alt=""><div class="t199__logo-text-mobile t-title" field="title" style=""></div> </a><a class="t199__mmenu-toggler t199__js__menu-toggler" href="#"><button type="button" 
class="t-menuburger t-menuburger_first " 
aria-label="Навигационное меню" 
aria-expanded="false"><span style="background-color:#fff;"></span><span style="background-color:#fff;"></span><span style="background-color:#fff;"></span><span style="background-color:#fff;"></span></button><script>function t_menuburger_init(recid) {
var rec = document.querySelector('#rec' + recid);
if (!rec) return;
var burger = rec.querySelector('.t-menuburger');
if (!burger) return;
var isSecondStyle = burger.classList.contains('t-menuburger_second');
if (isSecondStyle && !window.isMobile && !('ontouchend' in document)) {
burger.addEventListener('mouseenter', function() {
if (burger.classList.contains('t-menuburger-opened')) return;
burger.classList.remove('t-menuburger-unhovered');
burger.classList.add('t-menuburger-hovered');
});
burger.addEventListener('mouseleave', function() {
if (burger.classList.contains('t-menuburger-opened')) return;
burger.classList.remove('t-menuburger-hovered');
burger.classList.add('t-menuburger-unhovered');
setTimeout(function() {
burger.classList.remove('t-menuburger-unhovered');
}, 300);
});
}
burger.addEventListener('click', function() {
if (!burger.closest('.tmenu-mobile') &&
!burger.closest('.t450__burger_container') &&
!burger.closest('.t466__container') &&
!burger.closest('.t204__burger') &&
!burger.closest('.t199__js__menu-toggler')) {
burger.classList.toggle('t-menuburger-opened');
burger.classList.remove('t-menuburger-unhovered');
}
});
var menu = rec.querySelector('[data-menu="yes"]');
if (!menu) return;
var menuLinks = menu.querySelectorAll('.t-menu__link-item');
var submenuClassList = ['t978__menu-link_hook', 't978__tm-link', 't966__tm-link', 't794__tm-link', 't-menusub__target-link'];
Array.prototype.forEach.call(menuLinks, function (link) {
link.addEventListener('click', function () {
var isSubmenuHook = submenuClassList.some(function (submenuClass) {
return link.classList.contains(submenuClass);
});
if (isSubmenuHook) return;
burger.classList.remove('t-menuburger-opened');
});
});
menu.addEventListener('clickedAnchorInTooltipMenu', function () {
burger.classList.remove('t-menuburger-opened');
});
}
t_onReady(function() {
t_onFuncLoad('t_menuburger_init', function(){t_menuburger_init('540355735');});
});</script><style>.t-menuburger {
position: relative;
flex-shrink: 0;
width: 28px;
height: 20px;
padding: 0;
border: none;
background-color: transparent;
outline: none;
-webkit-transform: rotate(0deg);
transform: rotate(0deg);
transition: transform .5s ease-in-out;
cursor: pointer;
z-index: 999;
}
/*---menu burger lines---*/
.t-menuburger span {
display: block;
position: absolute;
width: 100%;
opacity: 1;
left: 0;
-webkit-transform: rotate(0deg);
transform: rotate(0deg);
transition: .25s ease-in-out;
height: 3px;
background-color: #000;
}
.t-menuburger span:nth-child(1) {
top: 0px;
}
.t-menuburger span:nth-child(2),
.t-menuburger span:nth-child(3) {
top: 8px;
}
.t-menuburger span:nth-child(4) {
top: 16px;
}
/*menu burger big*/
.t-menuburger__big {
width: 42px;
height: 32px;
}
.t-menuburger__big span {
height: 5px;
}
.t-menuburger__big span:nth-child(2),
.t-menuburger__big span:nth-child(3) {
top: 13px;
}
.t-menuburger__big span:nth-child(4) {
top: 26px;
}
/*menu burger small*/
.t-menuburger__small {
width: 22px;
height: 14px;
}
.t-menuburger__small span {
height: 2px;
}
.t-menuburger__small span:nth-child(2),
.t-menuburger__small span:nth-child(3) {
top: 6px;
}
.t-menuburger__small span:nth-child(4) {
top: 12px;
}
/*menu burger opened*/
.t-menuburger-opened span:nth-child(1) {
top: 8px;
width: 0%;
left: 50%;
}
.t-menuburger-opened span:nth-child(2) {
-webkit-transform: rotate(45deg);
transform: rotate(45deg);
}
.t-menuburger-opened span:nth-child(3) {
-webkit-transform: rotate(-45deg);
transform: rotate(-45deg);
}
.t-menuburger-opened span:nth-child(4) {
top: 8px;
width: 0%;
left: 50%;
}
.t-menuburger-opened.t-menuburger__big span:nth-child(1) {
top: 6px;
}
.t-menuburger-opened.t-menuburger__big span:nth-child(4) {
top: 18px;
}
.t-menuburger-opened.t-menuburger__small span:nth-child(1),
.t-menuburger-opened.t-menuburger__small span:nth-child(4) {
top: 6px;
}
/*---menu burger first style---*/
@media (hover), (min-width:0\0) {
.t-menuburger_first:hover span:nth-child(1) {
transform: translateY(1px);
}
.t-menuburger_first:hover span:nth-child(4) {
transform: translateY(-1px);
}
.t-menuburger_first.t-menuburger__big:hover span:nth-child(1) {
transform: translateY(3px);
}
.t-menuburger_first.t-menuburger__big:hover span:nth-child(4) {
transform: translateY(-3px);
}
}
/*---menu burger second style---*/
.t-menuburger_second span:nth-child(2),
.t-menuburger_second span:nth-child(3) {
width: 80%;
left: 20%;
right: 0;
}
@media (hover), (min-width:0\0) {
.t-menuburger_second.t-menuburger-hovered span:nth-child(2),
.t-menuburger_second.t-menuburger-hovered span:nth-child(3) {
animation: t-menuburger-anim 0.3s ease-out normal forwards;
}
.t-menuburger_second.t-menuburger-unhovered span:nth-child(2),
.t-menuburger_second.t-menuburger-unhovered span:nth-child(3) {
animation: t-menuburger-anim2 0.3s ease-out normal forwards;
}
}
.t-menuburger_second.t-menuburger-opened span:nth-child(2),
.t-menuburger_second.t-menuburger-opened span:nth-child(3){
left: 0;
right: 0;
width: 100%!important;
}
/*---menu burger third style---*/
.t-menuburger_third span:nth-child(4) {
width: 70%;
left: unset;
right: 0;
}
@media (hover), (min-width:0\0) {
.t-menuburger_third:not(.t-menuburger-opened):hover span:nth-child(4) {
width: 100%;
}
}
.t-menuburger_third.t-menuburger-opened span:nth-child(4) {
width: 0!important;
right: 50%;
}
/*---menu burger fourth style---*/
.t-menuburger_fourth {
height: 12px;
}
.t-menuburger_fourth.t-menuburger__small {
height: 8px;
}
.t-menuburger_fourth.t-menuburger__big {
height: 18px;
}
.t-menuburger_fourth span:nth-child(2),
.t-menuburger_fourth span:nth-child(3) {
top: 4px;
opacity: 0;
}
.t-menuburger_fourth span:nth-child(4) {
top: 8px;
}
.t-menuburger_fourth.t-menuburger__small span:nth-child(2),
.t-menuburger_fourth.t-menuburger__small span:nth-child(3) {
top: 3px;
}
.t-menuburger_fourth.t-menuburger__small span:nth-child(4) {
top: 6px;
}
.t-menuburger_fourth.t-menuburger__small span:nth-child(2),
.t-menuburger_fourth.t-menuburger__small span:nth-child(3) {
top: 3px;
}
.t-menuburger_fourth.t-menuburger__small span:nth-child(4) {
top: 6px;
}
.t-menuburger_fourth.t-menuburger__big span:nth-child(2),
.t-menuburger_fourth.t-menuburger__big span:nth-child(3) {
top: 6px;
}
.t-menuburger_fourth.t-menuburger__big span:nth-child(4) {
top: 12px;
}
@media (hover), (min-width:0\0) {
.t-menuburger_fourth:not(.t-menuburger-opened):hover span:nth-child(1) {
transform: translateY(1px);
}
.t-menuburger_fourth:not(.t-menuburger-opened):hover span:nth-child(4) {
transform: translateY(-1px);
}
.t-menuburger_fourth.t-menuburger__big:not(.t-menuburger-opened):hover span:nth-child(1) {
transform: translateY(3px);
}
.t-menuburger_fourth.t-menuburger__big:not(.t-menuburger-opened):hover span:nth-child(4) {
transform: translateY(-3px);
}
}
.t-menuburger_fourth.t-menuburger-opened span:nth-child(1),
.t-menuburger_fourth.t-menuburger-opened span:nth-child(4) {
top: 4px;
}
.t-menuburger_fourth.t-menuburger-opened span:nth-child(2),
.t-menuburger_fourth.t-menuburger-opened span:nth-child(3) {
opacity: 1;
}
/*---menu burger animations---*/
@keyframes t-menuburger-anim {
0% {
width: 80%;
left: 20%;
right: 0;
}
50% {
width: 100%;
left: 0;
right: 0;
}
100% {
width: 80%;
left: 0;
right: 20%;
}
}
@keyframes t-menuburger-anim2 {
0% {
width: 80%;
left: 0;
}
50% {
width: 100%;
right: 0;
left: 0;
}
100% {
width: 80%;
left: 20%;
right: 0;
}
}</style> </a><div class="t199__mmenu t199__js__menu"><nav class="t199__menu" 
aria-label="Основная навигация"><ul role="list" class="t199__menu-items t-menu__list"><li class="t199__menu-item-wrap"><a class="t199__menu-item t-title t-menu__link-item" 
href="/education/program#details" data-menu-submenu-hook="" 
>ПРОГРАММЫ</a></li><li class="t199__menu-item-wrap"><a class="t199__menu-item t-title t-menu__link-item" 
href="/education/program#timeline" data-menu-submenu-hook="" 
>ПОСТУПЛЕНИЕ</a></li><li class="t199__menu-item-wrap"><a class="t199__menu-item t-title t-menu__link-item" 
href="/education/program#courses" data-menu-submenu-hook="" 
>ПРЕДМЕТЫ</a></li><li class="t199__menu-item-wrap"><a class="t199__menu-item t-title t-menu__link-item" 
href="/education/program#instructors" data-menu-submenu-hook="" 
>ПРЕПОДАВАТЕЛИ</a></li><li class="t199__menu-item-wrap"><a class="t199__menu-item t-title t-menu__link-item" 
href="/education/projects"
target="_blank" data-menu-submenu-hook="" 
>ПРОЕКТЫ</a></li><li class="t199__menu-item-wrap"><a class="t199__menu-item t-title t-menu__link-item" 
href="/community/graduates"
target="_blank" data-menu-submenu-hook="" 
>ВЫПУСКНИКИ</a></li><li class="t199__menu-item-wrap"><a class="t199__menu-item t-title t-menu__link-item" 
href="/education/program/materials"
target="_blank" data-menu-submenu-hook="" 
>МАТЕРИАЛЫ</a></li><li class="t199__menu-item-wrap"><a class="t199__menu-item t-title t-menu__link-item" 
href="/education/faq"
target="_blank" data-menu-submenu-hook="" 
>FAQ</a></li></ul></nav><div class="t199__lang"><a class="t199__lang-item" style="" href="https://bioinf.me/en/education/">En</a></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t199_showMenu', function () {
t199_showMenu('540355735');
});
t_onFuncLoad('t199_positionHeader', function () {
t199_positionHeader('540355735');
});
t_onFuncLoad('t_menu__interactFromKeyboard', function () {
t_menu__interactFromKeyboard('540355735');
});
t_onFuncLoad('t_menu__highlightActiveLinks', function () {
t_menu__highlightActiveLinks(".t199__menu-item");
});
t_onFuncLoad('t_menu__findAnchorLinks', function () {
t_menu__findAnchorLinks('540355735', '.t199__menu-item');
});
});</script><style>@media screen and (max-width: 1024px) {
}</style><style>#rec540355735 .t-menu__link-item{
-webkit-transition: color 0.3s ease-in-out, opacity 0.3s ease-in-out;
transition: color 0.3s ease-in-out, opacity 0.3s ease-in-out; 
}
#rec540355735 .t-menu__link-item.t-active:not(.t978__menu-link){
color:#3a81e8 !important; }
#rec540355735 .t-menu__link-item:not(.t-active):not(.tooltipstered):hover {
opacity: 0.9 !important;}
#rec540355735 .t-menu__link-item:not(.t-active):not(.tooltipstered):focus-visible{
opacity: 0.9 !important;}
@supports (overflow:-webkit-marquee) and (justify-content:inherit)
{
#rec540355735 .t-menu__link-item,
#rec540355735 .t-menu__link-item.t-active {
opacity: 1 !important;
}
}</style></div><div id="rec540355736" class="r t-rec" style=" " data-animationappear="off" data-record-type="205" ><!-- cover --><div class="t-cover"
id="recorddiv540355736"
bgimgfield="img"
style="height:70vh; background-image:url('https://static.tildacdn.com/tild3334-6563-4839-a630-643034353465/-/resize/20x/aa_logo.png');"
><div class="t-cover__carrier" id="coverCarry540355736"data-content-cover-id="540355736"data-content-cover-bg="https://static.tildacdn.com/tild3334-6563-4839-a630-643034353465/aa_logo.png"data-display-changed="true"data-content-cover-height="70vh"data-content-cover-parallax=""style="height:70vh;background-attachment:scroll; "itemscope itemtype="http://schema.org/ImageObject"><meta itemprop="image" content="https://static.tildacdn.com/tild3334-6563-4839-a630-643034353465/aa_logo.png"></div><div class="t-cover__filter" style="height:70vh;background-image: -moz-linear-gradient(top, rgba(0,0,0,0.70), rgba(0,0,0,0.80));background-image: -webkit-linear-gradient(top, rgba(0,0,0,0.70), rgba(0,0,0,0.80));background-image: -o-linear-gradient(top, rgba(0,0,0,0.70), rgba(0,0,0,0.80));background-image: -ms-linear-gradient(top, rgba(0,0,0,0.70), rgba(0,0,0,0.80));background-image: linear-gradient(top, rgba(0,0,0,0.70), rgba(0,0,0,0.80));filter: progid:DXImageTransform.Microsoft.gradient(startColorStr='#4c000000', endColorstr='#33000000');"></div><div class="t-container"><div class="t-width t-width_10" style="margin:0 auto;"><div class="t-cover__wrapper t-valign_middle" style="height:70vh; position: relative;z-index:1;"><div class="t182"><div data-hook-content="covercontent"><div class="t182__wrapper"><div class="t182__title t-title t-title_xl" style="" field="title">Research projects</div><div class="t182__descr t-descr t-descr_lg" style="" field="descr">Academic year 2022/23</div></div></div></div></div></div></div></div></div><div id="rec544999860" class="r t-rec t-rec_pt_30 t-rec_pb_30" style="padding-top:30px;padding-bottom:30px; " data-record-type="397" ><!-- t397 --><div class="t397"><div class="t-container"><div class="t397__col t-width t-width_10"><ul class="t397__wrapper t-align_center" 
role="tablist" data-tab-current="1"><li role="presentation" 
class="t397__tab t397__tab_active t397__width_50" data-tab-rec-ids="640558760,640602302,640627857,640439907" data-tab-number="1" 
style="border-bottom: 1px solid #eee;;"><button 
type="button" 
class="t397__title t-name t-name_xs" 
style="font-weight:400;" 
id="tab1_544999860" 
role="tab" 
aria-selected="false" 
aria-controls="rec640558760" 
tabindex="-1" 
field="title">Проекты программы по биоинформатике</button></li><li role="presentation" 
class="t397__tab t397__width_50" data-tab-rec-ids="545000636,548237132,548237123,548237113,548236589,548250724" data-tab-number="2" 
style="border-bottom: 1px solid #eee;;"><button 
type="button" 
class="t397__title t-name t-name_xs" 
style="font-weight:400;" 
id="tab2_544999860" 
role="tab" 
aria-selected="false" 
aria-controls="rec545000636" 
tabindex="-1" 
field="title">Проекты программы по биостатистике</button></li></ul><div class="t397__wrapper_mobile"><div class="t397__firefoxfix"></div><select autocomplete="off" class="t397__select t-name"><option value="640558760,640602302,640627857,640439907">Проекты программы по биоинформатике</option><option value="545000636,548237132,548237123,548237113,548236589,548250724">Проекты программы по биостатистике</option></select></div></div></div></div><style>#rec544999860 .t397__tab_active {
border-bottom-color: #222 !important;
}
#rec544999860 .t397__tab_active .t397__title {
color: #222 !important;
font-weight: 600 !important;
}
#rec544999860 .t397__select {
border: 1px solid #222;
color: #222;
}
#rec544999860 .t397__wrapper_mobile:after {
border-color: #222 transparent transparent transparent;
}
#rec544999860 .t397__firefoxfix {
top: 1px;
bottom: 1px;
right: 1px;
}
#allrecords [aria-labelledby$="544999860"]:focus-visible {
outline-color: #2015FF;
outline-offset: 2px;
outline-style: auto;
}</style><script>t_onReady(function () {
t_onFuncLoad('t397_init', function () {
t397_init('544999860');
});
});</script></div><div id="rec640558760" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-animationappear="off" data-record-type="585" ><!-- T585 --><div class="t585"><div class="t-container"><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion1_640558760" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__0456404399070"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><strong></strong><span style="font-weight: 700;">Amino acid transfer learning for antibody binding prediction</span>| <span style="font-weight: 100;">Amsterdam University Medical Centers</span><br /><br /><span style="font-weight: 700;">student:</span> <span style="font-weight: 100;">Natalia Khotkina</span><br /><span style="font-weight: 700;">supervisor:</span> <span style="font-weight: 100;">Dasha Balashova</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1aeLAHGlvO429fUQEEzi5hEbiH-vTelCNf1m1AaPO70M/edit#slide=id.g23f6cf35bb4_0_15" target="_blank" style="">Slides</a><br /><a href="https://github.com/NatashaKhotkina/Amino_acid_transfer_learning_for_antibody_binding_prediction" target="_blank" style=""><span style="font-weight: 100;">Github</span></a></span></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion1_640558760"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__0456404399070"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">In (Taft et al., 2021) there are limitations on sets of amino acid substitutions at specific positions. Machine learning / deep learning (ML/DL) models can't predict some cases on validation data if such cases are absent in training data. The use of properties and vector representations of sequences can be helpful in overcoming such limitations. Research question: can we use transfer-learning techniques for protein sequences and test ML/DL predictions with some amino acid (AA) positional patterns that are absent in the training dataset?<ol></ol><ol></ol></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion2_640558760" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694779528041"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><strong></strong><span style="font-weight: 700;">Predicting SARS-CoV-2 antibody escape with machine learning</span>| <span style="font-weight: 100;">Amsterdam University Medical Centers</span> <br /><br /><span style="font-weight: 700;">students:</span> <span style="font-weight: 100;">Vadim Smirnov</span><br /><span style="font-weight: 700;">supervisor:</span> <span style="font-weight: 100;">Dasha Balashova</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1yFZ98H1eWS6g7HPxOSYWm8pU0N-qK6rCUFuRPCpnGDc/edit?usp=sharing" style="" target="_blank">Slides</a><br /></span><a href="https://github.com/Fulmenius/Predicting-antibody-escape-with-ML" style="" target="_blank"><span style="font-weight: 100;">Github</span></a></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion2_640558760"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694779528041"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">In (Taft et al., 2021) there are limitations on sets of amino acid substitutions at specific positions. Machine learning / deep learning (ML/DL) models can't predict some cases on validation data if such cases are absent in training data. The use of properties and vector representations of sequences can be helpful in overcoming such limitations. Research question: can we use transfer-learning techniques for protein sequences and test ML/DL predictions with some amino acid (AA) positional patterns that are absent in the training dataset?<ol></ol><ol></ol></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion3_640558760" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694779530009"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><strong></strong><span style="font-weight: 700;">Analysis of the genetic changes effect on the manifestation of the prion-like factor [MCS +] in Saccharomyces cerevisiae strains</span>| <span style="font-weight: 100;">Department of Genetics and Biotechnology, St. Petersburg State University</span><br /><br /><span style="font-weight: 700;">student:</span> <span style="font-weight: 100;">Sidorenko Oksana</span><br /><span style="font-weight: 700;">supervisor:</span> <span style="font-weight: 100;">Lavrentii Danilov, Alexandr Rubel</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1RJun1_K0OdkTF1Rxx7f6OYghn5jkRozcRUupiXAxy04/edit?usp=share_link" style="" target="_blank">Slides</a><br /></span><a href="https://github.com/CyanusCentaurea/Yeast_genome_project" style="" target="_blank"><span style="font-weight: 100;">Github</span></a></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion3_640558760"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694779530009"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">This study aims to assemble the genomes of two strains, assess the presence of possible chromosomal rearrangements, find and analyze single nucleotide polymorphisms (SNPs) unique to each strain.<ol></ol><ol></ol></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion4_640558760" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694779531880"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><strong></strong><span style="font-weight: 700;">Antigen-aware antibody design with encoder-decoder architectures</span>| <span style="font-weight: 100;">BIOCAD</span><br /><br /><span style="font-weight: 700;">student:</span> <span style="font-weight: 100;">Georgii Sarapulov</span><br /><span style="font-weight: 700;">supervisor:</span> <span style="font-weight: 100;">Natalia Zenkova</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1weXVdoY6sqy5zCPRSNbvn0-BgFHDVRwUE9Rd8u-otMA" style="" target="_blank">Slides</a><br /></span><a href="https://github.com/norsage/a3d" style="" target="_blank"><span style="font-weight: 100;">Github</span></a></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion4_640558760"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694779531880"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes"><em>De novo</em> design of antibodies binding to specific antigens has enormous significance for developing therapeutic treatments for various diseases. We propose a data-driven approach supported by available antigen-antibody complex structures from the SAbDab database.<ol></ol><ol></ol></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion5_640558760" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694779533062"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><strong></strong><span style="font-weight: 700;">Analysis of developmental gene expression dynamics during anterior and posterior regeneration of Platynereis dumerilii (Nereididae, Annelida) and Pygospio elegans (Spionidae, Annelida)</span>| <span style="font-weight: 100;">Zoological Institute of the Russian Academy of Sciences</span><br /><br /><span style="font-weight: 700;">students:</span> <span style="font-weight: 100;">Anastasiia Dudkovskaia, Fomenko Sergey</span><br /><span style="font-weight: 700;">supervisors:</span> <span style="font-weight: 100;">Elena Novikova, Maksim Nesterenko</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/10ZU6M8NkPYYVRCNApuEuYVPQMiR5YDuBdkItgm5pG5M/edit?usp=sharing" style="" target="_blank">Slides</a><br /></span><a href="https://github.com/mintuit/BI_2023_Annelida_Regeneration/" style="" target="_blank"><span style="font-weight: 100;">Github</span></a></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion5_640558760"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694779533062"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">Regeneration is a natural process that occurs daily in the body, but it can also be triggered as a response to traumatic injuries, such as amputation or autotomy. After such an injury, the process of regeneration is complex and involves several overlapping events. In annelids, there are differences between anterior and posterior regeneration, including the size of the regeneration buds, the origin of blastema cells, duration, segmentation patterns, and gene expression. Because of their diverse regenerative abilities, annelids are an ideal model for studying and comparing regenerative mechanisms [1]. We focus on two Annelida, Pygospio elegans, capable of anterior and posterior regeneration, and Platynereis dumerilii, capable of only posterior regeneration. Given the homeobox genes play a key role in the anteroposterior axis patterning and organs morphogenesis, we aimed to study the expression pattern of this superfamily of transcription factors during anterior and posterior regeneration <br /><ol></ol><ol></ol></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion6_640558760" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694779534381"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><strong></strong><span style="font-weight: 700;">Classification of brain activity using Synolitic networks</span>| <span style="font-weight: 100;">University College London, National Research University Higher School of Economics</span><br /><br /><span style="font-weight: 700;">student:</span> <span style="font-weight: 100;">Daniil Vlasenko</span><br /><span style="font-weight: 700;">supervisors:</span> <span style="font-weight: 100;">Alexey Zaikin, Denis Zakharov</span><br /><br /><span style="font-weight: 100;"><a href="https://drive.google.com/file/d/1B20R46opP6t8zw25W7qfyX9SEcmIDkkx/view?usp=sharing" style="" target="_blank">Slides</a><br /></span><a href="https://github.com/Daniil-Vlasenko/BI_Progect" style="" target="_blank"><span style="font-weight: 100;">Github</span></a></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion6_640558760"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694779534381"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">By now, methods of representing fMRI data in the form of graphs have already been used in the tasks of classifying brain activity. But rather simple classical methods, such as Pearson's linear correlation or partial correlation, are used to reflect relationships between brain regions in such graphs. Although these methods reveal the correlation of features, such correlation, by itself, does not provide useful information for classification. In this paper, we propose to eliminate this drawback and refine the methods of data representation. Thus, the aim of this work is to propose and test a method of representing fMRI data in the form of graphs that would reflect information meaningful for subsequent classification about the relationships between brain regions. We have named this method synolithic, as it was inspired by synolithic networks, which allow the application of graph analysis methods to multidimensional complex data. Here we will consider the simplest case, in which we will distinguish between only two brain modes, or, in other words, when the task is binary classification.<ol></ol><ol></ol></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion7_640558760" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694779535481"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><strong></strong><span style="font-weight: 700;">Comparative transcriptomic analysis of the abdominal ganglia of the hermit crabs Pagurus minutus, non-infected and parasitised by Peltogaster reticulata ("Crustacea": Rhizocephala)</span>| <span style="font-weight: 100;">All-Russia Research Institute for Agricultural Microbiology</span><br /><br /><span style="font-weight: 700;">students:</span> <span style="font-weight: 100;">Anastasia Lianguzova, Elizaveta Gafarova, Darya Golofeeva</span><br /><span style="font-weight: 700;">supervisor:</span> <span style="font-weight: 100;">Maksim Nesterenko</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1fdvFlChGYvV4cWIo85fQBM66i9gYXkpvyZgR37IygLQ/edit?usp=drivesdk" style="" target="_blank">Slides</a><br /></span><a href="https://github.com/anlianguzova/BI_project_reticulata/" style="" target="_blank"><span style="font-weight: 100;">Github</span></a></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion7_640558760"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694779535481"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">Rhizocephala are highly specialised parasites singular among Metazoa. These parasites infect various marine crustaceans, mostly Decapoda. The body of an adult female parasite is divided into two functional parts: externa, a sac-like reproductive formation carried outside the host's body, and interna, a system of tubular rootlets located in the host's haemocoel. The interna rootlets rarely cause mechanical damage to the host's tissues and organs. However, the rootlets penetrate the host's nervous ganglia, forming specialised sites of contact with the nervous system. This interaction with the host organism allows rhizocephalans to alter many aspects of infected individuals' biology, including morphology, physiology, and behaviour. Infection with rhizocephalans leads to feminisation of the male hosts, the most striking demonstration of which is an appearance of behavioural patterns typical for the females bearing offspring. Unfortunately, the data regarding the molecular mechanisms underlying this host-parasite interaction is limited. <br /></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion8_640558760" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694779536426"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><strong></strong><span style="font-family: &quot;Open Sans&quot;;"><span style="font-weight: 700;" data-redactor-style="font-weight: 700;">Comparison of genotype-to-phenotype relations in mouse and human</span></span>| <span style="font-weight: 100;">Bioinformatics Institute</span><br /><br /><span style="font-weight: 700;">students:</span> <span style="font-weight: 100;">Polina Bogaichuk, Nadezhda Pavlova</span><br /><span style="font-weight: 700;">supervisor:</span> <span style="font-weight: 100;">Yury Barbitoff</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1PAbbUZXu1zdMcRK2Up5Oavg5jRZJ0YWV7vUA4HKvtEk/edit#slide=id.p1" style="" target="_blank">Slides</a><br /></span><a href="https://github.com/pavlovanadia/genotype_to_phenotype" style="" target="_blank"><span style="font-weight: 100;">Github</span></a></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion8_640558760"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694779536426"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">Our knowledge of the aetiologies underpinning hereditary phenotypic features has greatly and quickly evolved due to the use of model organisms, enabling a comparison between genetic variants and associated phenotypes observed in these organisms with corresponding associations in humans [1]. However, the translation of the results of genetic studies conducted on model organisms, such as mice, to humans does not show high reproducibility comparison to genes with phenotypes described and thus presents difficulties due to simplifying gene-phenotype associations [2]. The present study aimed to investigate the concordance between the gene-phenotype associations in human and mouse, analyzing the correspondence of traits associated with orthologous genes as well as patterns of pleiotropy in both organisms. The Human Phenotype Ontology (HPO) [3] and the Mouse Phenotype Ontology (MPO) [4] are standardized vocabularies that describe phenotypic abnormalities caused by genetic variations. Due to their hierarchical structure, these ontologies contain both lower-level, specific terms representing observable traits and higher-level, broader terms encompassing related specific phenotypes. By using these ontologies and available genetic and phenotypic data from HPO, as well as Mouse Genome Informatics (MGI) [5], we aimed to find common and species-specific connections between genes and traits. <br /></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion9_640558760" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694779537579"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><strong></strong><span style="font-weight: 700;">Cell types annotation of cycling T lymphocytes in the tumor microenvironment</span>| <span style="font-weight: 100;">Medical University of Vienna</span><br /><br /><span style="font-weight: 700;">students:</span> <span style="font-weight: 100;">Kristina Zheltova, Anastasiia Mikhailichenko</span><br /><span style="font-weight: 700;">supervisor:</span> <span style="font-weight: 100;">Sergey Isaev</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1pcF9Be7AmeHbFLMtLLrGkLlY7dSyQBBbkrEX2iELRIU/edit#slide=id.g247ffa9a033_0_28" style="" target="_blank">Slides</a><br /></span><a href="https://github.com/serjisa/cycling_T" style="" target="_blank"><span style="font-weight: 100;">Github</span></a></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion9_640558760"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694779537579"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">Single-cell transcriptomics is a powerful tool for analyzing the tissues of living organisms. One of its applications is the analysis of the population of T-lymphocytes of people suffering from various diseases, in particular cancer. Based on the analysis of single-cell transcriptomes of T-cells infiltrating tumors, it is possible to make assumptions about progress of the disease and the appropriate therapy for the patient, as well as to study the dynamics of the development of the disease in the course of fundamental research. From this point of view, it is especially important to know which groups of lymphocytes are cycling. However, the signal of cycling stage markers in such cells is stronger than the signal of cell type markers. When defining clusters, сycling cells usually form a separate cluster, which makes it impossible to tell which cell type the cycling cell belongs to. <br /> The main goal of this work is to develop a method for determining the types of cycling cells and its application for the analysis of T cells from the tumor microenvironment. <br /> We have prepared 3 classification pipelines and to validate their performance we used data from single-cell sequencing of T cell receptors (TCR). <br /> At a certain stage of maturation of a T-lymphocyte, its T cell receptors mature; during this maturation, the receptor gene sequence undergoes recombination and mutational changes, due to which it becomes unique in the hypervariable CDR3 locus [1]. Subsequently, all descendants of a particular lymphocyte can be determined from this unique sequence. The descendants of a single lymphocyte are called a clonotype. It was noted that most of the cells of the same clonotype belong to the same cell type, but some of them may belong to the group of dividing cells. Determining clonotypes in a group of dividing cells and finding representatives of the same clonotypes in clusters of certain cell types, we can reliably tell which cell type the dividing cells belong to. We used these properties to validate the pipelines we developed. <br /></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion10_640558760" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694779539038"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><strong></strong><span style="font-weight: 700;">Development of a tool for analysis of protein families</span>| <span style="font-weight: 100;">Institute of Science and Technology Austria (ISTA)</span><br /><br /><span style="font-weight: 700;">students:</span> <span style="font-weight: 100;">Anna Berdichevskaia, Evgenii Kovalenko</span><br /><span style="font-weight: 700;">supervisor:</span> <span style="font-weight: 100;">Stepan Ovchinnikov</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1PqgNXKCSoXCGcrfEiB-eB_MyZ4KdzeZoSAHDii8mXVQ/edit?usp=share_link" style="" target="_blank">Slides</a><br /></span><a href="https://github.com/stovc/pegp/" style="" target="_blank"><span style="font-weight: 100;">Github</span></a></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion10_640558760"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694779539038"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">Much of the investigation in bioinformatics is related to the phylogenetic reconstruction of evolutionary history of homologous proteins and domains among various groups of organisms. However, these investigations have traditionally relied on manual execution of necessary programs that are time-consuming and often subject to human biases. Meanwhile, multiple studies are associated with similar approaches. In particular, Darius Kazlauskas et al. (2020) investigated evolution of B-family DNA polymerases using approaches similar to the study of AAA+ ATPases (Iyer et al., 2004). Additionally, such a tool might be useful for large-scale studies in which took place the search for homologues and the identification of structural motifs and partial genomic analysis, for instance investigation of DNA polymerase III α subunits evolution (Timinskas et al., 2014) and research of phylogenetic relationship among plant class I Clp ATPases (Singh et al., 2010). <br /> These studies rely on similar methods for the reconstruction of protein (domain) evolution; hence, the Protein evolutionary genomics pipeline (PEGP) aims to address these challenges by automating the process and integrating multiple analysis steps into a single tool. By combining essential functionalities, PEGP enables researchers to save time and effort while obtaining accurate and comprehensive results. <br /></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion11_640558760" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694779540074"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><strong></strong><span style="font-weight: 700;">dipQUAST: quality assessment tool for diploid genome assemblies</span>| <span style="font-weight: 100;">Helmholtz Institute for Pharmaceutical Research Saarland (HIPS)</span><br /><br /><span style="font-weight: 700;">students:</span> <span style="font-weight: 100;">Alexandra Kasianova, Amina Ibragimova</span><br /><span style="font-weight: 700;">supervisor:</span> <span style="font-weight: 100;">Alexey Gurevich</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1RmvBmZUG-Q1aJD11ayQkpFitx34VjE2ijkhh9YPvd2A/edit#slide=id.p1" style="" target="_blank">Slides</a><br /></span><a href="https://github.com/ablab/quast/tree/dipquast" style="" target="_blank"><span style="font-weight: 100;">Github</span></a></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion11_640558760"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694779540074"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">This work is based on the existing QUAST (Quality Assessment Tool for Genome Assemblies) tool [4]. This tool assesses the quality of genome assembly for haploid data. However, QUAST does not take into account the copy number of the genome and considers each chromosome independently. Consequently, this quality assessment is inaccurate for diploid genomes. Thus, the aim of the study was to adapt QUAST for diploid reference genomes and two main formats of diploid genome assemblies: phased and consensus.</div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion12_640558760" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694779541525"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><strong></strong><span style="font-weight: 700;">Antibacterial action of amyloidogenic peptide R23I on proteome of T. thermophilus</span>| <span style="font-weight: 100;">Institute of Protein Research</span><br /><br /><span style="font-weight: 700;">student:</span> <span style="font-weight: 100;">Anna Malykhina</span><br /><span style="font-weight: 700;">supervisor:</span> <span style="font-weight: 100;">Galzitskaya Oxana</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1tPQ7F-tSzOYX9cxsAzkHbZI7deeInqYVCzjSEMgTQZQ/edit?usp=sharing" style="" target="_blank">Slides</a><br /></span><a href="https://github.com/anet1223/Project" style="" target="_blank"><span style="font-weight: 100;">Github</span></a></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion12_640558760"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694779541525"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">The massive use of antibiotics has led to the emergence of bacterial resistance, which traditional antibiotics development, high cost and long-time, is failed to overcome (1). Researches worldwide try to find new ways to deal with this problem, one of which is studying antimicrobial peptides (AMPs). AMPs are an important part of the natural immune defense mechanism of most organisms against pathogens (2). It has been reported that AMPs have an excellent broad-spectrum antibacterial and low biological toxicity (3). Of particular interest are AMPs, which, in addition to evident antimicrobial properties, also tend to self-assembly, form supramolecular complexes and induce amyloidogenesis (4). Despite traditional negative view on amyloidogenic properties as contributors to the neurodegenerative and progressive metabolic diseases, nowadays AMPs with such properties become acknowledged to have an impact on pathogenic bacteria and viruses (5). <br /> The amyloidogenic peptide R23I was selected from a number of peptides due to its pronounced antibacterial effect. The aim of the project was to determine the antibacterial mechanism of action of this substance based on changes in the proteomic profile according to mass spectrometry. <br /><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes"></div></div></div></div></div></div></div><div class="t585__border" style="height: 1px; background-color: #adadba"></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t585_init', function () {
t585_init('640558760');
});
});</script><style> #rec640558760 .t585__title { font-size: 24px; color: #3c3c50; font-family: 'Open Sans'; } #rec640558760 .t585__text { font-size: 20px; color: #3c3c50; font-family: 'Open Sans'; }</style></div><div id="rec640602302" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-animationappear="off" data-record-type="585" ><!-- T585 --><div class="t585"><div class="t-container"><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion1_640602302" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__2612175426420"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 700;">Interactions in multi-layer biological networks</span>| Bioinformatics Institute<br /><br /><span style="font-weight: 700;">student:</span> <span style="font-weight: 100;">Ekaterina Petrenko</span><br /><span style="font-weight: 700;">supervisor: </span><span style="font-weight: 100;">Yury Barbitoff</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1iCsXIFZtLJT6q05qNhvddBIBEhOSSTpl89PDAZwp4oM/edit" target="_blank" style="">Slides</a><br /><a href="https://github.com/uglymiracle/Project_BI/tree/main" target="_blank" style="">Github</a></span></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion1_640602302"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__2612175426420"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">Biological interactions are often represented in the form of networks enhancing the analysis of the molecular organization of cells and organisms. For example, protein-protein interactions (PPI), genetic interactions (GI), and gene coexpression (CX) data are commonly represented as networks. Although single-layer networks are widely used, there are very few studies that have involved several types of interactions, the so-called multilayer networks. <br /> However, such an analysis could allow us to consider the organism as a complex system. In this study, we analyzed multilayer networks for two organisms with different levels of complexity, yeast (Saccharomyces cerevisiae) and human (Homo sapiens). This project aims to analyze the correspondence between different network layers and identify gene modules with conserved interactions in several layers of the network. <br /></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion2_640602302" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694784619740"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 700;">Machine Learning for molecules: Pre-training in Single-step Retrosynthesis</span>| <span style="font-weight: 100;">University of Stuttgart</span><br /><br /><span style="font-weight: 700;">student:</span> <span style="font-weight: 100;">Aigul Nugmanova</span><br /><span style="font-weight: 700;">supervisor: </span><span style="font-weight: 100;">Nina Lukashina</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1Q2M57JDTke9dOwMJAvWbEBh6j6BVkr766d9zAAhtnfg/edit?usp=sharing" target="_blank" style="">Slides</a><br /><a href="https://github.com/ANugmanova/graphretro" target="_blank" style="">Github</a></span></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion2_640602302"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694784619740"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">Retrosynthesis is one of the most important tasks in organic chemistry. It helps chemists plan the synthesis of complex organic molecules in an efficient manner [1]. The one-step retrosynthesis task involves searching for reactants for the target product in a single reaction. A well-established approach divides the task into 2 stages. At first, we look for the place where the connection was formed. At the second, we compose the final reactants by adding a leaving group. In the project, we focused on the first step of this approach and investigated ways to improve it by using transfer learning. <br /> In many machine learning fields, pre-trained models have become the standard [2, 3]. However, there were several works [4, 5] that analyze different ways of pre-training on graph molecules and then fine-tuning on several chemical tasks and showed that there was no gain from pre-training. The main assumption why this may not work is that the pre-training process learns and converge quite quickly. Another hypothesis is that usually pre-training does not take into account the 3D structure of the molecule, which may contain useful information. Therefore, in our work, we decided to try 2 recent models that learn to encode information about the 3D structures of the molecule, and due to this, the tasks for pre-training become more complicated. <br /></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion3_640602302" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694784622252"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 700;">Meat wars: omics for antibacterial production safety</span>| <span style="font-weight: 100;">Institute of Biomedical Chemistry</span><br /><br /><span style="font-weight: 700;">students:</span> <span style="font-weight: 100;">Svetlana Tarbeeva, Anna Kozlova</span><br /><span style="font-weight: 700;">supervisor: </span><span style="font-weight: 100;">Ekaterina Ilgisonis</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/15DIDQvOU-cv6dnSV4cs__6WHgGMZYTrDdRqOdfKOsF4/edit?usp=drivesdk" target="_blank" style="">Slides</a><br /><a href="https://github.com/Martiansunflower/BI_Listeria_project" target="_blank" style="">Github</a></span></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion3_640602302"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694784622252"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">The problem of resistance of pathogenic bacteria to disinfectants in meat and poultry processing plants threatens the health and well-being of consumers, so increasingly, researchers are turning to sequencing data from biofilms obtained from food production facilities. Listeria monocytogenes is a pathogenic bacterium commonly found in food processing environments, particularly in meat production facilities. Its ability to form biofilms on various surfaces poses a significant risk for food contamination and subsequent foodborne illnesses. To better understand the factors contributing to biofilm growth and the development of resistance, the application of advanced sequencing technologies such as Oxford Nanopore Technologies (ONT) sequencing has emerged as a valuable tool. In this study, we aimed to investigate the growth of biofilms containing Listeria monocytogenes. <br /></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion4_640602302" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694784623547"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 700;">Somatic mutation calling validation using in silico approach</span>| <span style="font-weight: 100;">BostonGene</span><br /><br /><span style="font-weight: 700;">student:</span> <span style="font-weight: 100;">Gleb Khegai</span><br /><span style="font-weight: 700;">supervisors: </span><span style="font-weight: 100;">Anastasiia Yudina, Danil Stupichev</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1dNnKSAN0jQM4Ujiv-OcsupxeEnzkb_OIbkfU7bpEycI/edit?usp=sharing" target="_blank" style="">Slides</a><br /><a href="https://github.com/tlebchan/insilico-validation" target="_blank" style="">Github</a></span></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion4_640602302"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694784623547"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">There are many tools for analyzing WES (whole exome sequencing) data that can detect germline and somatic mutations, copy number changes, and other events in tumor and normal tissue samples. However, it is important not only to find the event, but also to prove whether the evidence is true. For these purposes, there is a procedure for the validation of a clinical test. As part of it, several technical and biological replicas of test samples are usually analyzed to check the assay for accuracy, sensitivity, robustness and reproducibility. When using model cell lines for validation, information about genomic events is usually obtained from previous studies collected in databases (COSMIC), or established during pre-validation experiments. <br /></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion5_640602302" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694784625255"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 700;">Host-pathogen interaction in Fusarium infection of flax</span>| <span style="font-weight: 100;">Centre for Computational Biology, Peter the Great St.Petersburg Polytechnic University</span><br /><br /><span style="font-weight: 700;">student:</span> <span style="font-weight: 100;">Vladislav Stanin</span><br /><span style="font-weight: 700;">supervisor: </span><span style="font-weight: 100;">Anastasia Samsonova, Alexandr Kanapin</span> <br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1VIZQnPoGoP1h_-FGAqonR2lRmXRDeUStvPsJq_kyftI/edit?usp=sharing" target="_blank" style="">Slides</a><br /><a href="https://github.com/vladissta/Fusarium_Flax" target="_blank" style="">Github</a></span></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion5_640602302"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694784625255"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 100;">Fusarium wilt of flax is an aggressive disease caused by soil-borne fungal pathogen Fusarium oxysporum f. sp. lini (FOLINI) posing a major threat to flax production worldwide. It invades the plant through roots and spreads inside the vascular bundle. After germination, it develops microconidia, blocking water and nutrient flow, which leads to plant wilt and death. The fungus produces mycotoxins and enzymes hydrolyzing cell-wall components that facilitate tissue penetration. Most domesticated flax varieties are either highly or moderately resistant to Fusarium wilt. However, a rapid decrease of genetic diversity of flax cultivars and the host-pathogen arms race contribute to increased risk of developing disease. <br /></span></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion6_640602302" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694784626636"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 700;">Transfer learning approach for prediction of protein-protein binding affinity</span>| <span style="font-weight: 100;">Laboratory of Biomolecular NMR, St. Petersburg State University</span><br /><br /><span style="font-weight: 700;">students:</span> <span style="font-weight: 100;">Anastasiia Kazovskaia, Mikhail Polovinkin</span><br /><span style="font-weight: 700;">supervisors: </span><span style="font-weight: 100;">Olga Lebedenko, Nikolai Skrynnikov</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1uU9UDU7E3RJV7oZvN7g8izLeGBq1z95WkqN_wZv0UPg/edit#slide=id.g24661c9a189_0_0" target="_blank" style="">Slides</a><br /><a href="https://github.com/Chicky-Picky/kd-prediction" target="_blank" style="">Github</a></span></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion6_640602302"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694784626636"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">Binding affinity is the strength of the binding interaction between a biomolecule and its binding partner (ligand), which is typically reported in the form of free binding energy, ΔGbind, or otherwise in the form of equilibrium dissociation constant, Kd = exp(ΔGbind/RT). Predicting accurate binding affinity of protein-protein complexes is a fundamental challenge in the area of protein science and drug discovery involving protein-based drugs. Over the last three decades, various computational methods to predict Kd have been developed based on free energy perturbation scheme, empirical scoring, and force-field potentials. [1-6] Nevertheless, all of these physics-based methods fail to accurately predict ΔGbind for diverse datasets. We expect that deep learning models can be used to obtain more reliable predictions in agreement with the experimental data. <br /></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion7_640602302" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694784627323"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 700;">Prediction of clinical response status to melanoma immunotherapy based on gut metagenomic data</span>| <span style="font-weight: 100;">Scientific Research Institute of Physical-Chemical Medicine</span><br /><br /><span style="font-weight: 700;">students:</span> <span style="font-weight: 100;">Mikhail Filippov, Tatiana Ziubko</span><br /><span style="font-weight: 700;">supervisor: </span><span style="font-weight: 100;">Evgenii Olekhnovich</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/121so363HBf_qWYcRUVtT4vy2sa3-jaJxFzi8A9GUtws/edit#slide=id.p1" target="_blank" style="">Slides</a><br /><a href="https://github.com/quasicephalus/Melanoma_microbiome" target="_blank" style="">Github</a></span></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion7_640602302"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694784627323"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">Identification of microbial signatures associated with positive therapy response will allow us to propose novel promising candidates for enhancing melanoma immunotherapy. Here we developed Machine learning-based predictive models utilizing gut metagenomic data to identify clinical response status in melanoma immunotherapy; microbial signatures as well as taxonomic assignments associated with it were also identified. <br /></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion8_640602302" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694784629123"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 700;">RNA-seq Analysis of Senescent Mesenchymal Stem Cells</span>| Intitute of Cytology RAS<br /><br /><span style="font-weight: 700;">student:</span> <span style="font-weight: 100;">Lyublinskaya Ekaterina</span><br /><span style="font-weight: 700;">supervisor: </span><span style="font-weight: 100;">Julia Ivanova</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1mydpPFMrWJo-n00DHSmzst6_qOHfyi6S2zoiMUDftAc/edit?usp=sharing" target="_blank" style="">Slides</a><br /><a href="https://github.com/kate-lyubl/senescence_project" target="_blank" style="">Github</a></span></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion8_640602302"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694784629123"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">Advancements in medical science have improved the lives of individuals with diseased or injured organs. However, organ transplantation from donors has limitations, including the time-consuming search for suitable donors and the risk of organ rejection. Regenerative medicine offers a promising alternative by developing biological substitutes to restore and maintain normal tissue function.<br />Endometrial mesenchymal stem cells (eMSCs) are clonogenic cells derived from the endometrium, a regenerative tissue in women. This study focused on replicative senescence of eMSCs and aimed at comparing transcriptome data at different stages of eMSC cultivation.<br /></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion9_640602302" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694784630342"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 700;">Studying the transcriptome of Sertoli-like cells to determine their origin and application potential</span>| <span style="font-weight: 100;">Zoological Institute RAS</span><br /><br /><span style="font-weight: 700;">student:</span> <span style="font-weight: 100;">Valerii Mun</span><br /><span style="font-weight: 700;">supervisor: </span><span style="font-weight: 100;">Elena Novikova</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1UGRZdHEsjdYOIcIstUjaq1CD2HqSphSpO76I5hhaqUc/edit#slide=id.p1" target="_blank" style="">Slides</a><br /><a href="https://github.com/valeriy2125/Sertoly_segm" target="_blank" style="">Github</a></span></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion9_640602302"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694784630342"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 100;">Sertoli cells (SC) are necessary for spermatogenesis in mammals. They are located in the spermatogenic epithelium of the seminiferous tubules of the testis and provide development and protection of germ cells. Such an important role makes the SCs a promising target for the development of in vitro spermatogenesis systems or techniques for the treatment of male infertility. However, the proliferation of this cell type in culture is very difficult. <br /> Recently, our laboratory discovered a new cell population in the rete testis area, which we named Sertoli-like cells (SLCs). They possess similar marker genes to SCs, but active proliferation in culture. In addition, under 3D culture conditions with neonatal SCs, SLCs are able to support the development of germ cells up to the initial stages of meiosis. Thus, SLCs could be an alternative to neonatal SCs in in vitro spermatogenesis technology, but the transcriptome of SLCs needs to be studied for a greater understanding of their origin and potential application. <br /> The aim of the current work is to identify markers of mouse Sertoli-like cells. <br /></span></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion10_640602302" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694784631830"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 700;">Spatial mapping of potential enzymes encoded in the global ocean microbiome</span>| <span style="font-weight: 100;">Stockholm Resilience Centre, Stockholm University</span><br /><br /><span style="font-weight: 700;">students:</span> <span style="font-weight: 100;">Polina Guseva, Anna Zhurakovskaya</span><br /><span style="font-weight: 700;">supervisor: </span><span style="font-weight: 100;">Erik Zhivkoplias</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1eifYr3fTBCv1kHPUX2OvX69BbsoQ4Hco4zLoOjIh8Xg/edit?usp=sharing" target="_blank" style="">Slides</a><br /><a href="https://github.com/GusevaPolina/FreeMetagenomics" target="_blank" style="">Github</a></span></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion10_640602302"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694784631830"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">The aim of the project is to investigate the biotechnological potential of marine organisms for the purpose of discovering natural products in areas beyond national jurisdiction (ABNJ). Our goal is to compare the biosynthetic gene clusters identified in metagenomes sampled from ABNJ with those present in species that have already been utilized in genome-based bioprospecting efforts. <br /></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion11_640602302" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694784632507"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 700;">Studying thymic selection with deep profiling of T-cell receptor repertoires</span>| <span style="font-weight: 100;">ITMO University; Russian National Research Medical University</span><br /><br /><span style="font-weight: 700;">student:</span> <span style="font-weight: 100;">Luppov Daniil</span><br /><span style="font-weight: 700;">supervisors: </span><span style="font-weight: 100;">Elizaveta Vlasova, Michail Shugay</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/12NM-7CLGjYuhLo4dROhbbfClH7IH4oHbi5svjMNscro/edit?usp=sharing" target="_blank" style="">Slides</a><br /><a href="https://github.com/antigenomics/tcr-thymic-selection/tree/master" target="_blank" style="">Github</a></span></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion11_640602302"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694784632507"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">T-cells are one of the key elements of the adaptive immune system serving to irradiate infected cells, guide immune responses to foreign pathogens and form immunological memory. During T-cell genesis the T-cell receptor (TCR) gene determining antigen specificity is formed by the process called V(D)J rearrangement. TCR genes are screened during thymic selection, a complex physiological process which serves to ensure that T-cells that are both functional and non-self-reactive remain in the body. Only about 1% of all cells which start their way through thymus survive [1]. This process is not fully understood and key factors guiding TCR selection are subject of ongoing research.</div></div></div></div></div></div><div class="t585__border" style="height: 1px; background-color: #adadba"></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t585_init', function () {
t585_init('640602302');
});
});</script><style> #rec640602302 .t585__title { font-size: 24px; color: #3c3c50; font-family: 'Open Sans'; } #rec640602302 .t585__text { font-size: 20px; color: #3c3c50; font-family: 'Open Sans'; }</style></div><div id="rec640627857" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-animationappear="off" data-record-type="585" ><!-- T585 --><div class="t585"><div class="t-container"><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion1_640627857" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694788048334"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 700;">Sufficient mutation landscape for machine learning of antibody binding</span>| <span style="font-weight: 100;">Amsterdam University Medical Centers</span><br /><br /><span style="font-weight: 700;">students:</span> <span style="font-weight: 100;">Alexander Gavrilenko, Ekaterina Grigoreva</span><br /><span style="font-weight: 700;">supervisor:</span> <span style="font-weight: 100;">Dasha Balashova</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1hKg5J5z66Mhy7L3woVDiQwSxoUDeROM8r3GXaV2SvUY/edit#slide=id.p1" target="_blank" style="">Slides</a><br /><a href="https://github.com/GavrilenkoA/ML_mutational_learning" target="_blank" style="">Github Alexander</a>, <a href="https://github.com/grigorievaekaterina/BI_antibodies_project" target="_blank" style="">Github Ekaterina</a></span></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion1_640627857"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694788048334"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">Since the emergence of the Covid-19 pandemic, there has been an accumulation of data on SARS-CoV2 mutations. The evolution of SARS-CoV-2 variants is primarily driven by mutations in the ACE2 receptor-binding domain (RBD), which aids in the spread of the virus within the body. The RBD is the target site for neutralizing antibodies to bind, and the ability of antibodies to bind to this site determines the spread of the disease. In situations where there is a wealth of mutation data, machine learning (ML) models can be trained to predict ACE2 binding and antibody evasion. However, neutralizing antibodies to SARS-CoV-2 share common sequence and structural features, leading to their categorization into four common classes based on groups of targeted RBD epitopes [1]. We hypothesize that common mutations affecting RBD binding could be used as additional data for training ML models with limited data. This approach could be used to estimate the effectiveness of therapeutic antibodies in binding RBD without requiring time-consuming deep mutational scanning experiments [2]. <br /></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion2_640627857" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694788270992"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 700;">DNsorder: An Application for Designing DNAzymes Sensors Detection of Nucleic Acids</span>| <span style="font-weight: 100;">Chemistry Department, University of Central Florida</span><br /><br /><span style="font-weight: 700;">student:</span> <span style="font-weight: 100;">Viktor Smirnov</span><br /><span style="font-weight: 700;">supervisor:</span> <span style="font-weight: 100;">Dmitriy Kolpashchikov</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1BFOoTPNIO68T3xPx4iKJoG1f97kWDw_LT9nkO6IRJ-0/edit#slide=id.p1" target="_blank" style="">Slides</a><br /><a href="https://github.com/Smirness/DNsorder" target="_blank" style="">Github</a></span></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion2_640627857"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694788270992"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">DNAzymes are now widely used in biosensing. These functional oligonucleotides can act as molecular recognition elements in biosensors, providing high specificity to the target analyte. DNAzyme-based sensors for nucleic acid detection are small molecules designed to detect specific DNA or RNA sequences (analytes). In the absence of an analyte, such sensors are catalytically inactive. Binding the analyte to the analyte recognition site triggers the catalytic activity, which can be translated into fluorescent or visually detectable signals. Depending on their implementation, there are those that are sensitive to SNPs, they may be able to distinguish between strains of viruses and microorganisms. <br /></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion3_640627857" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694788272647"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 700;">Hybrid assembly of the Myodes centralis genome</span>| <span style="font-weight: 100;">The Zoological Institute RAS</span><br /><br /><span style="font-weight: 700;">student:</span> <span style="font-weight: 100;">Bogdan Gerda</span><br /><span style="font-weight: 700;">supervisors:</span> <span style="font-weight: 100;">Olga Bondareva, Gennadiiy Zakharov, Semen Bodrov</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1oB9XRPUCt79T0nbbdnD2Kl0ONRv35mzPsiCNjgjmjus/edit#slide=id.g247fd7cd618_0_0" target="_blank" style="">Slides</a><br /><a href="https://github.com/ldelt/Myodes-centralis-assembly" target="_blank" style="">Github</a></span></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion3_640627857"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694788272647"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">The lack of a suitable reference is one of the key problems faced by phylogenetic studies. Existing reference assemblies are often evolutionarily too far from the studied organism, which complicates the study of the phylogenetics of the latter. To study the hybridization of the Arvicolinae subfamily, the Laboratory of Evolutionary Genomics and Paleogenomics of The Zoological Institute RAS decided to assemble its own reference of Myodes centralis, based on Oxford Nanopore and Ilumina RNA-Seq sequencing data.</div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion4_640627857" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694788273565"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 700;">SANS Serif optimization</span>| <span style="font-weight: 100;">All-Russia Research Institute for Agricultural Microbiology</span><br /><br /><span style="font-weight: 700;">students:</span> <span style="font-weight: 100;">Rustam Basyrov, Zaur Kambiev</span><br /><span style="font-weight: 700;">supervisor:</span> <span style="font-weight: 100;">Kirill Antonets</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1pb47MpDdRyZGVZxEW-c-fS5ktJtKH9Ts/edit?usp=share_link&amp;ouid=110852671299930888074&amp;rtpof=true&amp;sd=true" target="_blank" style="">Slides</a><br /><a href="https://github.com/kambizaur/BI2023_SANS_serif_optimization" target="_blank" style="">Github</a></span></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion4_640627857"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694788273565"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">SANS Serif is a tool for whole-genome-based phylogeny estimation that follows a pangenomic approach to efficiently calculate a set of splits in a phylogenetic tree or network. [1]. It doesn't use multiple alignment to tree reconstruction, but still takes a lot of time to work. In the current research, we aimed to find out bottlenecks in tool execution and tried to optimize them.</div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion5_640627857" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694788275102"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 700;">Optimizing the search for significant SNPs associated with important agronomic traits in plants</span>| <span style="font-weight: 100;">Nova plant</span><br /><br /><span style="font-weight: 700;">students:</span> <span style="font-weight: 100;">Liliia Bogdanova, Maryia Moladava</span><br /><span style="font-weight: 700;">supervisors:</span> <span style="font-weight: 100;">Elena Grigoreva, Lavrentiy Danilov</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1Hk2NBXU2Hwx49XzpW6yzJcfAmH0IHfqzQ0aMnrZWWd8/edit?usp=sharing" target="_blank" style="">Slides</a><br /></span></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion5_640627857"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694788275102"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">The identification of the genetic loci associated with agriculturally important phenotypes is a crucial task in plant selection, since it helps to optimize breeding strategies and accelerate crop breeding [1]. Genome-wide association study (GWAS) is a powerful tool commonly used to identify genetic variants responsible for the phenotype of interest. Most GWAS models are developed and optimized using human datasets and have limited applicability to plant studies, especially when mapping complex traits such as stress tolerance and yield [2]. GWAS in plant species should take into account specific characteristics of plants such as polyploidy, a complex genome and a complex population structure. <br /> In this work, we aim to analyze the existing approaches for GWAS analysis in plants, to test them on real data of plant genotypes and phenotypes, and to select the optimal approaches. <br /></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion6_640627857" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694789936894"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 700;">SNP grouping and evaluation of epistatic interaction for phenotypic traits in agricultural plants</span>| <span style="font-weight: 100;">Nova plant</span><br /><br /><span style="font-weight: 700;">students:</span> <span style="font-weight: 100;">Anastasiia Zolotar, Anna Balan, Sergei Volkov</span><br /><span style="font-weight: 700;">supervisors:</span> <span style="font-weight: 100;">Elena Grigoreva, Lavrentiy Danilov</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1MfTteWvJi0j6dESEybKsnl5rsYQrYmfZ" target="_blank" style="">Slides</a><br /></span></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion6_640627857"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694789936894"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">An epistatic interaction between single nucleotide polymorphisms (SNPs) can have important implication for understanding the genetic basis of complex traits and diseases. Epistasis refers to a phenomenon when an impact of one SNP on some trait significantly depends on a genotype of one or more other loci. In other words, the effect of one SNP on the phenotypic trait can be modified by the presence or absence of another SNP, leading to complex genetic interactions. Analysis of epistatic interactions between SNPs could facilitate search of genetic variants that contribute to complex traits beyond the additive effects of individual SNPs [1]. <br /> Researching epistatic interactions between SNPs is particularly important in agriculture technologies for two linked reasons: plants are often exposed to a wide range of environmental conditions, and complex genetic mechanisms underlying stress tolerance often involve multiple genes and pathways. <br /> Additionally, identifying epistatic interactions between SNPs in plants can help to elucidate the complex genetic pathways underlying stress responses, and identify new targets for crop improvement. For example, identifying epistatic interactions between SNPs involved in photosynthesis, water use efficiency, and stress signaling pathways could provide insights into how plants respond and adapt to environmental stresses like drought, heat, and cold. <br /> In this work, we aimed to find a software and workflow for grouping single nucleotide polymorphisms (SNPs) based on their association with a quantitative trait and performing epistatic analysis. <br /></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion7_640627857" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694788276798"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 700;">Development of DNA methylation data imputation algorithm for RRBS and WGBS protocols</span>| <span style="font-weight: 100;">Bioinformatics Institute</span><br /><br /><span style="font-weight: 700;">student:</span> <span style="font-weight: 100;">Emiliia Lekhman</span><br /><span style="font-weight: 700;">supervisors:</span> <span style="font-weight: 100;">Oleg Shpynov, Roman Chernyatchik</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1KEva2-LVCYS6W-N_-ftiouLbncGihgdvxmZufj1rG7U/edit" target="_blank" style="">Slides</a><br /><a href="https://github.com/emi-lex/methylation_jb" target="_blank" style="">Github</a></span></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion7_640627857"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694788276798"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">The thesis focuses on DNA methylation data imputation in RRBS and WGBS protocols, enhancing the accuracy of methylation clocks and enabling more precise age predictions. By addressing missing data challenges, the algorithms improve the quality of derived epigenetic clocks. <br /> In summary, this work proposes an efficient imputation algorithm to enhance the accuracy of DNA methylation analysis and predictions. It addresses missing data challenges, improves the reliability of applications relying on DNA methylation, and benefits aging research, disease prognosis, and personalized medicine. <br /></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion8_640627857" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694788277230"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 700;">Development of in-silico models of metagenomic communities with given properties and a pipeline for their generation</span>| <span style="font-weight: 100;">ITMO University</span><br /><br /><span style="font-weight: 700;">students:</span> <span style="font-weight: 100;">Anna Chechenina, Nikita Vaulin</span><br /><span style="font-weight: 700;">supervisor:</span> <span style="font-weight: 100;">Artem Ivanov, Vladimir Ulyantsev</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1-WaT3MQdpGwnPFK9pzK7y2lZ8pjSytBmfmkdqJ-XSXk/edit?usp=share_link" target="_blank" style="">Slides</a><br /><a href="https://github.com/nvaulin/samovar" target="_blank" style="">Github</a></span></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion8_640627857"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694788277230"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">Metagenomics – study of the microbial communities – is one of the most rapidly evolving disciplines within biology. It provides valuable insights into the evolutionary, biochemical, and biophysical dynamics across diverse environments. The typical metagenomics workflow includes sample preparation, library construction, sequencing, and subsequent analysis of next-generation sequencing (NGS) data. Given the complex composition of these communities, accurate determination of the diversity and functional annotation requires substantial computational resources and advanced algorithms. As a result of the increasing volume of the generated data, numerous research groups are working on the development of specific tools for analyzing metagenomics data. <br /> The validation of the results is a crucial stage in tool development process. This necessitates the availability of small- to middle-sized datasets with predefined properties, which can be utilized to compare the known characteristics with those reported by the tool. Although public databases contain abundant raw and preprocessed data, the challenge of obtaining small NGS metagenomics datasets with known properties persists. <br /></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion9_640627857" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694788279293"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 700;">Taxonomy of the Serratia genus</span>| <span style="font-weight: 100;">All-Russia Research Institute of Agricultural Microbiology</span><br /><br /><span style="font-weight: 700;">student:</span> <span style="font-weight: 100;">Ekaterina Sukhinina</span><br /><span style="font-weight: 700;">supervisor:</span> <span style="font-weight: 100;">Kirill Antonets</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1RrxFbDjS23riYS5Ge3sFfqa0HwVkC-d2/edit#slide=id.p1" target="_blank" style="">Slides</a><br /><a href="https://github.com/shkot190819/serratia" target="_blank" style="">Github</a></span></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion9_640627857"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694788279293"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">In 2020, two new species of Serratia, S. nevei and S. bockelmannii, were described in the Serratia genus, which includes a variety of pathogenic bacteria from different animal and plant groups, and they were distinguished primarily on the basis of biochemical and morphological characters [1]. Some of the genome assemblies in the NCBI database annotated as S. marcescens and not taxonomically validated most likely belong to two new species. <br /> The aim of our work was to investigate whether the subdivision into species of the genus Serratia is consistent with the actual phylogeny. <br /></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion10_640627857" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694788279826"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 700;">Selection of antibodies humanized by scaffolding</span>| <span style="font-weight: 100;">BIOCAD</span><br /><br /><span style="font-weight: 700;">student:</span> <span style="font-weight: 100;">Nikita Vyatkin</span><br /><span style="font-weight: 700;">supervisors:</span> <span style="font-weight: 100;">Georgiy Mamistvalov, Natalia Zenkova, Vlad Strashko</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1-lexRiPT-LantplnwVMBKoxGrAvplCL4N8KONQHpHZU/edit?usp=share_link" target="_blank" style="">Slides</a><br /><a href="https://github.com/Nik9kin/antibody_humanization_by_scaffolding" target="_blank" style="">Github</a></span></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion10_640627857"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694788279826"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">Therapeutic monoclonal antibodies (mAbs) are widely used to treat various human diseases, including cancer, infections, and immune disorders [1]. Immunizing mice with the target antigen (Ag) and generating a hybridoma to obtain the Ag-specific antibody (Ab) is the conventional procedure used to develop effective antibodies [2]. Murine mAbs cannot be directly applied to humans because, due to their foreignness, they themselves will elicit an immune response; therefore, such antibodies must be humanized. <br /> A common method for humanizing antibodies involves grafting complementarity-determining regions (CDRs) into a suitable human template. Then, several framework residues of humanized antibodies mutate back to those of mice, this process is called back mutation [3, 4]. Back mutations are done to increase the likelihood of retaining functionality of humanized antibodies. Since the variety of possible back mutations is large, a large number of humanized candidates arise from which one must choose those with the best Ag-binding affinity. <br /> In this work, we implemented a method for comparing humanized candidates based on their behavior in molecular dynamics simulations, described by Hsieh et al. [5]. The advantage of this approach is the absence of the need for an Ab-Ag cocrystal structure. Using the implemented pipeline, we compared humanized anti-TNF-ɑ mAb (Remicade) candidates and compared our results with those from the article. <br /></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion11_640627857" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694788283080"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 700;">Non-canonical start codons and where to find them</span>| <span style="font-weight: 100;">Institute of Science and Technology Austria</span><br /><br /><span style="font-weight: 700;">students:</span> <span style="font-weight: 100;">Valeriia Rubinova, </span><span style="font-weight: 100;">Bogdan Sotnikov</span><br /><span style="font-weight: 700;">supervisor:</span> <span style="font-weight: 100;">Olga Bochkareva</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1ckpmP_OfCg32V86HXWfR73QX8Udt9ECQ/edit?usp=share_link&amp;ouid=111857435385401051838&amp;rtpof=true&amp;sd=true" target="_blank" style="noopener">Slides</a><br /><a href="https://github.com/Asklepiad/BI_project_2022" target="_blank" style="">Github</a></span></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion11_640627857"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694788283080"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">It is known prokaryotes use triplets other than ATG as a start codon. There are about 10% of genes in the bacterial genome (a more strict proportion depends on taxon) with non-canonical start-codons (NCSCs): GTG, TTG and others in negligibly small proportion. There is not much data about the causes of the existence and "gene preferences" of NCSCs. The GTG and TTG usually bind weaker with the ribosome relative to the canonical ATG start codon. <br /> The main aims and objectives of our work were: <br /><ol><li>Create the pipeline for analysis of the conservation of gene start codons in RefSeq bacterial database. </li><li> Apply the pipeline to the dataset comprising the organisms from different ecological niches. </li><li> Reveal the connections between the gene start codon and its function.</li><li>Evaluate the distribution of bacterial NCSCs in different taxons and find the difference (if it exists) between different groups of bacteria by their function-start-codons patterns.</li></ol></div></div></div></div></div></div><div class="t585__border" style="height: 1px; background-color: #adadba"></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t585_init', function () {
t585_init('640627857');
});
});</script><style> #rec640627857 .t585__title { font-size: 24px; color: #3c3c50; font-family: 'Open Sans'; } #rec640627857 .t585__text { font-size: 20px; color: #3c3c50; font-family: 'Open Sans'; }</style></div><div id="rec545000636" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="font-size:24px;color:#3c3c50;" data-customstyle="yes"><strong></strong><strong>Association between depressive disorders and various factors in adolescents by country</strong><span style="font-weight: 100;"><strong><br /></strong><br /><strong>students: </strong>E. Khimich, A. Selezneva<br /><strong>supervisor:</strong> D. Serebrennikov<br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vTv7hDHKzOD4-PP_wNDRHE5vJ2hxf5UzI9JbQMhRCx2Bgp7XoJD_z7OXRUuCCHTlw/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style="">Slides</a><br /><a href="https://github.com/Khmch/Project_depression_biostat2022" target="_blank" style="">GitHub</a></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"><div style="font-size:18px;" data-customstyle="yes"></div></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec548237132" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="font-size:24px;color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong></strong><strong>Analysis of the «<a href="https://charityhospital.ru" target="_blank">Charity hospital</a>» database</strong><br /><br /><strong>team 1:</strong> I. Maslova, I. Tsepeleva, A. Bydanov, P. Pchelintseva<br /><strong>supervisors:</strong> E.Tomilov, S. Ievkov<span class="redactor-invisible-space"><br /></span><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vSEr2k4OE82OMwM6zS8CrJsHJf0ogDqx1uB_kOAA8VHBqBcu49dkUgQBQmgROXyCA/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style="">Slides</a><br /><a href="https://github.com/User00342/BioStat_2022/blob/main/%D0%9D%D0%BE%D0%B2%D0%BE%D0%B5/Readme.md" target="_blank" style="">Github</a><br /><br /><strong>team 2:</strong> E. Fokina, E. Potapenko, L. Golubnikova<br /><strong>supervisors:</strong> M. Slavenko, S. Ievkov<br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vQrRg3n459sZIJ7Bm7DrA6amuN1r8RWsB0C3lyiBOf00edCZ4DdbETCfhbnGZvX1w/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style="">Slides</a><br /><a href="https://github.com/golubnikova/charity-hospita" target="_blank" style="">Github</a></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"><div style="font-size:18px;" data-customstyle="yes"></div></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec548237123" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="font-size:24px;color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong><em data-redactor-tag="em"></em></strong><strong></strong><strong>Comparison of missing data imputation methods for evaluation reference values of morphometric parameters of human fetus in the perinatal period</strong><strong><br /></strong><br /><strong>students:</strong> N. Sheiko, E. Ivanova, A. Antonova, E. Molokova<br /><strong>supervisors:</strong> E. Tomilov, D. Zhakota<br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vQZXX9GG2yLcHu-VkpW3HRKrMNt-U7hSA7Oc9Eia3vgrDr3joVhaDehbsu6Fur9qw/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style="">Slides</a><br /><a href="https://github.com/growfins/FetalLosses" target="_blank" style="">GitHub</a><br /></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"><div style="font-size:18px;" data-customstyle="yes"></div></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec548237113" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="font-size:24px;color:#3c3c50;" data-customstyle="yes"><strong></strong><strong>Calculate reference values of morphometric parameters with R in perinatal period of human development</strong><span style="font-weight: 100;"><br /><br /><strong>students:</strong> A.I. Kulgaeva, K. Zhur, N. Tsurikova, V. A. Dravgelis<br /><strong>supervisors: </strong> M. Slavenko, D. Zhakota<br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vTNvxPWyDm8Q-ZDsyasyeJgKbJVfQYXkocihzMTJvFoneOx-qitYwelIr5jwraK9w/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style="">Slides</a><br /><a href="https://github.com/Dv1t/potan_biostat" target="_blank" style="">GitHub</a></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"><div style="font-size:18px;" data-customstyle="yes"></div></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec548236589" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="font-size:24px;color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong><em data-redactor-tag="em"></em></strong><strong></strong><strong>I</strong><strong>dentification of genes associated with immune response using open database ImmuneSpace</strong><strong></strong><br /><br /><strong>students:</strong> O. Mironenko, S. Suleimanov, O. Ilina<br /><strong>supervisors:</strong> M. Kuznetsov, E. Bakin<br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vT6PQw4itIXVNmFXK0VOBQ8Yd9x-dcPpHPo-fenDLl1Y8Xhj0ORNRT-k3-eYQn9tLVrB-fkh64uXeOt/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style="noopener">Slides</a><br /><a href="https://github.com/ejeej/ImmuneSpace_SDY984" target="_blank" style="">GitHub</a></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"><div style="font-size:18px;" data-customstyle="yes"></div></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec548250724" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-record-type="280" ><!-- T260--><div class="t260"><div class="t-container"><div class="t-col t-col_11 "><div class="t260__accordion" style="border-bottom: 1px solid #adadba;"><div class="t260__wrapper"><div class="t260__header" style="border-top: 1px solid #adadba;"><div class="t260__title t-name t-name_xl" field="title"><div style="font-size:24px;color:#3c3c50;" data-customstyle="yes"><span style="font-weight: 100;"><strong><em data-redactor-tag="em"></em></strong><strong></strong><strong></strong><strong></strong><strong>Biostatistical analysis of melanoma patients transcriptomic data from open database TCGA</strong><strong></strong><strong></strong><br /><br /><strong>students:</strong> O. Bagrova, D. Zubkov, D. Shavronskaya<br /><strong>supervisor:</strong> M. Kuznetsov, E. Bakin<br /><br /><a href="https://docs.google.com/presentation/d/e/2PACX-1vQKOM5RqXwsD2HeKNAscHPK7fHin8LMDpHEmL25kCjVMktdVGMFa27bmc0aJy-pkzzKn9l6PJPwdVTQ/pub?start=false&amp;loop=false&amp;delayms=3000" target="_blank" style="">Slides</a><br /><a href="https://github.com/flajole/TCGA_SKCM" target="_blank" style="">GitHub</a></span></div></div> <div class="t260__icon"><div class="t260__lines"><div class="t260__iconline t260__iconline_left" style="background: #3c3c50"></div><div class="t260__iconline t260__iconline_right" style="background: #3c3c50"></div></div><div class="t260__circle" style="background: #ffffff"></div></div></div><div class="t260__content"><div class="t260__textwrapper"><div class="t260__text t-descr t-descr_xs" field="descr"><div style="font-size:18px;" data-customstyle="yes"><div style="font-size:18px;" data-customstyle="yes"></div></div></div> </div></div></div></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t260_init', function () {
t260_init();
});
});</script></div><div id="rec640439907" class="r t-rec t-rec_pt_0" style="padding-top:0px; " data-animationappear="off" data-record-type="585" ><!-- T585 --><div class="t585"><div class="t-container"><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion1_640439907" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__2788175426422"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><strong></strong><span style="font-weight: 700;">The reproducibility of the Crustacean phylogenetic analyses on open source data</span>| <span style="font-weight: 100;">Irkutsk State University</span><br /><br /><span style="font-weight: 700;">students:</span> <span style="font-weight: 100;">Aleksandra Beliaeva, Alexandr Zhuravlev</span><br /><span style="font-weight: 700;">supervisor:</span> <span style="font-weight: 100;">Polina Drozdova</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1Sq9BtzIzBSC5AY4FtoBVt3B3Gy0ZlCsq" style="" target="_blank">Slides</a><br /></span><a href="https://github.com/BelyaevaAlex/-Crustacean-hoods-strive-for-power" style="" target="_blank"><span style="font-weight: 100;">Github</span></a></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion1_640439907"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__2788175426422"><div style="font-family:'Open Sans';" data-customstyle="yes">This project examines the issue of phylogenetic analysis reproducibility in several studies on freshwater crustaceans based on open data:<br /><ol><li>Moskalenko, Victoria N., Tatiana V. Neretina, and Lev Y. Yampolsky. "To the origin of lake baikal endemic gammarid radiations, with description of two new Eulimnogammarus spp." <em>Zootaxa</em> 4766.3 (2020): 457-471.</li><li>Bystřický, Pavel Karel, et al. "Distribution patterns at different spatial scales reveal reproductive isolation and frequent syntopy among divergent lineages of an amphipod species complex in Western Carpathian streams." Limnology and Oceanography 67.12 (2022): 2796-2808.</li><li>Bukin, Yu S., J. V. Petunina, and D. Yu Sherbakov. "The Mechanisms for genetic diversity of Baikal endemic amphipod Gmelinoides fasciatus: Relationships between the population processes and paleoclimatic history of the lake." Russian Journal of Genetics 54 (2018): 1059-1068.</li><li>Wattier, Remi, et al. "Continental-scale patterns of hyper-cryptic diversity within the freshwater model taxon Gammarus fossarum (Crustacea, Amphipoda)." Scientific reports 10.1 (2020): 16536</li></ol><ol></ol><ol></ol></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion2_640439907" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694768844218"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 700;">The diversity of opsins in Lake Baikal amphipods (Amphipoda: Gammaridae) </span>| <span style="font-weight: 100;">Irkutsk State University</span><br /><br /><span style="font-weight: 700;">student:</span> <span style="font-weight: 100;">Natalia Dudko</span><br /><span style="font-weight: 700;">supervisor: </span><span style="font-weight: 100;">Polina Drozdova</span><br /><span style="font-weight: 100;"><br /><a href="https://docs.google.com/presentation/d/1mlKDBTSiUbVEMuLFnjTs8T5N1saDotgPVHcQhYafRho/edit?usp=sharing" style="" target="_blank">Slides</a><br /></span><a href="https://github.com/dudko-pixel/Phylogen_project_opsins" style="" target="_blank"><span style="font-weight: 100;">Github</span></a></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion2_640439907"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694768844218"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">This project aimed to study the diversity of opsins in Baikal and other amphipods and investigate how they relate to their environments and evolutionary paths based on article:<br />Naumenko, Sergey A., et al. "Transcriptome‐based phylogeny of endemic Lake Baikal amphipod species flock: fast speciation accompanied by frequent episodes of positive selection." Molecular ecology 26.2 (2017): 536-553.</div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion3_640439907" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694768847534"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 700;">Pervasive duplication of tumor suppressors in Afrotherians during the evolution of large bodies and reduced cancer risk </span>| <span style="font-weight: 100;">LOEWE Centre for Translational Biodiversity Genomics</span><br /><br /><span style="font-weight: 700;">students:</span> <span style="font-weight: 100;">Lev Igamberdiev, Anastasiia Bukreeva, Valeria Lobanova</span><br /><span style="font-weight: 700;">supervisor:</span> <span style="font-weight: 100;">Yury Malovichko</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1hPmT2tPZ_9sMpbpzNdEH1BBDUn9g_CJV" style="" target="_blank">Slides</a><br /></span><a href="https://github.com/bi-LVYproject-2023/vazquez_2021" style="" target="_blank"><span style="font-weight: 100;">Github</span></a></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion3_640439907"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694768847534"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">This project aimed to reproduce the results of a study investigating the association between body mass, gene copy number of tumor suppressors, and cancer risk in mammals<br />Vazquez, Juan M., and Vincent J. Lynch. "Pervasive duplication of tumor suppressors in Afrotherians during the evolution of large bodies and reduced cancer risk." Elife 10 (2021): e65041. <br /></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion4_640439907" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694768848230"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 600;"><strong>Global gene expression of NSCLC TIL</strong> </span>| <span style="font-weight: 100;">BioNTech US</span><br /><br /><span style="font-weight: 700;">students: </span><span style="font-weight: 100;">Gleb Dilman, Irina Bogdanova</span><br /><span style="font-weight: 700;">supervisor:</span> <span style="font-weight: 100;">Ekaterina Esaulova</span><br /><br /><span style="font-weight: 100;"><a href="https://docs.google.com/presentation/d/1Ox-kRXvtKV6-rVOZ5BIh9rsqiH-IXytjMsCJah16BCM/edit?usp=sharing" style="" target="_blank">Slides</a><br /></span><a href="https://github.com/Irina-Bogdanova-bioinformatician/Research_project_BIOINF_2023" style="" target="_blank"><span style="font-weight: 100;">Github</span></a></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion4_640439907"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694768848230"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">The aim of the research was to reproduce the study of the transcription profile of tumor-infiltrating lymphocytes (TIL) described in the article<br />Caushi, Justina X., et al. "Transcriptional programs of neoantigen-specific TIL in anti-PD-1-treated lung cancers." Nature 596.7870 (2021): 126-132.<br /></div></div></div></div></div></div></div><div class="t-col t-col_11 "><div class="t585__accordion" data-accordion="false" data-scroll-to-expanded="false"><div class="t585__wrapper"><div class="t585__header " style="border-top: 1px solid #adadba"><button type="button" 
class="t585__trigger-button" 
aria-controls="accordion5_640439907" 
aria-expanded="false"
><span class="t585__title t-name t-name_xl" field="li_title__1694770233120"><div style="font-size:24px;font-family:'Open Sans';" data-customstyle="yes"><span style="font-weight: 700;">Transcriptional programs of neoantigen-specific TIL in anti-PD-1-treated lung cancers</span> | <span style="font-weight: 100;">BioNTech US</span><br /><br /><span style="font-weight: 700;">students:</span> <span style="font-weight: 100;">Fedor Grigoryev, Kseniia Dracheva</span><br /><span style="font-weight: 700;">supervisor:</span> <span style="font-weight: 100;">Ekaterina Esaulova</span><br /><span style="font-weight: 100;"><br /><a href="https://docs.google.com/presentation/d/1WLJFi-QM_nPk2lVd9J6icdrRmwQVKnV8/" style="" target="_blank">Slides Fedor</a>, <a href="https://docs.google.com/presentation/d/1LgIvsyPS-zfb_HAdX0kdvOEsM_xwzjzh" style="" target="_blank">Slides Kseniia</a><br /><a href="https://github.com/fedor-grigoryev/bi-2023-tcr-tils" style="" target="_blank">Github Fedor</a>, </span><a href="https://github.com/Kseniadv/BI_project_single_cell" style="" target="_blank"><span style="font-weight: 100;">Github Kseniia</span></a></div></span><span class="t585__icon"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: transparent;"></span></span><span class="t585__icon t585__icon-hover"><span class="t585__lines"><svg role="presentation" focusable="false" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1px" fill="none" fill-rule="evenodd" stroke-linecap="square"><g transform="translate(1.000000, 1.000000)" stroke="#222222"><path d="M0,11 L22,11"></path><path d="M11,0 L11,22"></path></g></g></svg></span><span class="t585__circle" style="background-color: #eee;"></span></span></button></div><div class="t585__content" 
id="accordion5_640439907"
hidden ><div class="t585__textwrapper"><div class="t585__text t-descr t-descr_xs" field="li_descr__1694770233120"><div style="font-size:20px;font-family:'Open Sans';" data-customstyle="yes">Aim of this research was re-processing and replication of the analysis for published single-cell RNAseq and matched TCRseq data: T cells from normal and cancerous tissues from multiple donors<br />Caushi, Justina X., et al. "Transcriptional programs of neoantigen-specific TIL in anti-PD-1-treated lung cancers." Nature 596.7870 (2021): 126-132.</div></div></div></div></div></div><div class="t585__border" style="height: 1px; background-color: #adadba"></div></div></div></div><script>t_onReady(function () {
t_onFuncLoad('t585_init', function () {
t585_init('640439907');
});
});</script><style> #rec640439907 .t585__title { font-size: 24px; color: #3c3c50; font-family: 'Open Sans'; } #rec640439907 .t585__text { font-size: 20px; color: #3c3c50; font-family: 'Open Sans'; }</style></div><div id="rec540355738" class="r t-rec" style=" " data-animationappear="off" data-record-type="407" ><!-- T407 --><div class="t407"><div class="t-container_100 t-card__container "><div class="t407__wrapper"><div class="t407__col t-card__col t-card__col_withoutbtn t407__col_notempty" style="background-color: #fafafa;"><div class="t407__textwrapper"><div class="t407__uptitle t-uptitle t-uptitle_xs" style="color: #3a81e8;color:#3a81e8;" field="subtitle" ><span style="font-weight: 700;">Previous</span></div><a 
class="t-card__link" 
href="/popularization" 
target="" 
><div class="t407__title t-name t-name_lg" style="color: #3a81e8;" field="title" ><span style="font-weight: 300;">Популяризация</span></div></a></div></div><div class="t407__col t-card__col t-card__col_withoutbtn t407__col_notempty" style="background-color: #f7f7f7;"><div class="t407__textwrapper t-align_right"><div class="t407__uptitle t-uptitle t-uptitle_xs" style="color: #3a81e8;color:#3a81e8;" field="subtitle2" ><span style="font-weight: 700;">Next</span></div><a 
class="t-card__link" 
href="/popularization" 
target="" 
><div class="t407__title t-name t-name_lg" style="color: #3a81e8;" field="title2" ><span style="font-weight: 300;">Наука</span></div></a></div></div></div></div></div><script type="text/javascript">t_onReady(function() {
t_onFuncLoad('t407_init', function () {
t407_init('540355738');
});
});</script></div><div id="rec548523247" class="r t-rec t-rec_pt_75 t-rec_pb_90" style="padding-top:75px;padding-bottom:90px;background-color:#ffffff; " data-animationappear="off" data-record-type="420" data-bg-color="#ffffff"><!-- T420 --><div class="t420"><div class="t-container t-align_left"><div class="t420__col t-col t-col_3"><a class="t420__logo-link" href="#" style="color:#ffffff;font-size:20px;text-transform:uppercase;"><img class="t420__logo t-img" 
src="https://static.tildacdn.com/tild6665-3836-4636-b630-346264663537/-/empty/bi_logo.png" data-original="https://static.tildacdn.com/tild6665-3836-4636-b630-346264663537/bi_logo.png" 
imgfield="img"
style="max-width: 60px;" alt="Об институте"
></a><div class="t-sociallinks"><ul role="list" class="t-sociallinks__wrapper" aria-label="Social media links"><!-- new soclinks --><li class="t-sociallinks__item t-sociallinks__item_vk"><a href="https://vk.com/bioinf" target="_blank" rel="nofollow" aria-label="vk" style="width: 25px; height: 25px;"><svg class="t-sociallinks__svg" role="presentation" width=25px height=25px viewBox="0 0 100 100" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M50 100c27.614 0 50-22.386 50-50S77.614 0 50 0 0 22.386 0 50s22.386 50 50 50ZM25 34c.406 19.488 10.15 31.2 27.233 31.2h.968V54.05c6.278.625 11.024 5.216 12.93 11.15H75c-2.436-8.87-8.838-13.773-12.836-15.647C66.162 47.242 71.783 41.62 73.126 34h-8.058c-1.749 6.184-6.932 11.805-11.867 12.336V34h-8.057v21.611C40.147 54.362 33.838 48.304 33.556 34H25Z" fill="#3c3c50"/></svg></a></li><li class="t-sociallinks__item t-sociallinks__item_youtube"><a href="https://www.youtube.com/user/bioinforussia" target="_blank" rel="nofollow" aria-label="youtube" style="width: 25px; height: 25px;"><svg class="t-sociallinks__svg" role="presentation" width=25px height=25px viewBox="0 0 100 100" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M50 100c27.614 0 50-22.386 50-50S77.614 0 50 0 0 22.386 0 50s22.386 50 50 50Zm17.9-67.374c3.838.346 6 2.695 6.474 6.438.332 2.612.626 6.352.626 10.375 0 7.064-.626 11.148-.626 11.148-.588 3.728-2.39 5.752-6.18 6.18-4.235.48-13.76.7-17.992.7-4.38 0-13.237-.184-17.66-.552-3.8-.317-6.394-2.44-6.916-6.218-.38-2.752-.626-6.022-.626-11.222 0-5.788.209-8.238.7-10.853.699-3.732 2.48-5.54 6.548-5.96C36.516 32.221 40.55 32 49.577 32c4.413 0 13.927.228 18.322.626Zm-23.216 9.761v14.374L58.37 49.5l-13.686-7.114Z" fill="#3c3c50"/></svg></a></li><li class="t-sociallinks__item t-sociallinks__item_telegram"><a href="https://t.me/bioinforussia" target="_blank" rel="nofollow" aria-label="telegram" style="width: 25px; height: 25px;"><svg class="t-sociallinks__svg" role="presentation" width=25px height=25px viewBox="0 0 100 100" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M50 100c27.614 0 50-22.386 50-50S77.614 0 50 0 0 22.386 0 50s22.386 50 50 50Zm21.977-68.056c.386-4.38-4.24-2.576-4.24-2.576-3.415 1.414-6.937 2.85-10.497 4.302-11.04 4.503-22.444 9.155-32.159 13.734-5.268 1.932-2.184 3.864-2.184 3.864l8.351 2.577c3.855 1.16 5.91-.129 5.91-.129l17.988-12.238c6.424-4.38 4.882-.773 3.34.773l-13.49 12.882c-2.056 1.804-1.028 3.35-.129 4.123 2.55 2.249 8.82 6.364 11.557 8.16.712.467 1.185.778 1.292.858.642.515 4.111 2.834 6.424 2.319 2.313-.516 2.57-3.479 2.57-3.479l3.083-20.226c.462-3.511.993-6.886 1.417-9.582.4-2.546.705-4.485.767-5.362Z" fill="#3c3c50"/></svg></a></li><!-- /new soclinks --></ul></div><div class="t420__text t-descr t-descr_xxs" style="" field="text">© Bioinformatics Institute</div></div><div class="t420__col t-col t-col_3"><div class="t420__title t-name t-name_xs t420__title_uppercase" field="title" style="color: #3c3c50;">Об институте</div><div class="t420__descr t-descr t-descr_xxs" field="descr" style="color: #3c3c50;"><ul> <li>Команда</li><li><a href="https://bioinf.me/media" target="_blank" style="">Медиа о нас</a></li> <li><a href="http://bioinf.me/press" target="_blank" style="">Фирменный стиль</a></li><li><a href="https://bioinf.me/contacts" style="">Реквизиты</a></li><li><a href="https://bioinf.me/personaldata/gamechangers" style="">Условия конфиденциальности</a></li><li><a href="https://bioinf.me/education/license" style="">Образовательная лицензия</a></li><li><a href="https://bioinf.me/termsofuse" style="">Публичная оферта</a></li></ul></div></div><div class="t420__floatbeaker_lr3"></div><div><div class="t420__col t-col t-col_3"><div class="t420__title t-name t-name_xs t420__title_uppercase" field="title2" style="color: #3c3c50;">направления</div><div class="t420__descr t-descr t-descr_xxs" field="descr2" style="color: #3c3c50;"><ul><li><a href="/education" style="">Образование</a></li><li><a href="/research" style="">Наука</a></li><li><a href="/popularization" style="">Популяризация</a></li><li><a href="/community" style="">Сообщество</a></li><li><a href="https://bioinf.me/jobs" style="">Вакансии</a></li></ul></div></div><div class="t420__col t-col t-col_3"><div class="t420__title t-name t-name_xs t420__title_uppercase" field="title3" style="color: #3c3c50;">Мы в сети</div><div class="t420__descr t-descr t-descr_xxs" field="descr3" style="color: #3c3c50;"><ul><li><a href="https://vk.com/bioinf" target="_blank" style="noopener">ВКонтакте</a></li><li><a href="https://www.youtube.com/user/bioinforussia" target="_blank" style="noopener">YouTube</a></li><li><a href="https://t.me/bioinforussia" target="_blank" style="noopener">Telegram-канал</a></li><li><a href="https://t.me/bioinf_career" target="_blank">Чат про карьеру и образование</a></li></ul></div></div></div></div></div></div></div><!--/allrecords--><!-- Stat --><script type="text/javascript" data-tilda-cookie-type="analytics">setTimeout(function(){ (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
ga('create', 'UA-25060708-1', 'auto');
ga('send', 'pageview');
}, 2000); window.mainTracker = 'user';</script> <script type="text/javascript">if (! window.mainTracker) { window.mainTracker = 'tilda'; }
setTimeout(function(){ (function (d, w, k, o, g) { var n=d.getElementsByTagName(o)[0],s=d.createElement(o),f=function(){n.parentNode.insertBefore(s,n);}; s.type = "text/javascript"; s.async = true; s.key = k; s.id = "tildastatscript"; s.src=g; if (w.opera=="[object Opera]") {d.addEventListener("DOMContentLoaded", f, false);} else { f(); } })(document, window, 'f37288ba2263b15d015d8db22191bc5a','script','https://static.tildacdn.com/js/tilda-stat-1.0.min.js');
}, 2000); </script></body></html>